{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 33,
   "metadata": {},
   "outputs": [],
   "source": [
    "%matplotlib inline\n",
    "import numpy as np\n",
    "import Bio\n",
    "import matplotlib.pyplot as plt\n",
    "from Bio import Phylo\n",
    "from Bio import SeqIO\n",
    "from Bio.SeqRecord import SeqRecord\n",
    "from Bio.Seq import Seq\n",
    "from io import StringIO\n",
    "from Bio import AlignIO\n",
    "#parsed =SeqIO.parse(\"seqs.fa\",\"fasta\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 82,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LOCUS       ACC97716                 252 aa            linear   BCT 11-DEC-2013\n",
      "DEFINITION  Glycolysis [Elusimicrobium minutum Pei191].\n",
      "ACCESSION   ACC97716\n",
      "VERSION     ACC97716.1\n",
      "DBLINK      BioProject: PRJNA19701\n",
      "            BioSample: SAMN00000042\n",
      "DBSOURCE    accession CP001055.1\n",
      "KEYWORDS    .\n",
      "SOURCE      Elusimicrobium minutum Pei191\n",
      "  ORGANISM  Elusimicrobium minutum Pei191\n",
      "            Bacteria; Elusimicrobia; Elusimicrobia; Elusimicrobiales;\n",
      "            Elusimicrobiaceae; Elusimicrobium.\n",
      "REFERENCE   1  (residues 1 to 252)\n",
      "  AUTHORS   Herlemann,D.P., Geissinger,O., Ikeda-Ohtsubo,W., Kunin,V., Sun,H.,\n",
      "            Lapidus,A., Hugenholtz,P. and Brune,A.\n",
      "  TITLE     Genomic analysis of 'Elusimicrobium minutum,' the first cultivated\n",
      "            representative of the phylum 'Elusimicrobia' (formerly termite\n",
      "            group 1)\n",
      "  JOURNAL   Appl. Environ. Microbiol. 75 (9), 2841-2849 (2009)\n",
      "   PUBMED   19270133\n",
      "REFERENCE   2  (residues 1 to 252)\n",
      "  AUTHORS   Lucas,S., Copeland,A., Lapidus,A., Glavina del Rio,T., Pitluck,S.,\n",
      "            Sun,H., Schmutz,J., Larimer,F., Land,M., Hauser,L., Kyrpides,N.,\n",
      "            Anderson,I., Herlemann,D.P.R., Geissinger,O., Ikeda-Ohtsubo,W.,\n",
      "            Kunin,V., Humgenholtz,P., Brune,A. and Richardson,P.\n",
      "  CONSRTM   US DOE Joint Genome Institute\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (28-APR-2008) US DOE Joint Genome Institute, 2800\n",
      "            Mitchell Drive B100, Walnut Creek, CA 94598-1698, USA\n",
      "COMMENT     Method: conceptual translation.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..252\n",
      "                     /organism=\"Elusimicrobium minutum Pei191\"\n",
      "                     /strain=\"Pei191\"\n",
      "                     /db_xref=\"taxon:445932\"\n",
      "     Protein         1..252\n",
      "                     /product=\"Glycolysis\"\n",
      "                     /EC_number=\"5.3.1.1\"\n",
      "     Region          4..251\n",
      "                     /region_name=\"tpiA\"\n",
      "                     /note=\"triosephosphate isomerase; Provisional; PRK00042\"\n",
      "                     /db_xref=\"CDD:234589\"\n",
      "     Site            order(11,13,97,169,175,215,234,236..237)\n",
      "                     /site_type=\"other\"\n",
      "                     /note=\"substrate binding site [chemical binding]\"\n",
      "                     /db_xref=\"CDD:238190\"\n",
      "     Site            order(11,14,46..48,50,53,66,84,87..88,99..100)\n",
      "                     /site_type=\"other\"\n",
      "                     /note=\"dimer interface [polypeptide binding]\"\n",
      "                     /db_xref=\"CDD:238190\"\n",
      "     Site            order(13,97,169)\n",
      "                     /site_type=\"active\"\n",
      "                     /note=\"catalytic triad [active]\"\n",
      "                     /db_xref=\"CDD:238190\"\n",
      "     CDS             1..252\n",
      "                     /locus_tag=\"Emin_0151\"\n",
      "                     /coded_by=\"CP001055.1:159384..160142\"\n",
      "                     /note=\"PFAM: triosephosphate isomerase;\n",
      "                     KEGG: pca:Pcar_1333 triosephosphate isomerase\"\n",
      "                     /transl_table=11\n",
      "                     /db_xref=\"InterPro:IPR000652\"\n",
      "ORIGIN      \n",
      "        1 mtkripfisg nwkmhntiae skalakaiad ckenknnadi miapsytsla avaeavkgsk\n",
      "       61 vavaaqdvhf eekgaftsav apsqikdaga shviighser rslfgdtdai lnkkvaaalk\n",
      "      121 hgltvvfcvg etlaereadk tlgviesqle nglkgftaeq lssmvvayep vwaigtgkta\n",
      "      181 tpeqaqevha strkylssky gedfaskvri vyggsvktdn vdaimaqpdv dgalvggqsl\n",
      "      241 vaeqfiriin fn\n",
      "//\n",
      "\n",
      "\n",
      "LOCUS       TIGAR_HUMAN              270 aa            linear   PRI 12-SEP-2018\n",
      "DEFINITION  RecName: Full=Fructose-2,6-bisphosphatase TIGAR; AltName:\n",
      "            Full=TP53-induced glycolysis and apoptosis regulator; AltName:\n",
      "            Full=TP53-induced glycolysis regulatory phosphatase.\n",
      "ACCESSION   Q9NQ88\n",
      "VERSION     Q9NQ88.1\n",
      "DBSOURCE    UniProtKB: locus TIGAR_HUMAN, accession Q9NQ88;\n",
      "            class: standard.\n",
      "            extra accessions:B2R840\n",
      "            created: Dec 6, 2005.\n",
      "            sequence updated: Oct 1, 2000.\n",
      "            annotation updated: Sep 12, 2018.\n",
      "            xrefs: AJ272206.1, CAC01127.1, AY425618.1, AAQ98969.1, AK313226.1,\n",
      "            BAG36037.1, CH471116.2, EAW88849.1, BC012340.1, AAH12340.1,\n",
      "            NP_065108.1, 3DCY_A\n",
      "            xrefs (non-sequence databases): CCDS:CCDS8525.1, UniGene:Hs.504545,\n",
      "            PDBsum:3DCY, ProteinModelPortal:Q9NQ88, SMR:Q9NQ88, BioGrid:121369,\n",
      "            DIP:DIP-60093N, IntAct:Q9NQ88, STRING:9606.ENSP00000179259,\n",
      "            DEPOD:Q9NQ88, iPTMnet:Q9NQ88, PhosphoSitePlus:Q9NQ88,\n",
      "            BioMuta:TIGAR, DMDM:74734311, EPD:Q9NQ88, MaxQB:Q9NQ88,\n",
      "            PaxDb:Q9NQ88, PeptideAtlas:Q9NQ88, PRIDE:Q9NQ88,\n",
      "            ProteomicsDB:82106, DNASU:57103, Ensembl:ENST00000179259,\n",
      "            Ensembl:ENSP00000179259, Ensembl:ENSG00000078237, GeneID:57103,\n",
      "            KEGG:hsa:57103, UCSC:uc001qmp.4, CTD:57103, DisGeNET:57103,\n",
      "            EuPathDB:HostDB:ENSG00000078237.6, GeneCards:TIGAR, HGNC:1185,\n",
      "            HPA:CAB034010, HPA:HPA040354, HPA:HPA044111, MIM:610775,\n",
      "            neXtProt:NX_Q9NQ88, OpenTargets:ENSG00000078237, PharmGKB:PA25506,\n",
      "            eggNOG:ENOG410IUDB, eggNOG:COG0406, GeneTree:ENSGT00390000013224,\n",
      "            HOGENOM:HOG000060277, HOVERGEN:HBG108569, InParanoid:Q9NQ88,\n",
      "            KO:K14634, OMA:NFEEGRE, OrthoDB:EOG091G0W1L, PhylomeDB:Q9NQ88,\n",
      "            TreeFam:TF329053, BioCyc:MetaCyc:HS01277-MONOMER,\n",
      "            Reactome:R-HSA-5628897, ChiTaRS:TIGAR, EvolutionaryTrace:Q9NQ88,\n",
      "            GeneWiki:C12orf5, GenomeRNAi:57103, PRO:PR:Q9NQ88,\n",
      "            Proteomes:UP000005640, Bgee:ENSG00000078237, CleanEx:HS_C12orf5,\n",
      "            ExpressionAtlas:Q9NQ88, Genevisible:Q9NQ88, GO:0005737, GO:0005829,\n",
      "            GO:0005622, GO:0005741, GO:0005634, GO:0004083, GO:0004331,\n",
      "            GO:0006915, GO:0006914, GO:0071279, GO:0006974, GO:0071456,\n",
      "            GO:0030388, GO:0006003, GO:0060576, GO:1904024, GO:0045820,\n",
      "            GO:1901525, GO:1901215, GO:0043069, GO:2000378, GO:0010666,\n",
      "            GO:0045739, GO:1903301, GO:0043456, GO:1902153, GO:0010332,\n",
      "            GO:0002931, GO:0009410, CDD:cd07067, Gene3D:3.40.50.1240,\n",
      "            InterPro:IPR013078, InterPro:IPR029033, InterPro:IPR001345,\n",
      "            Pfam:PF00300, SMART:SM00855, SUPFAM:SSF53254, PROSITE:PS00175\n",
      "KEYWORDS    3D-structure; Acetylation; Apoptosis; Autophagy; Complete proteome;\n",
      "            Cytoplasm; Hydrolase; Mitochondrion; Nucleus; Reference proteome.\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1  (residues 1 to 270)\n",
      "  AUTHORS   White,K.E., Evans,W.E., O'Riordan,J.L.H., Speer,M.C., Econs,M.J.,\n",
      "            Lorenz-Depiereux,B., Grabowski,M., Meitinger,T. and Strom,T.M.\n",
      "  TITLE     Autosomal dominant hypophosphataemic rickets is associated with\n",
      "            mutations in FGF23\n",
      "  JOURNAL   Nat. Genet. 26 (3), 345-348 (2000)\n",
      "   PUBMED   11062477\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [MRNA].\n",
      "REFERENCE   2  (residues 1 to 270)\n",
      "  AUTHORS   Cheng,J., Dang,X., Wang,J., Ji,D., Wang,C., Yang,Q. and Liu,Y.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (??-SEP-2003)\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [MRNA].\n",
      "REFERENCE   3  (residues 1 to 270)\n",
      "  AUTHORS   Ota,T., Suzuki,Y., Nishikawa,T., Otsuki,T., Sugiyama,T., Irie,R.,\n",
      "            Wakamatsu,A., Hayashi,K., Sato,H., Nagai,K., Kimura,K., Makita,H.,\n",
      "            Sekine,M., Obayashi,M., Nishi,T., Shibahara,T., Tanaka,T.,\n",
      "            Ishii,S., Yamamoto,J., Saito,K., Kawai,Y., Isono,Y., Nakamura,Y.,\n",
      "            Nagahari,K., Murakami,K., Yasuda,T., Iwayanagi,T., Wagatsuma,M.,\n",
      "            Shiratori,A., Sudo,H., Hosoiri,T., Kaku,Y., Kodaira,H., Kondo,H.,\n",
      "            Sugawara,M., Takahashi,M., Kanda,K., Yokoi,T., Furuya,T.,\n",
      "            Kikkawa,E., Omura,Y., Abe,K., Kamihara,K., Katsuta,N., Sato,K.,\n",
      "            Tanikawa,M., Yamazaki,M., Ninomiya,K., Ishibashi,T., Yamashita,H.,\n",
      "            Murakawa,K., Fujimori,K., Tanai,H., Kimata,M., Watanabe,M.,\n",
      "            Hiraoka,S., Chiba,Y., Ishida,S., Ono,Y., Takiguchi,S., Watanabe,S.,\n",
      "            Yosida,M., Hotuta,T., Kusano,J., Kanehori,K., Takahashi-Fujii,A.,\n",
      "            Hara,H., Tanase,T.O., Nomura,Y., Togiya,S., Komai,F., Hara,R.,\n",
      "            Takeuchi,K., Arita,M., Imose,N., Musashino,K., Yuuki,H., Oshima,A.,\n",
      "            Sasaki,N., Aotsuka,S., Yoshikawa,Y., Matsunawa,H., Ichihara,T.,\n",
      "            Shiohata,N., Sano,S., Moriya,S., Momiyama,H., Satoh,N., Takami,S.,\n",
      "            Terashima,Y., Suzuki,O., Nakagawa,S., Senoh,A., Mizoguchi,H.,\n",
      "            Goto,Y., Shimizu,F., Wakebe,H., Hishigaki,H., Watanabe,T.,\n",
      "            Sugiyama,A., Takemoto,M., Kawakami,B., Yamazaki,M., Watanabe,K.,\n",
      "            Kumagai,A., Itakura,S., Fukuzumi,Y., Fujimori,Y., Komiyama,M.,\n",
      "            Tashiro,H., Tanigami,A., Fujiwara,T., Ono,T., Yamada,K., Fujii,Y.,\n",
      "            Ozaki,K., Hirao,M., Ohmori,Y., Kawabata,A., Hikiji,T., Kobatake,N.,\n",
      "            Inagaki,H., Ikema,Y., Okamoto,S., Okitani,R., Kawakami,T.,\n",
      "            Noguchi,S., Itoh,T., Shigeta,K., Senba,T., Matsumura,K.,\n",
      "            Nakajima,Y., Mizuno,T., Morinaga,M., Sasaki,M., Togashi,T.,\n",
      "            Oyama,M., Hata,H., Watanabe,M., Komatsu,T., Mizushima-Sugano,J.,\n",
      "            Satoh,T., Shirai,Y., Takahashi,Y., Nakagawa,K., Okumura,K.,\n",
      "            Nagase,T., Nomura,N., Kikuchi,H., Masuho,Y., Yamashita,R.,\n",
      "            Nakai,K., Yada,T., Nakamura,Y., Ohara,O., Isogai,T. and Sugano,S.\n",
      "  TITLE     Complete sequencing and characterization of 21,243 full-length\n",
      "            human cDNAs\n",
      "  JOURNAL   Nat. Genet. 36 (1), 40-45 (2004)\n",
      "   PUBMED   14702039\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [LARGE SCALE MRNA].\n",
      "REFERENCE   4  (residues 1 to 270)\n",
      "  AUTHORS   Mural,R.J., Istrail,S., Sutton,G.G., Florea,L., Halpern,A.L.,\n",
      "            Mobarry,C.M., Lippert,R., Walenz,B., Shatkay,H., Dew,I.,\n",
      "            Miller,J.R., Flanigan,M.J., Edwards,N.J., Bolanos,R., Fasulo,D.,\n",
      "            Halldorsson,B.V., Hannenhalli,S., Turner,R., Yooseph,S., Lu,F.,\n",
      "            Nusskern,D.R., Shue,B.C., Zheng,X.H., Zhong,F., Delcher,A.L.,\n",
      "            Huson,D.H., Kravitz,S.A., Mouchard,L., Reinert,K., Remington,K.A.,\n",
      "            Clark,A.G., Waterman,M.S., Eichler,E.E., Adams,M.D.,\n",
      "            Hunkapiller,M.W., Myers,E.W. and Venter,J.C.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (??-SEP-2005)\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [LARGE SCALE GENOMIC DNA].\n",
      "REFERENCE   5  (residues 1 to 270)\n",
      "  AUTHORS   Gerhard,D.S., Wagner,L., Feingold,E.A., Shenmen,C.M., Grouse,L.H.,\n",
      "            Schuler,G., Klein,S.L., Old,S., Rasooly,R., Good,P., Guyer,M.,\n",
      "            Peck,A.M., Derge,J.G., Lipman,D., Collins,F.S., Jang,W., Sherry,S.,\n",
      "            Feolo,M., Misquitta,L., Lee,E., Rotmistrovsky,K., Greenhut,S.F.,\n",
      "            Schaefer,C.F., Buetow,K., Bonner,T.I., Haussler,D., Kent,J.,\n",
      "            Kiekhaus,M., Furey,T., Brent,M., Prange,C., Schreiber,K.,\n",
      "            Shapiro,N., Bhat,N.K., Hopkins,R.F., Hsie,F., Driscoll,T.,\n",
      "            Soares,M.B., Casavant,T.L., Scheetz,T.E., Brown-stein,M.J.,\n",
      "            Usdin,T.B., Toshiyuki,S., Carninci,P., Piao,Y., Dudekula,D.B.,\n",
      "            Ko,M.S., Kawakami,K., Suzuki,Y., Sugano,S., Gruber,C.E.,\n",
      "            Smith,M.R., Simmons,B., Moore,T., Waterman,R., Johnson,S.L.,\n",
      "            Ruan,Y., Wei,C.L., Mathavan,S., Gunaratne,P.H., Wu,J., Garcia,A.M.,\n",
      "            Hulyk,S.W., Fuh,E., Yuan,Y., Sneed,A., Kowis,C., Hodgson,A.,\n",
      "            Muzny,D.M., McPherson,J., Gibbs,R.A., Fahey,J., Helton,E.,\n",
      "            Ketteman,M., Madan,A., Rodrigues,S., Sanchez,A., Whiting,M.,\n",
      "            Madari,A., Young,A.C., Wetherby,K.D., Granite,S.J., Kwong,P.N.,\n",
      "            Brinkley,C.P., Pearson,R.L., Bouffard,G.G., Blakesly,R.W.,\n",
      "            Green,E.D., Dickson,M.C., Rodriguez,A.C., Grimwood,J., Schmutz,J.,\n",
      "            Myers,R.M., Butterfield,Y.S., Griffith,M., Griffith,O.L.,\n",
      "            Krzywinski,M.I., Liao,N., Morin,R., Palmquist,D., Petrescu,A.S.,\n",
      "            Skalska,U., Smailus,D.E., Stott,J.M., Schnerch,A., Schein,J.E.,\n",
      "            Jones,S.J., Holt,R.A., Baross,A., Marra,M.A., Clifton,S.,\n",
      "            Makowski,K.A., Bosak,S. and Malek,J.\n",
      "  CONSRTM   MGC Project Team\n",
      "  TITLE     The status, quality, and expansion of the NIH full-length cDNA\n",
      "            project: the Mammalian Gene Collection (MGC)\n",
      "  JOURNAL   Genome Res. 14 (10B), 2121-2127 (2004)\n",
      "   PUBMED   15489334\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [LARGE SCALE MRNA].;\n",
      "            TISSUE=B-cell\n",
      "            Erratum:[Genome Res. 2006 Jun;16(6):804. Morrin, Ryan [corrected to\n",
      "            Morin, Ryan]]\n",
      "REFERENCE   6  (residues 1 to 270)\n",
      "  AUTHORS   Jen,K.Y. and Cheung,V.G.\n",
      "  TITLE     Identification of novel p53 target genes in ionizing radiation\n",
      "            response\n",
      "  JOURNAL   Cancer Res. 65 (17), 7666-7673 (2005)\n",
      "   PUBMED   16140933\n",
      "  REMARK    INDUCTION.\n",
      "REFERENCE   7  (residues 1 to 270)\n",
      "  AUTHORS   Bensaad,K., Tsuruta,A., Selak,M.A., Vidal,M.N., Nakano,K.,\n",
      "            Bartrons,R., Gottlieb,E. and Vousden,K.H.\n",
      "  TITLE     TIGAR, a p53-inducible regulator of glycolysis and apoptosis\n",
      "  JOURNAL   Cell 126 (1), 107-120 (2006)\n",
      "   PUBMED   16839880\n",
      "  REMARK    FUNCTION, MUTAGENESIS OF HIS-11; GLU-102 AND HIS-198, AND\n",
      "            INDUCTION.\n",
      "REFERENCE   8  (residues 1 to 270)\n",
      "  AUTHORS   Matsuoka,S., Ballif,B.A., Smogorzewska,A., McDonald,E.R. 3rd,\n",
      "            Hurov,K.E., Luo,J., Bakalarski,C.E., Zhao,Z., Solimini,N.,\n",
      "            Lerenthal,Y., Shiloh,Y., Gygi,S.P. and Elledge,S.J.\n",
      "  TITLE     ATM and ATR substrate analysis reveals extensive protein networks\n",
      "            responsive to DNA damage\n",
      "  JOURNAL   Science 316 (5828), 1160-1166 (2007)\n",
      "   PUBMED   17525332\n",
      "  REMARK    IDENTIFICATION BY MASS SPECTROMETRY [LARGE SCALE ANALYSIS].;\n",
      "            TISSUE=Embryonic kidney\n",
      "REFERENCE   9  (residues 1 to 270)\n",
      "  AUTHORS   Bensaad,K., Cheung,E.C. and Vousden,K.H.\n",
      "  TITLE     Modulation of intracellular ROS levels by TIGAR controls autophagy\n",
      "  JOURNAL   EMBO J. 28 (19), 3015-3026 (2009)\n",
      "   PUBMED   19713938\n",
      "  REMARK    FUNCTION, AND INDUCTION.\n",
      "REFERENCE   10 (residues 1 to 270)\n",
      "  AUTHORS   Li,H. and Jogl,G.\n",
      "  TITLE     Structural and biochemical studies of TIGAR (TP53-induced\n",
      "            glycolysis and apoptosis regulator)\n",
      "  JOURNAL   J. Biol. Chem. 284 (3), 1748-1754 (2009)\n",
      "   PUBMED   19015259\n",
      "  REMARK    CATALYTIC ACTIVITY.\n",
      "REFERENCE   11 (residues 1 to 270)\n",
      "  AUTHORS   Choudhary,C., Kumar,C., Gnad,F., Nielsen,M.L., Rehman,M.,\n",
      "            Walther,T.C., Olsen,J.V. and Mann,M.\n",
      "  TITLE     Lysine acetylation targets protein complexes and co-regulates major\n",
      "            cellular functions\n",
      "  JOURNAL   Science 325 (5942), 834-840 (2009)\n",
      "   PUBMED   19608861\n",
      "  REMARK    ACETYLATION [LARGE SCALE ANALYSIS] AT LYS-50, AND IDENTIFICATION BY\n",
      "            MASS SPECTROMETRY [LARGE SCALE ANALYSIS].\n",
      "REFERENCE   12 (residues 1 to 270)\n",
      "  AUTHORS   Burkard,T.R., Planyavsky,M., Kaupe,I., Breitwieser,F.P.,\n",
      "            Burckstummer,T., Bennett,K.L., Superti-Furga,G. and Colinge,J.\n",
      "  TITLE     Initial characterization of the human central proteome\n",
      "  JOURNAL   BMC Syst Biol 5, 17 (2011)\n",
      "   PUBMED   21269460\n",
      "  REMARK    IDENTIFICATION BY MASS SPECTROMETRY [LARGE SCALE ANALYSIS].\n",
      "            Publication Status: Online-Only\n",
      "REFERENCE   13 (residues 1 to 270)\n",
      "  AUTHORS   Won,K.Y., Lim,S.J., Kim,G.Y., Kim,Y.W., Han,S.A., Song,J.Y. and\n",
      "            Lee,D.K.\n",
      "  TITLE     Regulatory role of p53 in cancer metabolism via SCO2 and TIGAR in\n",
      "            human breast cancer\n",
      "  JOURNAL   Hum. Pathol. 43 (2), 221-228 (2012)\n",
      "   PUBMED   21820150\n",
      "  REMARK    TISSUE SPECIFICITY.\n",
      "REFERENCE   14 (residues 1 to 270)\n",
      "  AUTHORS   Wanka,C., Steinbach,J.P. and Rieger,J.\n",
      "  TITLE     Tp53-induced glycolysis and apoptosis regulator (TIGAR) protects\n",
      "            glioma cells from starvation-induced cell death by up-regulating\n",
      "            respiration and improving cellular redox homeostasis\n",
      "  JOURNAL   J. Biol. Chem. 287 (40), 33436-33446 (2012)\n",
      "   PUBMED   22887998\n",
      "  REMARK    FUNCTION, INDUCTION, TISSUE SPECIFICITY, AND MUTAGENESIS OF HIS-11;\n",
      "            GLU-102 AND HIS-198.\n",
      "REFERENCE   15 (residues 1 to 270)\n",
      "  AUTHORS   Cheung,E.C., Ludwig,R.L. and Vousden,K.H.\n",
      "  TITLE     Mitochondrial localization of TIGAR under hypoxia stimulates HK2\n",
      "            and lowers ROS and cell death\n",
      "  JOURNAL   Proc. Natl. Acad. Sci. U.S.A. 109 (50), 20491-20496 (2012)\n",
      "   PUBMED   23185017\n",
      "  REMARK    FUNCTION, INTERACTION WITH HK2, SUBCELLULAR LOCATION, AND\n",
      "            MUTAGENESIS OF HIS-11; GLU-102; HIS-198 AND 258-ASN--ASP-261.\n",
      "REFERENCE   16 (residues 1 to 270)\n",
      "  AUTHORS   Ye,L., Zhao,X., Lu,J., Qian,G., Zheng,J.C. and Ge,S.\n",
      "  TITLE     Knockdown of TIGAR by RNA interference induces apoptosis and\n",
      "            autophagy in HepG2 hepatocellular carcinoma cells\n",
      "  JOURNAL   Biochem. Biophys. Res. Commun. 437 (2), 300-306 (2013)\n",
      "   PUBMED   23817040\n",
      "  REMARK    FUNCTION.\n",
      "REFERENCE   17 (residues 1 to 270)\n",
      "  AUTHORS   Cheung,E.C., Athineos,D., Lee,P., Ridgway,R.A., Lambie,W.,\n",
      "            Nixon,C., Strathdee,D., Blyth,K., Sansom,O.J. and Vousden,K.H.\n",
      "  TITLE     TIGAR is required for efficient intestinal regeneration and\n",
      "            tumorigenesis\n",
      "  JOURNAL   Dev. Cell 25 (5), 463-477 (2013)\n",
      "   PUBMED   23726973\n",
      "  REMARK    FUNCTION, AND MUTAGENESIS OF HIS-11; GLU-102 AND HIS-198.\n",
      "REFERENCE   18 (residues 1 to 270)\n",
      "  AUTHORS   Yu,H.P., Xie,J.M., Li,B., Sun,Y.H., Gao,Q.G., Ding,Z.H., Wu,H.R.\n",
      "            and Qin,Z.H.\n",
      "  TITLE     TIGAR regulates DNA damage and repair through pentosephosphate\n",
      "            pathway and Cdk5-ATM pathway\n",
      "  JOURNAL   Sci Rep 5, 9853 (2015)\n",
      "   PUBMED   25928429\n",
      "  REMARK    FUNCTION, AND SUBCELLULAR LOCATION.\n",
      "            Publication Status: Online-Only\n",
      "REFERENCE   19 (residues 1 to 270)\n",
      "  CONSRTM   Center for eukaryotic structural genomics (CESG)\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (??-FEB-2009) to the PDB data bank\n",
      "  REMARK    X-RAY CRYSTALLOGRAPHY (1.75 ANGSTROMS) OF 2-270.\n",
      "COMMENT     [FUNCTION] Fructose-bisphosphatase hydrolyzing\n",
      "            fructose-2,6-bisphosphate as well as fructose-1,6-bisphosphate\n",
      "            (PubMed:19015259). Acts as a negative regulator of glycolysis by\n",
      "            lowering intracellular levels of fructose-2,6-bisphosphate in a\n",
      "            p53/TP53-dependent manner, resulting in the pentose phosphate\n",
      "            pathway (PPP) activation and NADPH production (PubMed:16839880,\n",
      "            PubMed:22887998). Contributes to the generation of reduced\n",
      "            glutathione to cause a decrease in intracellular reactive oxygen\n",
      "            species (ROS) content, correlating with its ability to protect\n",
      "            cells from oxidative or metabolic stress-induced cell death\n",
      "            (PubMed:16839880, PubMed:19713938, PubMed:23726973,\n",
      "            PubMed:22887998, PubMed:23817040). Plays a role in promoting\n",
      "            protection against cell death during hypoxia by decreasing\n",
      "            mitochondria ROS levels in a HK2-dependent manner through a\n",
      "            mechanism that is independent of its fructose-bisphosphatase\n",
      "            activity (PubMed:23185017). In response to cardiac damage stress,\n",
      "            mediates p53-induced inhibition of myocyte mitophagy through ROS\n",
      "            levels reduction and the subsequent inactivation of BNIP3. Reduced\n",
      "            mitophagy results in an enhanced apoptotic myocyte cell death, and\n",
      "            exacerbates cardiac damage (By similarity). Plays a role in adult\n",
      "            intestinal regeneration; contributes to the growth, proliferation\n",
      "            and survival of intestinal crypts following tissue ablation\n",
      "            (PubMed:23726973). Plays a neuroprotective role against ischemic\n",
      "            brain damage by enhancing PPP flux and preserving mitochondria\n",
      "            functions (By similarity). Protects glioma cells from hypoxia- and\n",
      "            ROS-induced cell death by inhibiting glycolysis and activating\n",
      "            mitochondrial energy metabolism and oxygen consumption in a\n",
      "            TKTL1-dependent and p53/TP53-independent manner (PubMed:22887998).\n",
      "            Plays a role in cancer cell survival by promoting DNA repair\n",
      "            through activating PPP flux in a CDK5-ATM-dependent signaling\n",
      "            pathway during hypoxia and/or genome stress-induced DNA damage\n",
      "            responses (PubMed:25928429). Involved in intestinal tumor\n",
      "            progression (PubMed:23726973). {ECO:0000250|UniProtKB:Q8BZA9,\n",
      "            ECO:0000269|PubMed:16839880, ECO:0000269|PubMed:19015259,\n",
      "            ECO:0000269|PubMed:19713938, ECO:0000269|PubMed:22887998,\n",
      "            ECO:0000269|PubMed:23185017, ECO:0000269|PubMed:23726973,\n",
      "            ECO:0000269|PubMed:23817040, ECO:0000269|PubMed:25928429}.\n",
      "            [CATALYTIC ACTIVITY] Beta-D-fructose 2,6-bisphosphate + H(2)O =\n",
      "            D-fructose 6-phosphate + phosphate. {ECO:0000269|PubMed:19015259}.\n",
      "            [SUBUNIT] Interacts with HK2; the interaction increases hexokinase\n",
      "            HK2 activity in a hypoxia- and HIF1A-dependent manner, resulting in\n",
      "            the regulation of mitochondrial membrane potential, thus increasing\n",
      "            NADPH production and decreasing intracellular ROS levels\n",
      "            (PubMed:23185017). {ECO:0000269|PubMed:23185017}.\n",
      "            [INTERACTION] P52789:HK2; NbExp=3; IntAct=EBI-3920747, EBI-741469.\n",
      "            [SUBCELLULAR LOCATION] Cytoplasm {ECO:0000269|PubMed:23185017,\n",
      "            ECO:0000269|PubMed:25928429}. Nucleus\n",
      "            {ECO:0000269|PubMed:25928429}. Mitochondrion\n",
      "            {ECO:0000269|PubMed:23185017}. Note=Translocated to the\n",
      "            mitochondria during hypoxia in a HIF1A-dependent manner\n",
      "            (PubMed:23185017). Colocalizes with HK2 in the mitochondria during\n",
      "            hypoxia (PubMed:23185017). Translocated to the nucleus during\n",
      "            hypoxia and/or genome stress-induced DNA damage responses in cancer\n",
      "            cells (PubMed:25928429). Translocation to the mitochondria is\n",
      "            enhanced in ischemic cortex after reperfusion and/or during oxygen\n",
      "            and glucose deprivation (OGD)/reoxygenation insult in primary\n",
      "            neurons (By similarity). {ECO:0000250|UniProtKB:Q8BZA9,\n",
      "            ECO:0000269|PubMed:23185017, ECO:0000269|PubMed:25928429}.\n",
      "            [TISSUE SPECIFICITY] Expressed in the brain (PubMed:22887998).\n",
      "            Expressed in breast tumors (PubMed:21820150). Expressed in\n",
      "            glioblastomas (PubMed:22887998). {ECO:0000269|PubMed:21820150,\n",
      "            ECO:0000269|PubMed:22887998}.\n",
      "            [INDUCTION] Up-regulated by p53/TP53 (at protein level)\n",
      "            (PubMed:16839880). Rapidly up-regulated by p53/TP53\n",
      "            (PubMed:16140933, PubMed:16839880, PubMed:19713938). Up-regulated\n",
      "            in glioma cell line in a p53/TP53-independent manner\n",
      "            (PubMed:22887998). {ECO:0000269|PubMed:16140933,\n",
      "            ECO:0000269|PubMed:16839880, ECO:0000269|PubMed:19713938,\n",
      "            ECO:0000269|PubMed:22887998}.\n",
      "            [SIMILARITY] Belongs to the phosphoglycerate mutase family.\n",
      "            {ECO:0000305}.\n",
      "            [CAUTION] Not expected to have any kinase activity. {ECO:0000305}.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..270\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "     gene            1..270\n",
      "                     /gene=\"TIGAR\"\n",
      "                     /gene_synonym=\"C12orf5\"\n",
      "     Protein         1..270\n",
      "                     /product=\"Fructose-2,6-bisphosphatase TIGAR\"\n",
      "                     /EC_number=\"3.1.3.46\"\n",
      "                     /note=\"TP53-induced glycolysis and apoptosis regulator;\n",
      "                     TP53-induced glycolysis regulatory phosphatase\"\n",
      "                     /UniProtKB_evidence=\"Evidence at protein level\"\n",
      "     Region          1..270\n",
      "                     /region_name=\"Mature chain\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Fructose-2,6-bisphosphatase TIGAR.\n",
      "                     /FTId=PRO_0000179957.\"\n",
      "     Region          2..10\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3DCY}.\"\n",
      "     Region          6..210\n",
      "                     /region_name=\"His_Phos_1\"\n",
      "                     /note=\"Histidine phosphatase superfamily (branch 1);\n",
      "                     pfam00300\"\n",
      "                     /db_xref=\"CDD:306748\"\n",
      "     Site            11\n",
      "                     /site_type=\"mutagenized\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"H->A: Abolishes the ability to lower cellular\n",
      "                     fructose-2,6-bisphosphate levels, to inhibit the\n",
      "                     glycolytic activity, to reduce levels of ROS, to increase\n",
      "                     oxygen consumption and to protect toward hypoxic cell\n",
      "                     death; when associated with A-11 and A-102. Retains the\n",
      "                     ability to interact and enhance HK2 activity, to localize\n",
      "                     to the mitochondria, to limit mitochondrial ROS level\n",
      "                     increase during hypoxia and to rescued partially crypt\n",
      "                     growth; when associated with A-102 and A-198. Loss of the\n",
      "                     ability to protect against cell death during hypoxia; when\n",
      "                     associated with A-102; A-198 and 258-N--D-261 Del.\n",
      "                     {ECO:0000269|PubMed:16839880, ECO:0000269|PubMed:22887998,\n",
      "                     ECO:0000269|PubMed:23185017,\n",
      "                     ECO:0000269|PubMed:23726973}.\"\n",
      "     Site            11\n",
      "                     /site_type=\"active\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Tele-phosphohistidine intermediate.\n",
      "                     {ECO:0000250|UniProtKB:Q7ZVE3}.\"\n",
      "     Region          15..19\n",
      "                     /region_name=\"Helical region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3DCY}.\"\n",
      "     Region          24..27\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3DCY}.\"\n",
      "     Region          33..45\n",
      "                     /region_name=\"Helical region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3DCY}.\"\n",
      "     Region          46..48\n",
      "                     /region_name=\"Hydrogen bonded turn\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3DCY}.\"\n",
      "     Site            50\n",
      "                     /site_type=\"modified\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"N6-acetyllysine. {ECO:0000244|PubMed:19608861}.\"\n",
      "     Region          52..56\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3DCY}.\"\n",
      "     Region          60..70\n",
      "                     /region_name=\"Helical region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3DCY}.\"\n",
      "     Region          81..83\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3DCY}.\"\n",
      "     Region          85..87\n",
      "                     /region_name=\"Helical region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3DCY}.\"\n",
      "     Site            89\n",
      "                     /site_type=\"active\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Proton donor/acceptor.\n",
      "                     {ECO:0000250|UniProtKB:Q7ZVE3}.\"\n",
      "     Region          93..95\n",
      "                     /region_name=\"Helical region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3DCY}.\"\n",
      "     Region          100..109\n",
      "                     /region_name=\"Helical region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3DCY}.\"\n",
      "     Site            102\n",
      "                     /site_type=\"mutagenized\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"E->A: Abolishes the ability to lower cellular\n",
      "                     fructose-2,6-bisphosphate levels, to inhibit the\n",
      "                     glycolytic activity, to reduce levels of ROS, to increase\n",
      "                     oxygen consumption and to protect toward hypoxic cell\n",
      "                     death; when associated with A-11 and A-198. Retains the\n",
      "                     ability to interact and enhance HK2 activity, to localize\n",
      "                     to the mitochondria, to limit mitochondrial ROS level\n",
      "                     increase during hypoxia and to rescued partially crypt\n",
      "                     growth; when associated with A-11 and A-198. Loss of the\n",
      "                     ability to protect against cell death during hypoxia; when\n",
      "                     associated with A-11; A-198 and 258-N--D-261 Del.\n",
      "                     {ECO:0000269|PubMed:16839880, ECO:0000269|PubMed:22887998,\n",
      "                     ECO:0000269|PubMed:23185017,\n",
      "                     ECO:0000269|PubMed:23726973}.\"\n",
      "     Region          114..116\n",
      "                     /region_name=\"Hydrogen bonded turn\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3DCY}.\"\n",
      "     Region          125..149\n",
      "                     /region_name=\"Helical region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3DCY}.\"\n",
      "     Region          161..167\n",
      "                     /region_name=\"Helical region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3DCY}.\"\n",
      "     Region          192..197\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3DCY}.\"\n",
      "     Site            198\n",
      "                     /site_type=\"mutagenized\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"H->A: Abolishes the ability to lower cellular\n",
      "                     fructose-2,6-bisphosphate levels, to inhibit the\n",
      "                     glycolytic activity, to reduce levels of ROS, to increase\n",
      "                     oxygen consumption and to protect toward hypoxic cell\n",
      "                     death; when associated with A-11 and A-102. Retains the\n",
      "                     ability to interact and enhance HK2 activity, to localize\n",
      "                     to the mitochondria, to limit mitochondrial ROS level\n",
      "                     increase during hypoxia and to rescued partially crypt\n",
      "                     growth; when associated with A-11 and A-102. Loss of the\n",
      "                     ability to protect against cell death during hypoxia; when\n",
      "                     associated with A-11; A-102 and 258-N--D-261 Del.\n",
      "                     {ECO:0000269|PubMed:16839880, ECO:0000269|PubMed:22887998,\n",
      "                     ECO:0000269|PubMed:23185017,\n",
      "                     ECO:0000269|PubMed:23726973}.\"\n",
      "     Site            198\n",
      "                     /site_type=\"other\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Transition state stabilizer.\n",
      "                     {ECO:0000250|UniProtKB:Q7ZVE3}.\"\n",
      "     Region          199..211\n",
      "                     /region_name=\"Helical region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3DCY}.\"\n",
      "     Region          223..227\n",
      "                     /region_name=\"Helical region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3DCY}.\"\n",
      "     Region          235..242\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3DCY}.\"\n",
      "     Region          244..247\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3DCY}.\"\n",
      "     Region          251..259\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3DCY}.\"\n",
      "     Site            258..261\n",
      "                     /site_type=\"mutagenized\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Missing: Inhibits the ability to interact and\n",
      "                     enhance HK2 activity, to localize to the mitochondria, to\n",
      "                     protect against the decrease of mitochondrial membrane\n",
      "                     potential and to limit mitochondrial ROS level increase\n",
      "                     during hypoxia. Does not abolish the ability to lower\n",
      "                     cellular fructose-2,6-bisphosphate levels during hypoxia.\n",
      "                     Loss of the ability to protect against cell death during\n",
      "                     hypoxia; when associated with A-11; A-102 and A-198.\n",
      "                     {ECO:0000269|PubMed:23185017}.\"\n",
      "ORIGIN      \n",
      "        1 marfaltvvr hgetrfnkek iiqgqgvdep lsetgfkqaa aagiflnnvk fthafssdlm\n",
      "       61 rtkqtmhgil erskfckdmt vkydsrlrer kygvvegkal selramakaa reecpvftpp\n",
      "      121 ggetldqvkm rgidffeflc qlilkeadqk eqfsqgspsn cletslaeif plgknhsskv\n",
      "      181 nsdsgipgla asvlvvshga ymrslfdyfl tdlkcslpat lsrselmsvt pntgmslfii\n",
      "      241 nfeegrevkp tvqcicmnlq dhlngltetr\n",
      "//\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LOCUS       TIGAR_BOVIN              270 aa            linear   MAM 12-SEP-2018\n",
      "DEFINITION  RecName: Full=Fructose-2,6-bisphosphatase TIGAR; AltName:\n",
      "            Full=TP53-induced glycolysis and apoptosis regulator; AltName:\n",
      "            Full=TP53-induced glycolysis regulatory phosphatase.\n",
      "ACCESSION   Q1JQA7\n",
      "VERSION     Q1JQA7.1\n",
      "DBSOURCE    UniProtKB: locus TIGAR_BOVIN, accession Q1JQA7;\n",
      "            class: standard.\n",
      "            created: Feb 10, 2009.\n",
      "            sequence updated: Jun 13, 2006.\n",
      "            annotation updated: Sep 12, 2018.\n",
      "            xrefs: BC116101.1, AAI16102.1, NP_001069838.1\n",
      "            xrefs (non-sequence databases): UniGene:Bt.61374,\n",
      "            ProteinModelPortal:Q1JQA7, SMR:Q1JQA7,\n",
      "            STRING:9913.ENSBTAP00000022146, PaxDb:Q1JQA7,\n",
      "            Ensembl:ENSBTAT00000022146, Ensembl:ENSBTAP00000022146,\n",
      "            Ensembl:ENSBTAG00000016650, GeneID:615392, KEGG:bta:615392,\n",
      "            CTD:57103, VGNC:35863, eggNOG:ENOG410IUDB, eggNOG:COG0406,\n",
      "            GeneTree:ENSGT00390000013224, HOGENOM:HOG000060277,\n",
      "            HOVERGEN:HBG108569, InParanoid:Q1JQA7, KO:K14634, OMA:NFEEGRE,\n",
      "            OrthoDB:EOG091G0W1L, TreeFam:TF329053, Reactome:R-BTA-5628897,\n",
      "            Proteomes:UP000009136, Bgee:ENSBTAG00000016650, GO:0005737,\n",
      "            GO:0005829, GO:0005741, GO:0005739, GO:0005634, GO:0004083,\n",
      "            GO:0004331, GO:0016791, GO:0006915, GO:0006914, GO:0071279,\n",
      "            GO:0006974, GO:0071456, GO:0030388, GO:0006003, GO:0060576,\n",
      "            GO:1904024, GO:0045820, GO:1901525, GO:1901215, GO:0043069,\n",
      "            GO:2000378, GO:0010666, GO:0045739, GO:1903301, GO:0043456,\n",
      "            GO:1902153, GO:0010332, GO:0002931, GO:0009410, CDD:cd07067,\n",
      "            Gene3D:3.40.50.1240, InterPro:IPR013078, InterPro:IPR029033,\n",
      "            InterPro:IPR001345, Pfam:PF00300, SMART:SM00855, SUPFAM:SSF53254,\n",
      "            PROSITE:PS00175\n",
      "KEYWORDS    Acetylation; Apoptosis; Autophagy; Complete proteome; Cytoplasm;\n",
      "            Hydrolase; Mitochondrion; Nucleus; Reference proteome.\n",
      "SOURCE      Bos taurus (cattle)\n",
      "  ORGANISM  Bos taurus\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Laurasiatheria; Cetartiodactyla; Ruminantia;\n",
      "            Pecora; Bovidae; Bovinae; Bos.\n",
      "REFERENCE   1  (residues 1 to 270)\n",
      "  CONSRTM   NIH - Mammalian Gene Collection (MGC) project\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (??-MAY-2006)\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [LARGE SCALE MRNA].;\n",
      "            STRAIN=Hereford; TISSUE=Ascending colon\n",
      "COMMENT     [FUNCTION] Fructose-bisphosphatase hydrolyzing\n",
      "            fructose-2,6-bisphosphate as well as fructose-1,6-bisphosphate (By\n",
      "            similarity). Acts as a negative regulator of glycolysis by lowering\n",
      "            intracellular levels of fructose-2,6-bisphosphate in a\n",
      "            p53/TP53-dependent manner, resulting in the pentose phosphate\n",
      "            pathway (PPP) activation and NADPH production. Contributes to the\n",
      "            generation of reduced glutathione to cause a decrease in\n",
      "            intracellular reactive oxygen species (ROS) content, correlating\n",
      "            with its ability to protect cells from oxidative or metabolic\n",
      "            stress-induced cell death. Plays a role in promoting protection\n",
      "            against cell death during hypoxia by decreasing mitochondria ROS\n",
      "            levels in a HK2-dependent manner through a mechanism that is\n",
      "            independent of its fructose-bisphosphatase activity. In response to\n",
      "            cardiac damage stress, mediates p53-induced inhibition of myocyte\n",
      "            mitophagy through ROS levels reduction and the subsequent\n",
      "            inactivation of BNIP3. Reduced mitophagy results in an enhanced\n",
      "            apoptotic myocyte cell death, and exacerbates cardiac damage. Plays\n",
      "            a role in adult intestinal regeneration; contributes to the growth,\n",
      "            proliferation and survival of intestinal crypts following tissue\n",
      "            ablation. Plays a neuroprotective role against ischemic brain\n",
      "            damage by enhancing PPP flux and preserving mitochondria functions.\n",
      "            Protects glioma cells from hypoxia- and ROS-induced cell death by\n",
      "            inhibiting glycolysis and activating mitochondrial energy\n",
      "            metabolism and oxygen consumption in a TKTL1-dependent and\n",
      "            p53/TP53-independent manner. Plays a role in cancer cell survival\n",
      "            by promoting DNA repair through activating PPP flux in a\n",
      "            CDK5-ATM-dependent signaling pathway during hypoxia and/or genome\n",
      "            stress-induced DNA damage responses. Involved in intestinal tumor\n",
      "            progression. {ECO:0000250|UniProtKB:Q8BZA9,\n",
      "            ECO:0000250|UniProtKB:Q9NQ88}.\n",
      "            [CATALYTIC ACTIVITY] Beta-D-fructose 2,6-bisphosphate + H(2)O =\n",
      "            D-fructose 6-phosphate + phosphate. {ECO:0000250|UniProtKB:Q9NQ88}.\n",
      "            [SUBUNIT] Interacts with HK2; the interaction increases hexokinase\n",
      "            HK2 activity in a hypoxia- and HIF1A-dependent manner, resulting in\n",
      "            the regulation of mitochondrial membrane potential, thus increasing\n",
      "            NADPH production and decreasing intracellular ROS levels.\n",
      "            {ECO:0000250|UniProtKB:Q9NQ88}.\n",
      "            [SUBCELLULAR LOCATION] Cytoplasm {ECO:0000250|UniProtKB:Q8BZA9}.\n",
      "            Nucleus {ECO:0000250|UniProtKB:Q9NQ88}. Mitochondrion\n",
      "            {ECO:0000250|UniProtKB:Q8BZA9}. Note=Translocated to the\n",
      "            mitochondria during hypoxia in a HIF1A-dependent manner.\n",
      "            Colocalizes with HK2 in the mitochondria during hypoxia.\n",
      "            Translocated to the nucleus during hypoxia and/or genome\n",
      "            stress-induced DNA damage responses in cancer cells. Translocation\n",
      "            to the mitochondria is enhanced in ischemic cortex after\n",
      "            reperfusion and/or during oxygen and glucose deprivation\n",
      "            (OGD)/reoxygenation insult in primary neurons.\n",
      "            {ECO:0000250|UniProtKB:Q8BZA9, ECO:0000250|UniProtKB:Q9NQ88}.\n",
      "            [SIMILARITY] Belongs to the phosphoglycerate mutase family.\n",
      "            {ECO:0000305}.\n",
      "            [CAUTION] Not expected to have any kinase activity. {ECO:0000305}.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..270\n",
      "                     /organism=\"Bos taurus\"\n",
      "                     /db_xref=\"taxon:9913\"\n",
      "     gene            1..270\n",
      "                     /gene=\"TIGAR\"\n",
      "     Protein         1..270\n",
      "                     /product=\"Fructose-2,6-bisphosphatase TIGAR\"\n",
      "                     /EC_number=\"3.1.3.46\"\n",
      "                     /note=\"TP53-induced glycolysis and apoptosis regulator;\n",
      "                     TP53-induced glycolysis regulatory phosphatase\"\n",
      "                     /UniProtKB_evidence=\"Evidence at transcript level\"\n",
      "     Region          1..270\n",
      "                     /region_name=\"Mature chain\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Fructose-2,6-bisphosphatase TIGAR.\n",
      "                     /FTId=PRO_0000363066.\"\n",
      "     Region          6..210\n",
      "                     /region_name=\"His_Phos_1\"\n",
      "                     /note=\"Histidine phosphatase superfamily (branch 1);\n",
      "                     pfam00300\"\n",
      "                     /db_xref=\"CDD:278716\"\n",
      "     Site            order(10..11,61,198..199)\n",
      "                     /site_type=\"active\"\n",
      "                     /note=\"catalytic core [active]\"\n",
      "                     /db_xref=\"CDD:132718\"\n",
      "     Site            11\n",
      "                     /site_type=\"active\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Tele-phosphohistidine intermediate.\n",
      "                     {ECO:0000250|UniProtKB:Q7ZVE3}.\"\n",
      "     Site            50\n",
      "                     /site_type=\"modified\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"N6-acetyllysine. {ECO:0000250|UniProtKB:Q9NQ88}.\"\n",
      "     Site            89\n",
      "                     /site_type=\"active\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Proton donor/acceptor.\n",
      "                     {ECO:0000250|UniProtKB:Q7ZVE3}.\"\n",
      "     Site            198\n",
      "                     /site_type=\"other\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Transition state stabilizer.\n",
      "                     {ECO:0000250|UniProtKB:Q7ZVE3}.\"\n",
      "ORIGIN      \n",
      "        1 mtrfaltvvr hgetrlnkek iiqgqgidep lsetgfkqaa aagiflkdvk fthvfssdlt\n",
      "       61 rtkqtvhgil ekskfckdmt vkydsrlrer kygvaegrpl selramakaa geecpaftpp\n",
      "      121 ggetldqlkr rgkdffeflc qlilkeagqn eqfsqeapss clesslaeif plgkncastf\n",
      "      181 nsdsgtpgla asvlvvshga yirslldyfl tdlkcsfpat lsrseltsvs pntgmtvfil\n",
      "      241 nfekggkgrp taqcvcvnlq ghlagvnktp\n",
      "//\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LOCUS       TIGAR_MOUSE              269 aa            linear   ROD 12-SEP-2018\n",
      "DEFINITION  RecName: Full=Fructose-2,6-bisphosphatase TIGAR; AltName:\n",
      "            Full=TP53-induced glycolysis and apoptosis regulator; AltName:\n",
      "            Full=TP53-induced glycolysis regulatory phosphatase.\n",
      "ACCESSION   Q8BZA9\n",
      "VERSION     Q8BZA9.1\n",
      "DBSOURCE    UniProtKB: locus TIGAR_MOUSE, accession Q8BZA9;\n",
      "            class: standard.\n",
      "            created: Dec 6, 2005.\n",
      "            sequence updated: Mar 1, 2003.\n",
      "            annotation updated: Sep 12, 2018.\n",
      "            xrefs: AK036082.1, BAC29298.1, AK145896.1, BAE26732.1, AK163290.1,\n",
      "            BAE37278.1, NP_795977.1\n",
      "            xrefs (non-sequence databases): CCDS:CCDS20563.1,\n",
      "            UniGene:Mm.101836, ProteinModelPortal:Q8BZA9, SMR:Q8BZA9,\n",
      "            STRING:10090.ENSMUSP00000048643, iPTMnet:Q8BZA9,\n",
      "            PhosphoSitePlus:Q8BZA9, EPD:Q8BZA9, MaxQB:Q8BZA9, PaxDb:Q8BZA9,\n",
      "            PRIDE:Q8BZA9, DNASU:319801, Ensembl:ENSMUST00000039913,\n",
      "            Ensembl:ENSMUSP00000048643, Ensembl:ENSMUSG00000038028,\n",
      "            GeneID:319801, KEGG:mmu:319801, UCSC:uc009dvq.2, CTD:57103,\n",
      "            MGI:2442752, eggNOG:ENOG410IUDB, eggNOG:COG0406,\n",
      "            GeneTree:ENSGT00390000013224, HOGENOM:HOG000060277,\n",
      "            HOVERGEN:HBG108569, InParanoid:Q8BZA9, KO:K14634, OMA:NFEEGRE,\n",
      "            OrthoDB:EOG091G0W1L, PhylomeDB:Q8BZA9, TreeFam:TF329053,\n",
      "            Reactome:R-MMU-5628897, PRO:PR:Q8BZA9, Proteomes:UP000000589,\n",
      "            Bgee:ENSMUSG00000038028, CleanEx:MM_9630033F20RIK,\n",
      "            ExpressionAtlas:Q8BZA9, Genevisible:Q8BZA9, GO:0005737, GO:0005829,\n",
      "            GO:0005622, GO:0005741, GO:0005739, GO:0005634, GO:0004083,\n",
      "            GO:0004331, GO:0006915, GO:0006914, GO:0071279, GO:0006974,\n",
      "            GO:0071456, GO:0030388, GO:0006003, GO:0060576, GO:1904024,\n",
      "            GO:0045820, GO:0033673, GO:1901525, GO:1901215, GO:0043069,\n",
      "            GO:2000378, GO:0010666, GO:0045739, GO:1903301, GO:0043456,\n",
      "            GO:1902153, GO:0010332, GO:0002931, GO:0009410, CDD:cd07067,\n",
      "            Gene3D:3.40.50.1240, InterPro:IPR013078, InterPro:IPR029033,\n",
      "            InterPro:IPR001345, Pfam:PF00300, SMART:SM00855, SUPFAM:SSF53254,\n",
      "            PROSITE:PS00175\n",
      "KEYWORDS    Apoptosis; Autophagy; Complete proteome; Cytoplasm; Hydrolase;\n",
      "            Mitochondrion; Nucleus; Reference proteome.\n",
      "SOURCE      Mus musculus (house mouse)\n",
      "  ORGANISM  Mus musculus\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Glires; Rodentia; Myomorpha;\n",
      "            Muroidea; Muridae; Murinae; Mus; Mus.\n",
      "REFERENCE   1  (residues 1 to 269)\n",
      "  AUTHORS   Carninci,P., Kasukawa,T., Katayama,S., Gough,J., Frith,M.C.,\n",
      "            Maeda,N., Oyama,R., Ravasi,T., Lenhard,B., Wells,C., Kodzius,R.,\n",
      "            Shimokawa,K., Bajic,V.B., Brenner,S.E., Batalov,S., Forrest,A.R.,\n",
      "            Zavolan,M., Davis,M.J., Wilming,L.G., Aidinis,V., Allen,J.E.,\n",
      "            Ambesi-Impiombato,A., Apweiler,R., Aturaliya,R.N., Bailey,T.L.,\n",
      "            Bansal,M., Baxter,L., Beisel,K.W., Bersano,T., Bono,H., Chalk,A.M.,\n",
      "            Chiu,K.P., Choudhary,V., Christoffels,A., Clutterbuck,D.R.,\n",
      "            Crowe,M.L., Dalla,E., Dalrymple,B.P., de Bono,B., Della Gatta,G.,\n",
      "            di Bernardo,D., Down,T., Engstrom,P., Fagiolini,M., Faulkner,G.,\n",
      "            Fletcher,C.F., Fukushima,T., Furuno,M., Futaki,S., Gariboldi,M.,\n",
      "            Georgii-Hemming,P., Gingeras,T.R., Gojobori,T., Green,R.E.,\n",
      "            Gustincich,S., Harbers,M., Hayashi,Y., Hensch,T.K., Hirokawa,N.,\n",
      "            Hill,D., Huminiecki,L., Iacono,M., Ikeo,K., Iwama,A., Ishikawa,T.,\n",
      "            Jakt,M., Kanapin,A., Katoh,M., Kawasawa,Y., Kelso,J., Kitamura,H.,\n",
      "            Kitano,H., Kollias,G., Krishnan,S.P., Kruger,A., Kummerfeld,S.K.,\n",
      "            Kurochkin,I.V., Lareau,L.F., Lazarevic,D., Lipovich,L., Liu,J.,\n",
      "            Liuni,S., McWilliam,S., Madan Babu,M., Madera,M., Marchionni,L.,\n",
      "            Matsuda,H., Matsuzawa,S., Miki,H., Mignone,F., Miyake,S.,\n",
      "            Morris,K., Mottagui-Tabar,S., Mulder,N., Nakano,N., Nakauchi,H.,\n",
      "            Ng,P., Nilsson,R., Nishiguchi,S., Nishikawa,S., Nori,F., Ohara,O.,\n",
      "            Okazaki,Y., Orlando,V., Pang,K.C., Pavan,W.J., Pavesi,G.,\n",
      "            Pesole,G., Petrovsky,N., Piazza,S., Reed,J., Reid,J.F., Ring,B.Z.,\n",
      "            Ringwald,M., Rost,B., Ruan,Y., Salzberg,S.L., Sandelin,A.,\n",
      "            Schneider,C., Schonbach,C., Sekiguchi,K., Semple,C.A., Seno,S.,\n",
      "            Sessa,L., Sheng,Y., Shibata,Y., Shimada,H., Shimada,K., Silva,D.,\n",
      "            Sinclair,B., Sperling,S., Stupka,E., Sugiura,K., Sultana,R.,\n",
      "            Takenaka,Y., Taki,K., Tammoja,K., Tan,S.L., Tang,S., Taylor,M.S.,\n",
      "            Tegner,J., Teichmann,S.A., Ueda,H.R., van Nimwegen,E., Verardo,R.,\n",
      "            Wei,C.L., Yagi,K., Yamanishi,H., Zabarovsky,E., Zhu,S., Zimmer,A.,\n",
      "            Hide,W., Bult,C., Grimmond,S.M., Teasdale,R.D., Liu,E.T.,\n",
      "            Brusic,V., Quackenbush,J., Wahlestedt,C., Mattick,J.S., Hume,D.A.,\n",
      "            Kai,C., Sasaki,D., Tomaru,Y., Fukuda,S., Kanamori-Katayama,M.,\n",
      "            Suzuki,M., Aoki,J., Arakawa,T., Iida,J., Imamura,K., Itoh,M.,\n",
      "            Kato,T., Kawaji,H., Kawagashira,N., Kawashima,T., Kojima,M.,\n",
      "            Kondo,S., Konno,H., Nakano,K., Ninomiya,N., Nishio,T., Okada,M.,\n",
      "            Plessy,C., Shibata,K., Shiraki,T., Suzuki,S., Tagami,M., Waki,K.,\n",
      "            Watahiki,A., Okamura-Oho,Y., Suzuki,H., Kawai,J. and Hayashizaki,Y.\n",
      "  CONSRTM   FANTOM Consortium; RIKEN Genome Exploration Research Group and\n",
      "            Genome Science Group (Genome Network Project Core Group)\n",
      "  TITLE     The transcriptional landscape of the mammalian genome\n",
      "  JOURNAL   Science 309 (5740), 1559-1563 (2005)\n",
      "   PUBMED   16141072\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [LARGE SCALE MRNA].;\n",
      "            STRAIN=C57BL/6J; TISSUE=Cerebellum, Egg, and Placenta\n",
      "            Erratum:[Science. 2006 Mar 24;311(5768):1713]\n",
      "REFERENCE   2  (residues 1 to 269)\n",
      "  AUTHORS   Sweet,S.M., Bailey,C.M., Cunningham,D.L., Heath,J.K. and\n",
      "            Cooper,H.J.\n",
      "  TITLE     Large scale localization of protein phosphorylation by use of\n",
      "            electron capture dissociation mass spectrometry\n",
      "  JOURNAL   Mol. Cell Proteomics 8 (5), 904-912 (2009)\n",
      "   PUBMED   19131326\n",
      "  REMARK    IDENTIFICATION BY MASS SPECTROMETRY [LARGE SCALE ANALYSIS].;\n",
      "            TISSUE=Embryonic fibroblast\n",
      "REFERENCE   3  (residues 1 to 269)\n",
      "  AUTHORS   Kimata,M., Matoba,S., Iwai-Kanai,E., Nakamura,H., Hoshino,A.,\n",
      "            Nakaoka,M., Katamura,M., Okawa,Y., Mita,Y., Okigaki,M., Ikeda,K.,\n",
      "            Tatsumi,T. and Matsubara,H.\n",
      "  TITLE     p53 and TIGAR regulate cardiac myocyte energy homeostasis under\n",
      "            hypoxic stress\n",
      "  JOURNAL   Am. J. Physiol. 299, H1908-H1916 (2010)\n",
      "   PUBMED   20935145\n",
      "  REMARK    INDUCTION.\n",
      "REFERENCE   4  (residues 1 to 269)\n",
      "  AUTHORS   Huttlin,E.L., Jedrychowski,M.P., Elias,J.E., Goswami,T., Rad,R.,\n",
      "            Beausoleil,S.A., Villen,J., Haas,W., Sowa,M.E. and Gygi,S.P.\n",
      "  TITLE     A tissue-specific atlas of mouse protein phosphorylation and\n",
      "            expression\n",
      "  JOURNAL   Cell 143 (7), 1174-1189 (2010)\n",
      "   PUBMED   21183079\n",
      "  REMARK    IDENTIFICATION BY MASS SPECTROMETRY [LARGE SCALE ANALYSIS].;\n",
      "            TISSUE=Brain, Brown adipose tissue, Heart, Kidney, Liver, Lung,\n",
      "            Pancreas, Spleen, and Testis\n",
      "REFERENCE   5  (residues 1 to 269)\n",
      "  AUTHORS   Hoshino,A., Matoba,S., Iwai-Kanai,E., Nakamura,H., Kimata,M.,\n",
      "            Nakaoka,M., Katamura,M., Okawa,Y., Ariyoshi,M., Mita,Y., Ikeda,K.,\n",
      "            Ueyama,T., Okigaki,M. and Matsubara,H.\n",
      "  TITLE     p53-TIGAR axis attenuates mitophagy to exacerbate cardiac damage\n",
      "            after ischemia\n",
      "  JOURNAL   J. Mol. Cell. Cardiol. 52 (1), 175-184 (2012)\n",
      "   PUBMED   22044588\n",
      "  REMARK    FUNCTION, DISRUPTION PHENOTYPE, AND INDUCTION.\n",
      "REFERENCE   6  (residues 1 to 269)\n",
      "  AUTHORS   Cheung,E.C., Athineos,D., Lee,P., Ridgway,R.A., Lambie,W.,\n",
      "            Nixon,C., Strathdee,D., Blyth,K., Sansom,O.J. and Vousden,K.H.\n",
      "  TITLE     TIGAR is required for efficient intestinal regeneration and\n",
      "            tumorigenesis\n",
      "  JOURNAL   Dev. Cell 25 (5), 463-477 (2013)\n",
      "   PUBMED   23726973\n",
      "  REMARK    FUNCTION, DISRUPTION PHENOTYPE, TISSUE SPECIFICITY, AND INDUCTION.\n",
      "REFERENCE   7  (residues 1 to 269)\n",
      "  AUTHORS   Li,M., Sun,M., Cao,L., Gu,J.H., Ge,J., Chen,J., Han,R., Qin,Y.Y.,\n",
      "            Zhou,Z.P., Ding,Y. and Qin,Z.H.\n",
      "  TITLE     A TIGAR-regulated metabolic pathway is critical for protection of\n",
      "            brain ischemia\n",
      "  JOURNAL   J. Neurosci. 34 (22), 7458-7471 (2014)\n",
      "   PUBMED   24872551\n",
      "  REMARK    FUNCTION, INDUCTION, SUBCELLULAR LOCATION, AND TISSUE SPECIFICITY.\n",
      "COMMENT     [FUNCTION] Fructose-bisphosphatase hydrolyzing\n",
      "            fructose-2,6-bisphosphate as well as fructose-1,6-bisphosphate (By\n",
      "            similarity). Acts as a negative regulator of glycolysis by lowering\n",
      "            intracellular levels of fructose-2,6-bisphosphate in a\n",
      "            p53/TP53-dependent manner, resulting in the pentose phosphate\n",
      "            pathway (PPP) activation and NADPH production (PubMed:23726973).\n",
      "            Contributes to the generation of reduced glutathione to cause a\n",
      "            decrease in intracellular reactive oxygen species (ROS) content,\n",
      "            correlating with its ability to protect cells from oxidative or\n",
      "            metabolic stress-induced cell death (PubMed:23726973). Plays a role\n",
      "            in promoting protection against cell death during hypoxia by\n",
      "            decreasing mitochondria ROS levels in a HK2-dependent manner\n",
      "            through a mechanism that is independent of its\n",
      "            fructose-bisphosphatase activity (By similarity). In response to\n",
      "            cardiac damage stress, mediates p53-induced inhibition of myocyte\n",
      "            mitophagy through ROS levels reduction and the subsequent\n",
      "            inactivation of BNIP3 (PubMed:22044588). Reduced mitophagy results\n",
      "            in an enhanced apoptotic myocyte cell death, and exacerbates\n",
      "            cardiac damage (PubMed:22044588). Plays a role in adult intestinal\n",
      "            regeneration; contributes to the growth, proliferation and survival\n",
      "            of intestinal crypts following tissue ablation (PubMed:23726973).\n",
      "            Plays a neuroprotective role against ischemic brain damage by\n",
      "            enhancing PPP flux and preserving mitochondria functions\n",
      "            (PubMed:24872551). Protects glioma cells from hypoxia- and\n",
      "            ROS-induced cell death by inhibiting glycolysis and activating\n",
      "            mitochondrial energy metabolism and oxygen consumption in a\n",
      "            TKTL1-dependent and p53/TP53-independent manner. Plays a role in\n",
      "            cancer cell survival by promoting DNA repair through activating PPP\n",
      "            flux in a CDK5-ATM-dependent signaling pathway during hypoxia\n",
      "            and/or genome stress-induced DNA damage responses (By similarity).\n",
      "            Involved in intestinal tumor progression (PubMed:23726973).\n",
      "            {ECO:0000250|UniProtKB:Q9NQ88, ECO:0000269|PubMed:22044588,\n",
      "            ECO:0000269|PubMed:23726973, ECO:0000269|PubMed:24872551}.\n",
      "            [CATALYTIC ACTIVITY] Beta-D-fructose 2,6-bisphosphate + H(2)O =\n",
      "            D-fructose 6-phosphate + phosphate. {ECO:0000250|UniProtKB:Q9NQ88}.\n",
      "            [SUBUNIT] Interacts with HK2; the interaction increases hexokinase\n",
      "            HK2 activity in a hypoxia- and HIF1A-dependent manner, resulting in\n",
      "            the regulation of mitochondrial membrane potential, thus increasing\n",
      "            NADPH production and decreasing intracellular ROS levels.\n",
      "            {ECO:0000250|UniProtKB:Q9NQ88}.\n",
      "            [SUBCELLULAR LOCATION] Cytoplasm {ECO:0000269|PubMed:24872551}.\n",
      "            Nucleus {ECO:0000250|UniProtKB:Q9NQ88}. Mitochondrion\n",
      "            {ECO:0000269|PubMed:24872551}. Note=Translocated to the\n",
      "            mitochondria during hypoxia in a HIF1A-dependent manner.\n",
      "            Colocalizes with HK2 in the mitochondria during hypoxia.\n",
      "            Translocated to the nucleus during hypoxia and/or genome\n",
      "            stress-induced DNA damage responses in cancer cells (By\n",
      "            similarity). Translocation to the mitochondria is enhanced in\n",
      "            ischemic cortex after reperfusion and/or during oxygen and glucose\n",
      "            deprivation (OGD)/reoxygenation insult in primary neurons\n",
      "            (PubMed:24872551). {ECO:0000250|UniProtKB:Q9NQ88,\n",
      "            ECO:0000269|PubMed:24872551}.\n",
      "            [TISSUE SPECIFICITY] Expressed in olfactory bulb, cerebellum, and\n",
      "            cortex (PubMed:24872551). Expressed in neurons and astrocytes\n",
      "            (PubMed:24872551) (at protein level). Expressed in intestinal crypt\n",
      "            (PubMed:23726973). {ECO:0000269|PubMed:23726973,\n",
      "            ECO:0000269|PubMed:24872551}.\n",
      "            [INDUCTION] Up-regulated by hypoxia in cardiac myocytes in a\n",
      "            p53/TP53-dependent manner (PubMed:20935145). Up-regulated in\n",
      "            ischemic cortex after reperfusion in a p53/TP5-independent manner\n",
      "            (PubMed:24872551). Up-regulated in primary neurons by oxygen and\n",
      "            glucose deprivation (OGD)/reoxygenation insult in a\n",
      "            p53/TP5-independent manner (at protein level) (PubMed:24872551).\n",
      "            Up-regulated in ischemic myocardium in a p53/TP5-dependent manner\n",
      "            (PubMed:22044588). Up-regulated in small intestine after gamma\n",
      "            irradiation damage (PubMed:23726973). {ECO:0000269|PubMed:20935145,\n",
      "            ECO:0000269|PubMed:22044588, ECO:0000269|PubMed:23726973,\n",
      "            ECO:0000269|PubMed:24872551}.\n",
      "            [DISRUPTION PHENOTYPE] Mice are viable and fertile and show no\n",
      "            obvious developmental defects (PubMed:23726973). Following\n",
      "            intestine ablation by gamma irradiation, adult mice display a\n",
      "            reduction in the size and number of proliferating intestinal crypts\n",
      "            and an increase in cell death (PubMed:23726973). Mice display\n",
      "            reduced intestinal tumor progression compared to wild-type mice\n",
      "            (PubMed:23726973). In response to ischemic myocardium injury,\n",
      "            display an increase in the ability to stimulate myocyte mitophagy\n",
      "            in ischemic border zones through a ROS-induced and BNIP3 activation\n",
      "            dependent manner leading to a reduction of defective mitochondria\n",
      "            and myocyte cell death, and hence a better recovery of cardiac\n",
      "            function (PubMed:22044588). {ECO:0000269|PubMed:22044588,\n",
      "            ECO:0000269|PubMed:23726973}.\n",
      "            [SIMILARITY] Belongs to the phosphoglycerate mutase family.\n",
      "            {ECO:0000305}.\n",
      "            [CAUTION] Not expected to have any kinase activity. {ECO:0000305}.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..269\n",
      "                     /organism=\"Mus musculus\"\n",
      "                     /db_xref=\"taxon:10090\"\n",
      "     gene            1..269\n",
      "                     /gene=\"Tigar\"\n",
      "     Protein         1..269\n",
      "                     /product=\"Fructose-2,6-bisphosphatase TIGAR\"\n",
      "                     /EC_number=\"3.1.3.46\"\n",
      "                     /note=\"TP53-induced glycolysis and apoptosis regulator;\n",
      "                     TP53-induced glycolysis regulatory phosphatase\"\n",
      "                     /UniProtKB_evidence=\"Evidence at protein level\"\n",
      "     Region          1..269\n",
      "                     /region_name=\"Mature chain\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Fructose-2,6-bisphosphatase TIGAR.\n",
      "                     /FTId=PRO_0000179958.\"\n",
      "     Region          6..241\n",
      "                     /region_name=\"His_Phos_1\"\n",
      "                     /note=\"Histidine phosphatase superfamily (branch 1);\n",
      "                     pfam00300\"\n",
      "                     /db_xref=\"CDD:278716\"\n",
      "     Site            11\n",
      "                     /site_type=\"active\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Tele-phosphohistidine intermediate.\n",
      "                     {ECO:0000250|UniProtKB:Q7ZVE3}.\"\n",
      "     Site            89\n",
      "                     /site_type=\"active\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Proton donor/acceptor.\n",
      "                     {ECO:0000250|UniProtKB:Q7ZVE3}.\"\n",
      "     Site            198\n",
      "                     /site_type=\"other\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Transition state stabilizer.\n",
      "                     {ECO:0000250|UniProtKB:Q7ZVE3}.\"\n",
      "ORIGIN      \n",
      "        1 mprfaltvir hgetrlnkek iiqgqgvdap lsetgfrqaa aagqflsnvq fthafssdlt\n",
      "       61 rtkqtihgil eksrfckdma vkydsrlrer mygvaegkpl selramakaa geecpmftpp\n",
      "      121 ggetveqvkm rgkdffdfic qlilgkagqr esvlpgapgs glesslaevf pvgkhgslga\n",
      "      181 npkggtlgla asilvvshga ymrslfgyfl sdlrcslpga rdklelssit pntgisvfii\n",
      "      241 dceearqpsi qcvcmnlqeh lngvtekqh\n",
      "//\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LOCUS       TIGRB_DANRE              257 aa            linear   VRT 02-NOV-2016\n",
      "DEFINITION  RecName: Full=Fructose-2,6-bisphosphatase TIGAR B; AltName:\n",
      "            Full=TP53-induced glycolysis and apoptosis regulator B.\n",
      "ACCESSION   Q7ZVE3\n",
      "VERSION     Q7ZVE3.2\n",
      "DBSOURCE    UniProtKB: locus TIGRB_DANRE, accession Q7ZVE3;\n",
      "            class: standard.\n",
      "            extra accessions:Q1L8M5\n",
      "            created: Feb 10, 2009.\n",
      "            sequence updated: Feb 10, 2009.\n",
      "            annotation updated: Nov 2, 2016.\n",
      "            xrefs: CR751235.5, CAK04686.1, BC045897.1, AAH45897.1, NP_956485.1,\n",
      "            3E9C_A, 3E9C_B, 3E9D_A, 3E9D_B, 3E9E_A, 3E9E_B\n",
      "            xrefs (non-sequence databases): UniGene:Dr.79917, PDBsum:3E9C,\n",
      "            PDBsum:3E9D, PDBsum:3E9E, ProteinModelPortal:Q7ZVE3, SMR:Q7ZVE3,\n",
      "            STRING:7955.ENSDARP00000067391, PaxDb:Q7ZVE3, PRIDE:Q7ZVE3,\n",
      "            GeneID:393160, KEGG:dre:393160, CTD:393160,\n",
      "            ZFIN:ZDB-GENE-040426-885, eggNOG:ENOG410IUDB, eggNOG:COG0406,\n",
      "            HOGENOM:HOG000060277, HOVERGEN:HBG108569, InParanoid:Q7ZVE3,\n",
      "            KO:K14634, PhylomeDB:Q7ZVE3, TreeFam:TF329053,\n",
      "            EvolutionaryTrace:Q7ZVE3, PRO:PR:Q7ZVE3, Proteomes:UP000000437,\n",
      "            GO:0005737, GO:0005829, GO:0005741, GO:0005739, GO:0005634,\n",
      "            GO:0004083, GO:0003824, GO:0004331, GO:0006915, GO:0006914,\n",
      "            GO:0030388, GO:0006003, CDD:cd07067, Gene3D:3.40.50.1240,\n",
      "            InterPro:IPR013078, InterPro:IPR029033, InterPro:IPR001345,\n",
      "            Pfam:PF00300, SMART:SM00855, SUPFAM:SSF53254, PROSITE:PS00175\n",
      "KEYWORDS    3D-structure; Apoptosis; Autophagy; Complete proteome; Cytoplasm;\n",
      "            Hydrolase; Mitochondrion; Nucleus; Reference proteome.\n",
      "SOURCE      Danio rerio (zebrafish)\n",
      "  ORGANISM  Danio rerio\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Actinopterygii; Neopterygii; Teleostei; Ostariophysi;\n",
      "            Cypriniformes; Cyprinidae; Danio.\n",
      "REFERENCE   1  (residues 1 to 257)\n",
      "  AUTHORS   Howe,K., Clark,M.D., Torroja,C.F., Torrance,J., Berthelot,C.,\n",
      "            Muffato,M., Collins,J.E., Humphray,S., McLaren,K., Matthews,L.,\n",
      "            McLaren,S., Sealy,I., Caccamo,M., Churcher,C., Scott,C.,\n",
      "            Barrett,J.C., Koch,R., Rauch,G.J., White,S., Chow,W., Kilian,B.,\n",
      "            Quintais,L.T., Guerra-Assuncao,J.A., Zhou,Y., Gu,Y., Yen,J.,\n",
      "            Vogel,J.H., Eyre,T., Redmond,S., Banerjee,R., Chi,J., Fu,B.,\n",
      "            Langley,E., Maguire,S.F., Laird,G.K., Lloyd,D., Kenyon,E.,\n",
      "            Donaldson,S., Sehra,H., Almeida-King,J., Loveland,J., Trevanion,S.,\n",
      "            Jones,M., Quail,M., Willey,D., Hunt,A., Burton,J., Sims,S.,\n",
      "            McLay,K., Plumb,B., Davis,J., Clee,C., Oliver,K., Clark,R.,\n",
      "            Riddle,C., Elliot,D., Threadgold,G., Harden,G., Ware,D., Begum,S.,\n",
      "            Mortimore,B., Kerry,G., Heath,P., Phillimore,B., Tracey,A.,\n",
      "            Corby,N., Dunn,M., Johnson,C., Wood,J., Clark,S., Pelan,S.,\n",
      "            Griffiths,G., Smith,M., Glithero,R., Howden,P., Barker,N.,\n",
      "            Lloyd,C., Stevens,C., Harley,J., Holt,K., Panagiotidis,G.,\n",
      "            Lovell,J., Beasley,H., Henderson,C., Gordon,D., Auger,K.,\n",
      "            Wright,D., Collins,J., Raisen,C., Dyer,L., Leung,K., Robertson,L.,\n",
      "            Ambridge,K., Leongamornlert,D., McGuire,S., Gilderthorp,R.,\n",
      "            Griffiths,C., Manthravadi,D., Nichol,S., Barker,G., Whitehead,S.,\n",
      "            Kay,M., Brown,J., Murnane,C., Gray,E., Humphries,M., Sycamore,N.,\n",
      "            Barker,D., Saunders,D., Wallis,J., Babbage,A., Hammond,S.,\n",
      "            Mashreghi-Mohammadi,M., Barr,L., Martin,S., Wray,P., Ellington,A.,\n",
      "            Matthews,N., Ellwood,M., Woodmansey,R., Clark,G., Cooper,J.,\n",
      "            Tromans,A., Grafham,D., Skuce,C., Pandian,R., Andrews,R.,\n",
      "            Harrison,E., Kimberley,A., Garnett,J., Fosker,N., Hall,R.,\n",
      "            Garner,P., Kelly,D., Bird,C., Palmer,S., Gehring,I., Berger,A.,\n",
      "            Dooley,C.M., Ersan-Urun,Z., Eser,C., Geiger,H., Geisler,M.,\n",
      "            Karotki,L., Kirn,A., Konantz,J., Konantz,M., Oberlander,M.,\n",
      "            Rudolph-Geiger,S., Teucke,M., Lanz,C., Raddatz,G., Osoegawa,K.,\n",
      "            Zhu,B., Rapp,A., Widaa,S., Langford,C., Yang,F., Schuster,S.C.,\n",
      "            Carter,N.P., Harrow,J., Ning,Z., Herrero,J., Searle,S.M.,\n",
      "            Enright,A., Geisler,R., Plasterk,R.H., Lee,C., Westerfield,M., de\n",
      "            Jong,P.J., Zon,L.I., Postlethwait,J.H., Nusslein-Volhard,C.,\n",
      "            Hubbard,T.J., Roest Crollius,H., Rogers,J. and Stemple,D.L.\n",
      "  TITLE     The zebrafish reference genome sequence and its relationship to the\n",
      "            human genome\n",
      "  JOURNAL   Nature 496 (7446), 498-503 (2013)\n",
      "   PUBMED   23594743\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [LARGE SCALE GENOMIC DNA].;\n",
      "            STRAIN=Tuebingen\n",
      "            Erratum:[Nature. 2014 Jan 9;505(7482):248. Cooper, James [corrected\n",
      "            to Cooper, James D]; Eliott, David [corrected to Elliot, David];\n",
      "            Mortimer, Beverly [corrected to Mortimore, Beverley]; Begum,\n",
      "            Sharmin [added]; Lloyd, Christine [added]; Lanz, Christa [added];\n",
      "            Raddatz, Gunter [added]; Schuster, Stephan C [added]]\n",
      "REFERENCE   2  (residues 1 to 257)\n",
      "  CONSRTM   NIH - Zebrafish Gene Collection (ZGC) project\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (??-JAN-2003)\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [LARGE SCALE MRNA].\n",
      "REFERENCE   3  (residues 1 to 257)\n",
      "  AUTHORS   Li,H. and Jogl,G.\n",
      "  TITLE     Structural and biochemical studies of TIGAR (TP53-induced\n",
      "            glycolysis and apoptosis regulator)\n",
      "  JOURNAL   J. Biol. Chem. 284 (3), 1748-1754 (2009)\n",
      "   PUBMED   19015259\n",
      "  REMARK    X-RAY CRYSTALLOGRAPHY (2.0 ANGSTROMS), CATALYTIC ACTIVITY, AND\n",
      "            ACTIVE SITE.\n",
      "COMMENT     On or before Feb 15, 2009 this sequence version replaced\n",
      "            gi:123888109, gi:82188632.\n",
      "            [FUNCTION] Fructose-bisphosphatase hydrolyzing\n",
      "            fructose-2,6-bisphosphate as well as fructose-1,6-bisphosphate.\n",
      "            Acts as a negative regulator of glycolysis by lowering\n",
      "            intracellular levels of fructose-2,6-bisphosphate in a\n",
      "            p53/TP53-dependent manner, resulting in the pentose phosphate\n",
      "            pathway (PPP) activation and NADPH production. Contributes to the\n",
      "            generation of reduced glutathione to cause a decrease in\n",
      "            intracellular reactive oxygen species (ROS) content, correlating\n",
      "            with its ability to protect cells from oxidative or metabolic\n",
      "            stress-induced cell death. May play a role in mitophagy inhibition.\n",
      "            {ECO:0000250|UniProtKB:Q8BZA9, ECO:0000250|UniProtKB:Q9NQ88}.\n",
      "            [CATALYTIC ACTIVITY] Beta-D-fructose 2,6-bisphosphate + H(2)O =\n",
      "            D-fructose 6-phosphate + phosphate. {ECO:0000250|UniProtKB:Q9NQ88}.\n",
      "            [SUBCELLULAR LOCATION] Cytoplasm {ECO:0000250|UniProtKB:Q8BZA9}.\n",
      "            Nucleus {ECO:0000250|UniProtKB:Q9NQ88}. Mitochondrion\n",
      "            {ECO:0000250|UniProtKB:Q8BZA9}.\n",
      "            [SIMILARITY] Belongs to the phosphoglycerate mutase family.\n",
      "            {ECO:0000305}.\n",
      "            [CAUTION] Not expected to have any kinase activity. {ECO:0000305}.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..257\n",
      "                     /organism=\"Danio rerio\"\n",
      "                     /db_xref=\"taxon:7955\"\n",
      "     gene            1..257\n",
      "                     /gene=\"tigarb\"\n",
      "                     /locus_tag=\"si:ch211-240j22.3\"\n",
      "                     /gene_synonym=\"zgc:56074\"\n",
      "     Protein         1..257\n",
      "                     /product=\"Fructose-2,6-bisphosphatase TIGAR B\"\n",
      "                     /EC_number=\"3.1.3.46\"\n",
      "                     /note=\"TP53-induced glycolysis and apoptosis regulator B\"\n",
      "                     /UniProtKB_evidence=\"Evidence at protein level\"\n",
      "     Region          1..257\n",
      "                     /region_name=\"Mature chain\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Fructose-2,6-bisphosphatase TIGAR B.\n",
      "                     /FTId=PRO_0000363068.\"\n",
      "     Region          2..10\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3E9C}.\"\n",
      "     Region          6..226\n",
      "                     /region_name=\"His_Phos_1\"\n",
      "                     /note=\"Histidine phosphatase superfamily (branch 1);\n",
      "                     pfam00300\"\n",
      "                     /db_xref=\"CDD:278716\"\n",
      "     Site            11\n",
      "                     /site_type=\"active\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Tele-phosphohistidine intermediate.\n",
      "                     {ECO:0000269|PubMed:19015259}.\"\n",
      "     Region          15..18\n",
      "                     /region_name=\"Helical region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3E9D}.\"\n",
      "     Region          24..26\n",
      "                     /region_name=\"Hydrogen bonded turn\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3E9E}.\"\n",
      "     Region          33..45\n",
      "                     /region_name=\"Helical region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3E9C}.\"\n",
      "     Region          44\n",
      "                     /region_name=\"Conflict\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"R -> H (in Ref. 1; CAK04686). {ECO:0000305}.\"\n",
      "     Region          46..48\n",
      "                     /region_name=\"Hydrogen bonded turn\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3E9C}.\"\n",
      "     Region          52..56\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3E9C}.\"\n",
      "     Region          60..71\n",
      "                     /region_name=\"Helical region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3E9C}.\"\n",
      "     Region          81..83\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3E9C}.\"\n",
      "     Region          85..87\n",
      "                     /region_name=\"Helical region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3E9C}.\"\n",
      "     Site            89\n",
      "                     /site_type=\"active\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Proton donor/acceptor.\n",
      "                     {ECO:0000305|PubMed:19015259}.\"\n",
      "     Region          93..95\n",
      "                     /region_name=\"Helical region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3E9D}.\"\n",
      "     Region          100..110\n",
      "                     /region_name=\"Helical region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3E9D}.\"\n",
      "     Region          113\n",
      "                     /region_name=\"Conflict\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"S -> A (in Ref. 1; CAK04686). {ECO:0000305}.\"\n",
      "     Region          114..116\n",
      "                     /region_name=\"Hydrogen bonded turn\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3E9D}.\"\n",
      "     Region          125..147\n",
      "                     /region_name=\"Helical region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3E9C}.\"\n",
      "     Region          134\n",
      "                     /region_name=\"Conflict\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"M -> K (in Ref. 1; CAK04686). {ECO:0000305}.\"\n",
      "     Region          144\n",
      "                     /region_name=\"Conflict\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"L -> F (in Ref. 2; AAH45897). {ECO:0000305}.\"\n",
      "     Region          149..151\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3E9C}.\"\n",
      "     Region          154\n",
      "                     /region_name=\"Conflict\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"A -> V (in Ref. 2; AAH45897). {ECO:0000305}.\"\n",
      "     Region          167..170\n",
      "                     /region_name=\"Hydrogen bonded turn\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3E9C}.\"\n",
      "     Region          177..182\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3E9C}.\"\n",
      "     Site            183\n",
      "                     /site_type=\"other\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Transition state stabilizer.\n",
      "                     {ECO:0000305|PubMed:19015259}.\"\n",
      "     Region          184..196\n",
      "                     /region_name=\"Helical region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3E9C}.\"\n",
      "     Region          189\n",
      "                     /region_name=\"Conflict\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"I -> V (in Ref. 1; CAK04686). {ECO:0000305}.\"\n",
      "     Region          200..202\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3E9C}.\"\n",
      "     Region          208..211\n",
      "                     /region_name=\"Helical region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3E9C}.\"\n",
      "     Region          220..230\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3E9C}.\"\n",
      "     Region          233..246\n",
      "                     /region_name=\"Beta-strand region\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"{ECO:0000244|PDB:3E9C}.\"\n",
      "ORIGIN      \n",
      "        1 mltfaltivr hgetqynrdk llqgqgidtp lsdtghqqaa aagrylkdlh ftnvfvsnlq\n",
      "       61 raiqtaeiil gnnlhssate mildpllrer gfgvaegrpk ehlknmanaa gqscrdytpp\n",
      "      121 ggetleqvkt rfkmflkslf qrmleehgsa lssapseadq pviagladdg aqnvpvhalm\n",
      "      181 vshgafiris vrhlvedlqc clpaglkmnq vfspcpntgi srfiftihre esvlratriq\n",
      "      241 gvfinrkdhl eevknsd\n",
      "//\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LOCUS       2WM1_A                   336 aa            linear   PRI 10-OCT-2012\n",
      "DEFINITION  Chain A, The Crystal Structure Of Human Alpha-Amino-Beta-\n",
      "            Carboxymuconate-Epsilon-Semialdehyde Decarboxylase In Complex With\n",
      "            1,3- Dihydroxyacetonephosphate Suggests A Regulatory Link Between\n",
      "            Nad Synthesis And Glycolysis.\n",
      "ACCESSION   2WM1_A\n",
      "VERSION     2WM1_A\n",
      "DBSOURCE    pdb: molecule 2WM1, chain 65, release Feb 10, 2010;\n",
      "            deposition: Jun 29, 2009;\n",
      "            class: Lyase;\n",
      "            source: Mmdb_id: 77756, Pdb_id 1: 2WM1;\n",
      "            Exp. method: X-Ray Diffraction.\n",
      "KEYWORDS    .\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1  (residues 1 to 336)\n",
      "  AUTHORS   Garavaglia,S., Perozzi,S., Galeazzi,L., Raffaelli,N. and Rizzi,M.\n",
      "  TITLE     The crystal structure of human\n",
      "            alpha-amino-beta-carboxymuconate-epsilon-semialdehyde decarboxylase\n",
      "            in complex with 1,3-dihydroxyacetonephosphate suggests a regulatory\n",
      "            link between NAD synthesis and glycolysis\n",
      "  JOURNAL   FEBS J. 276 (22), 6615-6623 (2009)\n",
      "   PUBMED   19843166\n",
      "REFERENCE   2  (residues 1 to 336)\n",
      "  AUTHORS   Garavaglia,S., Perozzi,S., Galeazzi,L., Raffaelli,N. and Rizzi,M.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (29-JUN-2009)\n",
      "COMMENT     SEQRES.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..336\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "     Region          join(1..18,68..336)\n",
      "                     /region_name=\"Domain 1\"\n",
      "                     /note=\"NCBI Domains\"\n",
      "     SecStr          2..5\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 1\"\n",
      "     Het             join(bond(6),bond(8),bond(174),bond(291))\n",
      "                     /heterogen=\"(ZN,1000)\"\n",
      "     Site            order(6,8,174,224,291)\n",
      "                     /site_type=\"active\"\n",
      "                     /db_xref=\"CDD:238617\"\n",
      "     SecStr          6..9\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 2\"\n",
      "     Region          19..53\n",
      "                     /region_name=\"Domain 2\"\n",
      "                     /note=\"NCBI Domains\"\n",
      "     SecStr          27..32\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 3\"\n",
      "     SecStr          35..42\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 4\"\n",
      "     SecStr          43..51\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 5\"\n",
      "     Region          50..330\n",
      "                     /region_name=\"Amidohydro_2\"\n",
      "                     /note=\"Amidohydrolase; pfam04909\"\n",
      "                     /db_xref=\"CDD:282727\"\n",
      "     SecStr          56..66\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 1\"\n",
      "     SecStr          69..75\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 6\"\n",
      "     SecStr          86..107\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 2\"\n",
      "     SecStr          110..114\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 7\"\n",
      "     SecStr          122..134\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 3\"\n",
      "     SecStr          139..145\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 8\"\n",
      "     SecStr          158..167\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 4\"\n",
      "     SecStr          169..175\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 9\"\n",
      "     SecStr          194..209\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 5\"\n",
      "     SecStr          219..223\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 10\"\n",
      "     SecStr          233..243\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 6\"\n",
      "     SecStr          262..266\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 11\"\n",
      "     SecStr          271..280\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 7\"\n",
      "     SecStr          285..288\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 12\"\n",
      "     SecStr          313..328\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 8\"\n",
      "ORIGIN      \n",
      "        1 mkidihshil pkewpdlkkr fgyggwvqlq hhskgeakll kdgkvfrvvr encwdpevri\n",
      "       61 remdqkgvtv qalstvpvmf sywakpedtl nlcqllnndl astvvsyprr fvglgtlpmq\n",
      "      121 apelavkeme rcvkelgfpg vqigthvnew dlnaqelfpv yaaaerlkcs lfvhpwdmqm\n",
      "      181 dgrmakywlp wlvgmpaett iaicsmimgg vfekfpklkv cfahgggafp ftvgrishgf\n",
      "      241 smrpdlcaqd npmnpkkylg sfytdalvhd plslklltdv igkdkvilgt dypfplgele\n",
      "      301 pgkliesmee fdeetknklk agnalaflgl erkqfe\n",
      "//\n",
      "\n",
      "\n",
      "LOCUS       TIGRA_DANRE              256 aa            linear   VRT 12-SEP-2018\n",
      "DEFINITION  RecName: Full=Probable fructose-2,6-bisphosphatase TIGAR A;\n",
      "            AltName: Full=TP53-induced glycolysis and apoptosis regulator A.\n",
      "ACCESSION   Q29RA5\n",
      "VERSION     Q29RA5.1\n",
      "DBSOURCE    UniProtKB: locus TIGRA_DANRE, accession Q29RA5;\n",
      "            class: standard.\n",
      "            created: Feb 10, 2009.\n",
      "            sequence updated: Apr 4, 2006.\n",
      "            annotation updated: Sep 12, 2018.\n",
      "            xrefs: BC114302.1, AAI14303.1, NP_001034925.1\n",
      "            xrefs (non-sequence databases): UniGene:Dr.83996,\n",
      "            ProteinModelPortal:Q29RA5, SMR:Q29RA5,\n",
      "            STRING:7955.ENSDARP00000097862, PaxDb:Q29RA5, PRIDE:Q29RA5,\n",
      "            Ensembl:ENSDART00000111158, Ensembl:ENSDARP00000097862,\n",
      "            Ensembl:ENSDARG00000051749, GeneID:664696, KEGG:dre:664696,\n",
      "            CTD:664696, ZFIN:ZDB-GENE-060312-25, eggNOG:KOG0235,\n",
      "            eggNOG:COG0588, GeneTree:ENSGT00390000013224, HOGENOM:HOG000060277,\n",
      "            HOVERGEN:HBG108569, InParanoid:Q29RA5, KO:K14634, OMA:NFEEGRE,\n",
      "            OrthoDB:EOG091G0W1L, PhylomeDB:Q29RA5, TreeFam:TF329053,\n",
      "            Reactome:R-DRE-5628897, PRO:PR:Q29RA5, Proteomes:UP000000437,\n",
      "            Bgee:ENSDARG00000051749, GO:0005737, GO:0005829, GO:0005741,\n",
      "            GO:0005739, GO:0005634, GO:0004083, GO:0004331, GO:0016791,\n",
      "            GO:0006915, GO:0006914, GO:0030388, GO:0006003, CDD:cd07067,\n",
      "            Gene3D:3.40.50.1240, InterPro:IPR013078, InterPro:IPR029033,\n",
      "            InterPro:IPR001345, Pfam:PF00300, SMART:SM00855, SUPFAM:SSF53254,\n",
      "            PROSITE:PS00175\n",
      "KEYWORDS    Apoptosis; Autophagy; Complete proteome; Cytoplasm; Hydrolase;\n",
      "            Mitochondrion; Nucleus; Reference proteome.\n",
      "SOURCE      Danio rerio (zebrafish)\n",
      "  ORGANISM  Danio rerio\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Actinopterygii; Neopterygii; Teleostei; Ostariophysi;\n",
      "            Cypriniformes; Cyprinidae; Danio.\n",
      "REFERENCE   1  (residues 1 to 256)\n",
      "  CONSRTM   NIH - Zebrafish Gene Collection (ZGC) project\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (??-MAR-2006)\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [LARGE SCALE MRNA].\n",
      "COMMENT     [FUNCTION] Fructose-bisphosphatase hydrolyzing\n",
      "            fructose-2,6-bisphosphate as well as fructose-1,6-bisphosphate.\n",
      "            Acts as a negative regulator of glycolysis by lowering\n",
      "            intracellular levels of fructose-2,6-bisphosphate in a\n",
      "            p53/TP53-dependent manner, resulting in the pentose phosphate\n",
      "            pathway (PPP) activation and NADPH production. Contributes to the\n",
      "            generation of reduced glutathione to cause a decrease in\n",
      "            intracellular reactive oxygen species (ROS) content, correlating\n",
      "            with its ability to protect cells from oxidative or metabolic\n",
      "            stress-induced cell death. May play a role in mitophagy inhibition.\n",
      "            {ECO:0000250|UniProtKB:Q8BZA9, ECO:0000250|UniProtKB:Q9NQ88}.\n",
      "            [CATALYTIC ACTIVITY] Beta-D-fructose 2,6-bisphosphate + H(2)O =\n",
      "            D-fructose 6-phosphate + phosphate. {ECO:0000250|UniProtKB:Q9NQ88}.\n",
      "            [SUBCELLULAR LOCATION] Cytoplasm {ECO:0000250|UniProtKB:Q8BZA9}.\n",
      "            Nucleus {ECO:0000250|UniProtKB:Q9NQ88}. Mitochondrion\n",
      "            {ECO:0000250|UniProtKB:Q8BZA9}.\n",
      "            [SIMILARITY] Belongs to the phosphoglycerate mutase family.\n",
      "            {ECO:0000305}.\n",
      "            [CAUTION] Not expected to have any kinase activity. {ECO:0000305}.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..256\n",
      "                     /organism=\"Danio rerio\"\n",
      "                     /db_xref=\"taxon:7955\"\n",
      "     gene            1..256\n",
      "                     /gene=\"tigara\"\n",
      "                     /locus_tag=\"zgc:136900\"\n",
      "     Protein         1..256\n",
      "                     /product=\"Probable fructose-2,6-bisphosphatase TIGAR A\"\n",
      "                     /EC_number=\"3.1.3.46\"\n",
      "                     /note=\"TP53-induced glycolysis and apoptosis regulator A\"\n",
      "                     /UniProtKB_evidence=\"Evidence at transcript level\"\n",
      "     Region          1..256\n",
      "                     /region_name=\"Mature chain\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Probable fructose-2,6-bisphosphatase TIGAR A.\n",
      "                     /FTId=PRO_0000363067.\"\n",
      "     Region          6..229\n",
      "                     /region_name=\"His_Phos_1\"\n",
      "                     /note=\"Histidine phosphatase superfamily (branch 1);\n",
      "                     pfam00300\"\n",
      "                     /db_xref=\"CDD:278716\"\n",
      "     Site            11\n",
      "                     /site_type=\"active\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Tele-phosphohistidine intermediate.\n",
      "                     {ECO:0000250|UniProtKB:Q7ZVE3}.\"\n",
      "     Site            89\n",
      "                     /site_type=\"active\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Proton donor/acceptor.\n",
      "                     {ECO:0000250|UniProtKB:Q7ZVE3}.\"\n",
      "     Site            186\n",
      "                     /site_type=\"other\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Transition state stabilizer.\n",
      "                     {ECO:0000250|UniProtKB:Q7ZVE3}.\"\n",
      "ORIGIN      \n",
      "        1 mlafgltvvr hgetqcnkdg llqgqkidsl lsdigiqqse aagqylrdvk ftnvfvsnmk\n",
      "       61 rakqtaeiiv rnnrtchdle lvadpslier sfgiaeggrv idmknmakaa gqplpeftpp\n",
      "      121 egetmeqvkl rikdflkamy qriandhqdk vqdggtssad esteapagla ndgvssvpvh\n",
      "      181 alvvghgaym siamryffed lkcpvprgld paqqfsicpn tgmcrfiitl kcssvdials\n",
      "      241 dikcvfinrr dhfktn\n",
      "//\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LOCUS       3DCY_A                   275 aa            linear   PRI 31-OCT-2017\n",
      "DEFINITION  Chain A, Regulator Protein.\n",
      "ACCESSION   3DCY_A\n",
      "VERSION     3DCY_A\n",
      "DBSOURCE    pdb: molecule 3DCY, chain 65, release Oct 27, 2017;\n",
      "            deposition: Jun 4, 2008;\n",
      "            class: Apoptosis Regulator;\n",
      "            source: Mmdb_id: 65515, Pdb_id 1: 3DCY;\n",
      "            Exp. method: X-Ray Diffraction.\n",
      "KEYWORDS    .\n",
      "SOURCE      Homo sapiens (human)\n",
      "  ORGANISM  Homo sapiens\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Mammalia; Eutheria; Euarchontoglires; Primates; Haplorrhini;\n",
      "            Catarrhini; Hominidae; Homo.\n",
      "REFERENCE   1  (residues 1 to 275)\n",
      "  AUTHORS   Mccoy,J.G., Bingman,C.A., Wesenberg,G.E. and Phillips,G.N. Jr.\n",
      "  TITLE     Crystal Structure A Tp53-Induced Glycolysis And Apo Regulator\n",
      "            Protein From Homo Sapiens\n",
      "  JOURNAL   Unpublished\n",
      "REFERENCE   2  (residues 1 to 275)\n",
      "  AUTHORS   Mccoy,J.G., Bingman,C.A., Wesenberg,G.E., Phillips,G.N. Jr., Center\n",
      "            and Eukaryotic Structural Genomics (Cesg).\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (04-JUN-2008)\n",
      "COMMENT     Crystal Structure A Tp53-induced Glycolysis And Apoptosis Regulator\n",
      "            Protein From Homo Sapiens.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..275\n",
      "                     /organism=\"Homo sapiens\"\n",
      "                     /db_xref=\"taxon:9606\"\n",
      "     SecStr          3..8\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 1\"\n",
      "     SecStr          9..16\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 2\"\n",
      "     Region          11..>146\n",
      "                     /region_name=\"His_Phos_1\"\n",
      "                     /note=\"Histidine phosphatase superfamily (branch 1);\n",
      "                     pfam00300\"\n",
      "                     /db_xref=\"CDD:278716\"\n",
      "     SecStr          38..51\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 1\"\n",
      "     SecStr          57..62\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 3\"\n",
      "     SecStr          65..77\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 2\"\n",
      "     SecStr          86..89\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 4\"\n",
      "     SecStr          96..99\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 5\"\n",
      "     SecStr          106..115\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 3\"\n",
      "     SecStr          125..128\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 6\"\n",
      "     SecStr          130..153\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 4\"\n",
      "     SecStr          165..173\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 5\"\n",
      "     SecStr          195..203\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 7\"\n",
      "     SecStr          204..215\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 6\"\n",
      "     SecStr          220..223\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 8\"\n",
      "     SecStr          239..247\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 9\"\n",
      "     SecStr          248..251\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 10\"\n",
      "     SecStr          255..263\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 11\"\n",
      "ORIGIN      \n",
      "        1 nlyfqsarfa ltvvrhgetr fnkekiiqgq gvdeplsetg fkqaaaagif lnnvkfthaf\n",
      "       61 ssdlxrtkqt xhgilerskf ckdxtvkyds rlrerkygvv egkalselra xakaareecp\n",
      "      121 vftppggetl dqvkxrgidf feflcqlilk eadqkeqfsq gspsnclets laeifplgkn\n",
      "      181 hsskvnsdsg ipglaasvlv vshgayxrsl fdyfltdlkc slpatlsrse lxsvtpntgx\n",
      "      241 slfiinfeeg revkptvqci cxnlqdhlng ltetr\n",
      "//\n",
      "\n",
      "\n",
      "LOCUS       XP_002490159             918 aa            linear   PLN 11-OCT-2017\n",
      "DEFINITION  Beta subunit of heterooctameric phosphofructokinase involved in\n",
      "            glycolysis [Komagataella phaffii GS115].\n",
      "ACCESSION   XP_002490159\n",
      "VERSION     XP_002490159.1\n",
      "DBLINK      BioProject: PRJNA39439\n",
      "            BioSample: SAMEA2272385\n",
      "DBSOURCE    REFSEQ: accession XM_002490114.1\n",
      "KEYWORDS    RefSeq.\n",
      "SOURCE      Komagataella phaffii GS115\n",
      "  ORGANISM  Komagataella phaffii GS115\n",
      "            Eukaryota; Fungi; Dikarya; Ascomycota; Saccharomycotina;\n",
      "            Saccharomycetes; Saccharomycetales; Phaffomycetaceae; Komagataella.\n",
      "REFERENCE   1\n",
      "  AUTHORS   De Schutter,K., Lin,Y.C., Tiels,P., Van Hecke,A., Glinka,S.,\n",
      "            Weber-Lehmann,J., Rouze,P., Van de Peer,Y. and Callewaert,N.\n",
      "  TITLE     Genome sequence of the recombinant protein production host Pichia\n",
      "            pastoris\n",
      "  JOURNAL   Nat. Biotechnol. 27 (6), 561-566 (2009)\n",
      "   PUBMED   19465926\n",
      "REFERENCE   2  (residues 1 to 918)\n",
      "  CONSRTM   NCBI Genome Project\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (11-OCT-2017) National Center for Biotechnology\n",
      "            Information, NIH, Bethesda, MD 20894, USA\n",
      "REFERENCE   3  (residues 1 to 918)\n",
      "  AUTHORS   Callewaert,N.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (05-MAY-2009) Callewaert N., Department for Molecular\n",
      "            Biomedical Research, Flanders Institute for Biotechnology (VIB) /\n",
      "            Gent University, Technologiepark 927, Ghent, 9052, BELGIUM\n",
      "COMMENT     PROVISIONAL REFSEQ: This record has not yet been subject to final\n",
      "            NCBI review. The reference sequence is identical to CAY67878.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..918\n",
      "                     /organism=\"Komagataella phaffii GS115\"\n",
      "                     /strain=\"GS115\"\n",
      "                     /db_xref=\"taxon:644223\"\n",
      "                     /chromosome=\"1\"\n",
      "     Protein         1..918\n",
      "                     /product=\"Beta subunit of heterooctameric\n",
      "                     phosphofructokinase involved in glycolysis\"\n",
      "                     /calculated_mol_wt=101313\n",
      "     Region          13..143\n",
      "                     /region_name=\"Glo_EDI_BRP_like\"\n",
      "                     /note=\"This domain superfamily is found in a variety of\n",
      "                     structurally related metalloproteins, including the type I\n",
      "                     extradiol dioxygenases, glyoxalase I and a group of\n",
      "                     antibiotic resistance proteins; cd06587\"\n",
      "                     /db_xref=\"CDD:211348\"\n",
      "     Site            order(16,48,59,64,79,132,140,142)\n",
      "                     /site_type=\"active\"\n",
      "                     /db_xref=\"CDD:211348\"\n",
      "     Region          180..913\n",
      "                     /region_name=\"PFK\"\n",
      "                     /note=\"Phosphofructokinase, a key regulatory enzyme in\n",
      "                     glycolysis, catalyzes the phosphorylation of\n",
      "                     fructose-6-phosphate to fructose-1,6-biphosphate. The\n",
      "                     members belong to PFK family that includes ATP- and\n",
      "                     pyrophosphate (PPi)- dependent phosphofructokinases;\n",
      "                     cl00204\"\n",
      "                     /db_xref=\"CDD:294137\"\n",
      "     Region          182..917\n",
      "                     /region_name=\"6PF1K_euk\"\n",
      "                     /note=\"6-phosphofructokinase, eukaryotic type; TIGR02478\"\n",
      "                     /db_xref=\"CDD:274152\"\n",
      "     Site            order(191,221,255,286..288,290..291,331,333,335,375..377,\n",
      "                     432,466,469)\n",
      "                     /site_type=\"active\"\n",
      "                     /db_xref=\"CDD:238216\"\n",
      "     Site            order(191,221,255,286..288,290..291)\n",
      "                     /site_type=\"other\"\n",
      "                     /note=\"ADP/pyrophosphate binding site [chemical binding]\"\n",
      "                     /db_xref=\"CDD:238216\"\n",
      "     Site            order(201,205,237,242,245,341,353,357,360,388..389,391,\n",
      "                     423,478..479,483,490,505,540)\n",
      "                     /site_type=\"other\"\n",
      "                     /note=\"dimerization interface [polypeptide binding]\"\n",
      "                     /db_xref=\"CDD:238216\"\n",
      "     Site            order(201,205,237..238,241..242,360,391,393,421,423..425)\n",
      "                     /site_type=\"other\"\n",
      "                     /note=\"allosteric effector site\"\n",
      "                     /db_xref=\"CDD:238216\"\n",
      "     Site            order(331,333,335,368,375..377,432,460,466,469)\n",
      "                     /site_type=\"other\"\n",
      "                     /note=\"fructose-1,6-bisphosphate binding site\"\n",
      "                     /db_xref=\"CDD:238216\"\n",
      "     CDS             1..918\n",
      "                     /locus_tag=\"PAS_chr1-4_0047\"\n",
      "                     /coded_by=\"XM_002490114.1:1..2757\"\n",
      "                     /db_xref=\"EnsemblGenomes-Gn:PAS_chr1-4_0047\"\n",
      "                     /db_xref=\"EnsemblGenomes-Tr:CAY67878\"\n",
      "                     /db_xref=\"GeneID:8196884\"\n",
      "                     /db_xref=\"GOA:C4QXA5\"\n",
      "                     /db_xref=\"InterPro:IPR000023\"\n",
      "                     /db_xref=\"InterPro:IPR009161\"\n",
      "                     /db_xref=\"InterPro:IPR015912\"\n",
      "                     /db_xref=\"InterPro:IPR022953\"\n",
      "                     /db_xref=\"InterPro:IPR029068\"\n",
      "ORIGIN      \n",
      "        1 mpdaslfngt sfitlfapni slfqasidfy tdrlgfaike tsnqklvwlq leedsnnvsi\n",
      "       61 qllldpehaa svsqidqnir nltrslyrkd wrsiqsniaf kssslsklvk llkdgghpvq\n",
      "      121 qspneispfe vytldplgsl igfsgfknpf avnersllpk vseekayrte ddseklftpi\n",
      "      181 rktigvmtsg gdspgmnpfv ravvragiyk gckvfciheg yeglvrggek yiketqwhdv\n",
      "      241 rgwlveggtn igtarckefr ersgrlkack nmidmgidal ivcggdgslt gadrfrsewp\n",
      "      301 slieellqte risqqqfety qnlnicgavg sidndmsstd atigafssld ricraidyid\n",
      "      361 atanshsraf ivevmgrhcg wlgllaglat sadyilipek passrewqdq mcdivskhra\n",
      "      421 rgkrktiviv aegaisndls piscdqvkdv lvnrlgldtr vttlghvqrg gtavafdriy\n",
      "      481 atlqgveavn avlecnadtp spmiaikedq itrvplvdav eltqqvaksi esrnfkrais\n",
      "      541 lrdsefvehm knfistnsad hvppslplek rkkvaiinvg apaggmnsav ysmatycmsr\n",
      "      601 ghvpyaihng fsglarhesv rsinwldieg wgslggseig tnrtlpndad igmiayffek\n",
      "      661 ygfdglilvg gfeafislhq lerarinyps lriplvlipa tisnnvpgte yslgsdtcln\n",
      "      721 sfmeycdvik qsaaatrnrv fvvevqggns gyiathaqla cgaqisyvpe egislaqlem\n",
      "      781 dinslkesfa ndqgktkfrq tdfevgecie aipghvqqgg ipspmdrvra srfairavsf\n",
      "      841 iekhsdkcqa fknsisfrqt deitstavvl gihksqlrft pirqlydfes dvprrmrkni\n",
      "      901 fwsnvreisd mlsgrtsl\n",
      "//\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LOCUS       TIGAR_XENLA              275 aa            linear   VRT 30-AUG-2017\n",
      "DEFINITION  RecName: Full=Fructose-2,6-bisphosphatase TIGAR; AltName:\n",
      "            Full=TP53-induced glycolysis and apoptosis regulator.\n",
      "ACCESSION   Q4V7R0\n",
      "VERSION     Q4V7R0.1\n",
      "DBSOURCE    UniProtKB: locus TIGAR_XENLA, accession Q4V7R0;\n",
      "            class: standard.\n",
      "            created: Feb 10, 2009.\n",
      "            sequence updated: Jul 5, 2005.\n",
      "            annotation updated: Aug 30, 2017.\n",
      "            xrefs: BC097765.1, AAH97765.1, NP_001089513.1\n",
      "            xrefs (non-sequence databases): UniGene:Xl.54011,\n",
      "            ProteinModelPortal:Q4V7R0, SMR:Q4V7R0, MaxQB:Q4V7R0, GeneID:734566,\n",
      "            KEGG:xla:734566, CTD:734566, Xenbase:XB-GENE-1008476,\n",
      "            HOVERGEN:HBG108569, KO:K14634, GO:0005737, GO:0005741, GO:0005739,\n",
      "            GO:0005634, GO:0004083, GO:0004331, GO:0006915, GO:0006914,\n",
      "            GO:0030388, GO:0006003, CDD:cd07067, Gene3D:3.40.50.1240,\n",
      "            InterPro:IPR013078, InterPro:IPR029033, InterPro:IPR001345,\n",
      "            Pfam:PF00300, SMART:SM00855, SUPFAM:SSF53254, PROSITE:PS00175\n",
      "KEYWORDS    Apoptosis; Autophagy; Cytoplasm; Hydrolase; Mitochondrion; Nucleus.\n",
      "SOURCE      Xenopus laevis (African clawed frog)\n",
      "  ORGANISM  Xenopus laevis\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Amphibia; Batrachia; Anura; Pipoidea; Pipidae; Xenopodinae;\n",
      "            Xenopus; Xenopus.\n",
      "REFERENCE   1  (residues 1 to 275)\n",
      "  CONSRTM   NIH - Xenopus Gene Collection (XGC) project\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (??-JUN-2005)\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [LARGE SCALE MRNA].;\n",
      "            TISSUE=Egg\n",
      "COMMENT     [FUNCTION] Fructose-bisphosphatase hydrolyzing\n",
      "            fructose-2,6-bisphosphate as well as fructose-1,6-bisphosphate.\n",
      "            Acts as a negative regulator of glycolysis by lowering\n",
      "            intracellular levels of fructose-2,6-bisphosphate in a\n",
      "            p53/TP53-dependent manner, resulting in the pentose phosphate\n",
      "            pathway (PPP) activation and NADPH production. Contributes to the\n",
      "            generation of reduced glutathione to cause a decrease in\n",
      "            intracellular reactive oxygen species (ROS) content, correlating\n",
      "            with its ability to protect cells from oxidative or metabolic\n",
      "            stress-induced cell death. May play a role in mitophagy inhibition.\n",
      "            {ECO:0000250|UniProtKB:Q8BZA9, ECO:0000250|UniProtKB:Q9NQ88}.\n",
      "            [CATALYTIC ACTIVITY] Beta-D-fructose 2,6-bisphosphate + H(2)O =\n",
      "            D-fructose 6-phosphate + phosphate. {ECO:0000250|UniProtKB:Q9NQ88}.\n",
      "            [SUBCELLULAR LOCATION] Cytoplasm {ECO:0000250|UniProtKB:Q8BZA9}.\n",
      "            Nucleus {ECO:0000250|UniProtKB:Q9NQ88}. Mitochondrion\n",
      "            {ECO:0000250|UniProtKB:Q8BZA9}.\n",
      "            [SIMILARITY] Belongs to the phosphoglycerate mutase family.\n",
      "            {ECO:0000305}.\n",
      "            [CAUTION] Not expected to have any kinase activity. {ECO:0000305}.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..275\n",
      "                     /organism=\"Xenopus laevis\"\n",
      "                     /db_xref=\"taxon:8355\"\n",
      "     gene            1..275\n",
      "                     /gene=\"tigar\"\n",
      "     Protein         1..275\n",
      "                     /product=\"Fructose-2,6-bisphosphatase TIGAR\"\n",
      "                     /EC_number=\"3.1.3.46\"\n",
      "                     /note=\"TP53-induced glycolysis and apoptosis regulator\"\n",
      "                     /UniProtKB_evidence=\"Evidence at transcript level\"\n",
      "     Region          1..275\n",
      "                     /region_name=\"Mature chain\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Fructose-2,6-bisphosphatase TIGAR.\n",
      "                     /FTId=PRO_0000363069.\"\n",
      "     Region          6..>141\n",
      "                     /region_name=\"His_Phos_1\"\n",
      "                     /note=\"Histidine phosphatase superfamily (branch 1);\n",
      "                     pfam00300\"\n",
      "                     /db_xref=\"CDD:278716\"\n",
      "     Site            11\n",
      "                     /site_type=\"active\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Tele-phosphohistidine intermediate.\n",
      "                     {ECO:0000250|UniProtKB:Q7ZVE3}.\"\n",
      "     Site            89\n",
      "                     /site_type=\"active\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Proton donor/acceptor.\n",
      "                     {ECO:0000250|UniProtKB:Q7ZVE3}.\"\n",
      "     Site            198\n",
      "                     /site_type=\"other\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Transition state stabilizer.\n",
      "                     {ECO:0000250|UniProtKB:Q7ZVE3}.\"\n",
      "ORIGIN      \n",
      "        1 marfaltivr hgetrynkek llqgqgidep lsemgfkqad aagrflsnvr fthvfssdli\n",
      "       61 rakqtacaim rnnqlsedik imydprlrer kygdaegrpl selkvmakka ggqcpsytpp\n",
      "      121 ggetleqvra cakdffeylc qlvmaessvk ekselgasgm vgimstdlap fvnhnkepti\n",
      "      181 fgesrdvtld asvllvshga ymrnwikyfv edlqftfppe lkksrelsvs pntgishfiv\n",
      "      241 tvgsgatrkp eiqcvcinlh ghlsdidadt shyqv\n",
      "//\n",
      "\n",
      "\n",
      "LOCUS       SMN22373                 998 aa            linear   PLN 28-APR-2017\n",
      "DEFINITION  similar to Saccharomyces cerevisiae YGR240C PFK1 Alpha subunit of\n",
      "            heterooctameric phosphofructokinase involved in glycolysis\n",
      "            [Kazachstania saulgeensis].\n",
      "ACCESSION   SMN22373\n",
      "VERSION     SMN22373.1\n",
      "DBLINK      BioProject: PRJEB20516\n",
      "            BioSample: SAMEA104016679\n",
      "DBSOURCE    embl accession FXLY01000011.1\n",
      "KEYWORDS    .\n",
      "SOURCE      Kazachstania saulgeensis\n",
      "  ORGANISM  Kazachstania saulgeensis\n",
      "            Eukaryota; Fungi; Dikarya; Ascomycota; Saccharomycotina;\n",
      "            Saccharomycetes; Saccharomycetales; Saccharomycetaceae;\n",
      "            Kazachstania.\n",
      "REFERENCE   1\n",
      "  AUTHORS   Casaregola,S.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (24-APR-2017) INRA-MICALIS, CIRM-Levures,\n",
      "            INRA/AgroParisTech, 78350 Jouy-en-Josas, France\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..998\n",
      "                     /organism=\"Kazachstania saulgeensis\"\n",
      "                     /isolation_source=\"Organic sourdough\"\n",
      "                     /db_xref=\"taxon:1789683\"\n",
      "                     /note=\"contig: scaffold_10\"\n",
      "     Protein         1..998\n",
      "                     /product=\"similar to Saccharomyces cerevisiae YGR240C PFK1\n",
      "                     Alpha subunit of heterooctameric phosphofructokinase\n",
      "                     involved in glycolysis\"\n",
      "     CDS             1..998\n",
      "                     /locus_tag=\"KASA_0H01727G\"\n",
      "                     /coded_by=\"complement(FXLY01000011.1:322177..325173)\"\n",
      "ORIGIN      \n",
      "        1 mtaadpyygv aftslvttde tlfkqtirfy hslgfatikd fdkfttgeds llsigtshds\n",
      "       61 iretwlesfk lseidelgfr vpqqeaknsk qsegallkir lvqpskntsg qiiansnlii\n",
      "      121 tyftaeleel qsrfpeaaie qfpnstlsil rladplgnki eftnfvhpad skptdkdffq\n",
      "      181 naennnvdsa tpaamlsnie klkatlagdk kpkkkiavmt sggdspgmna avravvrtgi\n",
      "      241 hygcdmfavy egyegllkgg eylkqmawed vrgwlseggt ligtarsmef kkregrkraa\n",
      "      301 gnlieqgida lvvcggdgsl tgadlfrhew pslvedlvkd grfteaqvap yknltivglv\n",
      "      361 gsidndmsgt dstigaysal ericemvdyi datakshsra fvvevmgrhc gwlalmagia\n",
      "      421 tgadyifipe ravpagkwqq elkdvcqrhr nkgrrnntvi vaegalddel npitaenvkd\n",
      "      481 vlidlgldtk vtilghvqrg gtavahdrwl atlqgvdavk avlentpetp spligilenk\n",
      "      541 iirmplvesv kltkqvadai esknfdkais lrdtefiely enflsttvrd dgtellppae\n",
      "      601 rlnigivhvg apsaalnaat raatlyclsh ghkpfaimng fsgliqtgrv kelswidven\n",
      "      661 whnlggseig tnrsvaaedm gtiayyfqkf kfdglmilgg fegfrslkql rdsrkdhpif\n",
      "      721 nipmtlipst isnnvpgtey slgvdtclnt lvnytddikq sasatrkrcf vvevqgghsg\n",
      "      781 ylasftglvt gavsvytpeh kinlssiaed lsllkenfsh dkgevrngki virneeascv\n",
      "      841 ytteliadiy ceqsdkkfga ktaipghvqq ggvpsskdrv tasrfavkcv kffenwnkrn\n",
      "      901 lpcdgsdahi lrfhynakge kiytvanedd saavicvngs qvtfkpiakl weeetsvelr\n",
      "      961 kgtevhweef nkigdilsgr lklraevlaa kqlaeaql\n",
      "//\n",
      "\n",
      "\n",
      "LOCUS       TIGAR_XENTR              275 aa            linear   VRT 12-APR-2017\n",
      "DEFINITION  RecName: Full=Fructose-2,6-bisphosphatase TIGAR; AltName:\n",
      "            Full=TP53-induced glycolysis and apoptosis regulator.\n",
      "ACCESSION   B1WAX6\n",
      "VERSION     B1WAX6.1\n",
      "DBSOURCE    UniProtKB: locus TIGAR_XENTR, accession B1WAX6;\n",
      "            class: standard.\n",
      "            created: Feb 10, 2009.\n",
      "            sequence updated: May 20, 2008.\n",
      "            annotation updated: Apr 12, 2017.\n",
      "            xrefs: BC161536.1, AAI61536.1, NP_001120569.1\n",
      "            xrefs (non-sequence databases): UniGene:Str.52599,\n",
      "            ProteinModelPortal:B1WAX6, SMR:B1WAX6,\n",
      "            STRING:8364.ENSXETP00000013129, PaxDb:B1WAX6, GeneID:100145723,\n",
      "            KEGG:xtr:100145723, CTD:57103, Xenbase:XB-GENE-1008472,\n",
      "            eggNOG:ENOG410IUDB, eggNOG:COG0406, HOGENOM:HOG000060277,\n",
      "            HOVERGEN:HBG108569, InParanoid:B1WAX6, KO:K14634,\n",
      "            Proteomes:UP000008143, GO:0005737, GO:0005829, GO:0005741,\n",
      "            GO:0005739, GO:0005634, GO:0004083, GO:0004331, GO:0016791,\n",
      "            GO:0006915, GO:0006914, GO:0030388, GO:0006003, CDD:cd07067,\n",
      "            Gene3D:3.40.50.1240, InterPro:IPR013078, InterPro:IPR029033,\n",
      "            InterPro:IPR001345, Pfam:PF00300, SMART:SM00855, SUPFAM:SSF53254,\n",
      "            PROSITE:PS00175\n",
      "KEYWORDS    Apoptosis; Autophagy; Complete proteome; Cytoplasm; Hydrolase;\n",
      "            Mitochondrion; Nucleus; Reference proteome.\n",
      "SOURCE      Xenopus tropicalis (tropical clawed frog)\n",
      "  ORGANISM  Xenopus tropicalis\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Amphibia; Batrachia; Anura; Pipoidea; Pipidae; Xenopodinae;\n",
      "            Xenopus; Silurana.\n",
      "REFERENCE   1  (residues 1 to 275)\n",
      "  CONSRTM   NIH - Xenopus Gene Collection (XGC) project\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (??-APR-2008)\n",
      "  REMARK    NUCLEOTIDE SEQUENCE [LARGE SCALE MRNA].;\n",
      "            TISSUE=Embryo\n",
      "COMMENT     [FUNCTION] Fructose-bisphosphatase hydrolyzing\n",
      "            fructose-2,6-bisphosphate as well as fructose-1,6-bisphosphate.\n",
      "            Acts as a negative regulator of glycolysis by lowering\n",
      "            intracellular levels of fructose-2,6-bisphosphate in a\n",
      "            p53/TP53-dependent manner, resulting in the pentose phosphate\n",
      "            pathway (PPP) activation and NADPH production. Contributes to the\n",
      "            generation of reduced glutathione to cause a decrease in\n",
      "            intracellular reactive oxygen species (ROS) content, correlating\n",
      "            with its ability to protect cells from oxidative or metabolic\n",
      "            stress-induced cell death. May play a role in mitophagy inhibition.\n",
      "            {ECO:0000250|UniProtKB:Q8BZA9, ECO:0000250|UniProtKB:Q9NQ88}.\n",
      "            [CATALYTIC ACTIVITY] Beta-D-fructose 2,6-bisphosphate + H(2)O =\n",
      "            D-fructose 6-phosphate + phosphate. {ECO:0000250|UniProtKB:Q9NQ88}.\n",
      "            [SUBCELLULAR LOCATION] Cytoplasm {ECO:0000250|UniProtKB:Q8BZA9}.\n",
      "            Nucleus {ECO:0000250|UniProtKB:Q9NQ88}. Mitochondrion\n",
      "            {ECO:0000250|UniProtKB:Q8BZA9}.\n",
      "            [SIMILARITY] Belongs to the phosphoglycerate mutase family.\n",
      "            {ECO:0000305}.\n",
      "            [CAUTION] Not expected to have any kinase activity. {ECO:0000305}.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..275\n",
      "                     /organism=\"Xenopus tropicalis\"\n",
      "                     /db_xref=\"taxon:8364\"\n",
      "     gene            1..275\n",
      "                     /gene=\"tigar\"\n",
      "     Protein         1..275\n",
      "                     /product=\"Fructose-2,6-bisphosphatase TIGAR\"\n",
      "                     /EC_number=\"3.1.3.46\"\n",
      "                     /note=\"TP53-induced glycolysis and apoptosis regulator\"\n",
      "                     /UniProtKB_evidence=\"Evidence at transcript level\"\n",
      "     Region          1..275\n",
      "                     /region_name=\"Mature chain\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Fructose-2,6-bisphosphatase TIGAR.\n",
      "                     /FTId=PRO_0000363070.\"\n",
      "     Region          6..>141\n",
      "                     /region_name=\"His_Phos_1\"\n",
      "                     /note=\"Histidine phosphatase superfamily (branch 1);\n",
      "                     pfam00300\"\n",
      "                     /db_xref=\"CDD:278716\"\n",
      "     Site            11\n",
      "                     /site_type=\"active\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Tele-phosphohistidine intermediate.\n",
      "                     {ECO:0000250|UniProtKB:Q7ZVE3}.\"\n",
      "     Site            89\n",
      "                     /site_type=\"active\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Proton donor/acceptor.\n",
      "                     {ECO:0000250|UniProtKB:Q7ZVE3}.\"\n",
      "     Site            198\n",
      "                     /site_type=\"other\"\n",
      "                     /experiment=\"experimental evidence, no additional details\n",
      "                     recorded\"\n",
      "                     /note=\"Transition state stabilizer.\n",
      "                     {ECO:0000250|UniProtKB:Q7ZVE3}.\"\n",
      "ORIGIN      \n",
      "        1 marfaltivr hgetrynkek llqgqgidep lseigfkqad avgrflsnvr fthvfssdli\n",
      "       61 rakqtacaim ennkisedik iiydrrlrer kygdaegrpl selkvmakka gdqcpsytpp\n",
      "      121 ggetleqvra rakdffeylc rlvleessak eqselgasgm ggvtsadlgp fvnhnkepae\n",
      "      181 lgesrdvtvh asvllvshga ymrnwikylv edlqftfppe lkksrelpvs pntgishfiv\n",
      "      241 tvssatprkp eiqcvcinlh shlsdinadt shyqv\n",
      "//\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LOCUS       CDO52072                 992 aa            linear   PLN 30-SEP-2015\n",
      "DEFINITION  similar to Saccharomyces cerevisiae YMR205C PFK2 Beta subunit of\n",
      "            heterooctameric phosphofructokinase involved in glycolysis\n",
      "            [Galactomyces candidus].\n",
      "ACCESSION   CDO52072\n",
      "VERSION     CDO52072.1\n",
      "DBLINK      BioProject: PRJEB5752\n",
      "            BioSample: SAMEA3158493\n",
      "DBSOURCE    embl accession CCBN010000002.1\n",
      "KEYWORDS    .\n",
      "SOURCE      Galactomyces candidus (Geotrichum candidus)\n",
      "  ORGANISM  Galactomyces candidus\n",
      "            Eukaryota; Fungi; Dikarya; Ascomycota; Saccharomycotina;\n",
      "            Saccharomycetes; Saccharomycetales; Dipodascaceae; Galactomyces.\n",
      "REFERENCE   1  (residues 1 to 992)\n",
      "  AUTHORS   Casaregola,S.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (11-MAR-2014) CIRM-Levures, INRA-MICALIS,\n",
      "            INRA/AgroParisTech, 78850 Thiverval-Grignon, FRANCE\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..992\n",
      "                     /organism=\"Galactomyces candidus\"\n",
      "                     /strain=\"CLIB 918\"\n",
      "                     /db_xref=\"taxon:1173061\"\n",
      "                     /note=\"scaffold02\"\n",
      "     Protein         1..992\n",
      "                     /product=\"similar to Saccharomyces cerevisiae YMR205C PFK2\n",
      "                     Beta subunit of heterooctameric phosphofructokinase\n",
      "                     involved in glycolysis\"\n",
      "     CDS             1..992\n",
      "                     /locus_tag=\"BN980_GECA02s05433g\"\n",
      "                     /coded_by=\"CCBN010000002.1:1182192..1185170\"\n",
      "                     /note=\"annotator: Djamila Onesime (Micalis)\"\n",
      "ORIGIN      \n",
      "        1 mssypnlvks lscfsvvtsd stlfertief ykqfgfihvh tfdhsnpess feksshssht\n",
      "       61 lesikeiwln alpdpeyghe dktddtgvti kirltstgpt aeqsiekits flseshkkld\n",
      "      121 wralapsiil qttdletvli tlekfslldn lqpyiqprpt svspvevvgl dplgnvigft\n",
      "      181 trenpfsvvd kpfistrnkq iatsgsvsgl tsraslnsnn pdignvlsrk riaimtsggd\n",
      "      241 apgmnpavra vvraaiyrgc eayaimegye gllrggdyik emgwddvrgw lsiggthigt\n",
      "      301 arcmgfkerd grrkaaknmi lngidalivc ggdgsltgad kfreewpsll eelvemnelt\n",
      "      361 aeqvkphehl yicglvgsid ndmastnvti gaysslqric eavdyidata qshsrafvie\n",
      "      421 vmgrhcgwla lmagiatgad ylfipeqpsh pgewqarmtk vvsrhrangk rksivivaeg\n",
      "      481 aidtdlnpit snmvkdtlvd lgldtrittl ghiqrggsav afdrllatmq gvdavdavls\n",
      "      541 atpdtpspmi givenkivrr nlveavaatk evaaaikakd fdramslrdt efaehynnfl\n",
      "      601 tinyadreta kmpsekplni avinigapag gmnsairama aycysrghqp faihngwsgl\n",
      "      661 arhesvhpln wleveewmir ggceigtnrs lpdtdigmva yyfqkykfdg liiiggfeaf\n",
      "      721 aslqqleaar asypafripm vcipatisnn vpgteyslgt dtclnslvny cdvikqsasa\n",
      "      781 srrrvfvvev qggnsgyvas yaglvtgaqa vylpedgisl rqleadidsl kesfendggk\n",
      "      841 nragklilrn ekssstlste iitniirdea kgrfdarsai pghvqqggnp spmdrvratr\n",
      "      901 faikacefie rsvdhisrep etagvsvmdr larvneerkt aavlgvrssk lvfspvdqvw\n",
      "      961 afetehkerr pkkdhwklma rtgdrlvnrv kv\n",
      "//\n",
      "\n",
      "\n",
      "LOCUS       CDO53591                 975 aa            linear   PLN 30-SEP-2015\n",
      "DEFINITION  similar to Saccharomyces cerevisiae YMR205C PFK2 Beta subunit of\n",
      "            heterooctameric phosphofructokinase involved in glycolysis\n",
      "            [Galactomyces candidus].\n",
      "ACCESSION   CDO53591\n",
      "VERSION     CDO53591.1\n",
      "DBLINK      BioProject: PRJEB5752\n",
      "            BioSample: SAMEA3158493\n",
      "DBSOURCE    embl accession CCBN010000005.1\n",
      "KEYWORDS    .\n",
      "SOURCE      Galactomyces candidus (Geotrichum candidus)\n",
      "  ORGANISM  Galactomyces candidus\n",
      "            Eukaryota; Fungi; Dikarya; Ascomycota; Saccharomycotina;\n",
      "            Saccharomycetes; Saccharomycetales; Dipodascaceae; Galactomyces.\n",
      "REFERENCE   1  (residues 1 to 975)\n",
      "  AUTHORS   Casaregola,S.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (11-MAR-2014) CIRM-Levures, INRA-MICALIS,\n",
      "            INRA/AgroParisTech, 78850 Thiverval-Grignon, FRANCE\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..975\n",
      "                     /organism=\"Galactomyces candidus\"\n",
      "                     /strain=\"CLIB 918\"\n",
      "                     /db_xref=\"taxon:1173061\"\n",
      "                     /note=\"scaffold05\"\n",
      "     Protein         1..975\n",
      "                     /product=\"similar to Saccharomyces cerevisiae YMR205C PFK2\n",
      "                     Beta subunit of heterooctameric phosphofructokinase\n",
      "                     involved in glycolysis\"\n",
      "     CDS             1..975\n",
      "                     /locus_tag=\"BN980_GECA05s04113g\"\n",
      "                     /coded_by=\"CCBN010000005.1:903012..905939\"\n",
      "                     /note=\"annotator: Djamila Onesime (Micalis)\"\n",
      "ORIGIN      \n",
      "        1 msqpkliksl sslsvvtdev alyeqtiafy eqlglvvvhs fdysnpvssr ersvqsshtl\n",
      "       61 eslketwlna mpdanfdhsd avddfgitlk vrltstsavk ptnsstalpe nvdwrsvape\n",
      "      121 vvfrtddlda ciailkknna kfqphptevs pvevvaqdpl gnliaftmrk npfsvvdkpf\n",
      "      181 vsttgrevpl srssseeptm spqytnrdpn vpdafrrrri avmtsggdap gmnaavravv\n",
      "      241 raaiyrgcea yailegyegl lqggkfikem gwddvrgwla tggthigtar cagfrersgr\n",
      "      301 lkaaknmiin gidalivcgg dgsltgadkf reewpsllke lvetgemtad qvkphehlyi\n",
      "      361 cglvgsidnd mastdttiga ysslericea vdyidataqs hsrafvievm grhcgwlalm\n",
      "      421 agiatgadfi fipeqpphdq ewqarmskiv krhrangkrk tivivaegai nsnldtitpe\n",
      "      481 qvkdvlvdlg ldtrittlgh vqrggnavaa drllatlqgv davdavlsat pdtpspmigi\n",
      "      541 ienkivrrnl veavaatqev akaikakdfa ramslrdtef aehfqnflti nsadrespkk\n",
      "      601 paekplniav inigapaggm nsairglaay cfsrghkpya ihngwtglar hesvrelswl\n",
      "      661 evedwmirgg ceigtnrnlp ntdigmiayy fqkykfdgli iiggfeafas lhqleearas\n",
      "      721 ypafripmvc ipatisnnvp gteyslgsdt clnslvnycd viklsasasr rrvfvvevqg\n",
      "      781 gnsgyvaaya glatgahavy lpeegislrq leadidnike afeddqgknr agklilrnek\n",
      "      841 ssrtlsteii vniirdeakg rfdaraaipg hvqqggnpsp mdrvratrfa vqacefieeh\n",
      "      901 qmpvetslin klvsreelrk tavvlgvrss klvyspidql wahetehker rpkkihwmhl\n",
      "      961 aqvgdrlvgr nklsr\n",
      "//\n",
      "\n",
      "\n",
      "LOCUS       CAY67878                 918 aa            linear   PLN 27-FEB-2015\n",
      "DEFINITION  Beta subunit of heterooctameric phosphofructokinase involved in\n",
      "            glycolysis [Komagataella phaffii GS115].\n",
      "ACCESSION   CAY67878\n",
      "VERSION     CAY67878.1\n",
      "DBLINK      BioProject: PRJEA37871\n",
      "            BioSample: SAMEA2272385\n",
      "DBSOURCE    embl accession FN392319.1\n",
      "KEYWORDS    .\n",
      "SOURCE      Komagataella phaffii GS115\n",
      "  ORGANISM  Komagataella phaffii GS115\n",
      "            Eukaryota; Fungi; Dikarya; Ascomycota; Saccharomycotina;\n",
      "            Saccharomycetes; Saccharomycetales; Phaffomycetaceae; Komagataella.\n",
      "REFERENCE   1\n",
      "  AUTHORS   De Schutter,K., Lin,Y.C., Tiels,P., Van Hecke,A., Glinka,S.,\n",
      "            Weber-Lehmann,J., Rouze,P., Van de Peer,Y. and Callewaert,N.\n",
      "  TITLE     Genome sequence of the recombinant protein production host Pichia\n",
      "            pastoris\n",
      "  JOURNAL   Nat. Biotechnol. 27 (6), 561-566 (2009)\n",
      "   PUBMED   19465926\n",
      "REFERENCE   2  (residues 1 to 918)\n",
      "  AUTHORS   Callewaert,N.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (05-MAY-2009) Callewaert N., Department for Molecular\n",
      "            Biomedical Research, Flanders Institute for Biotechnology (VIB) /\n",
      "            Gent University, Technologiepark 927, Ghent, 9052, BELGIUM\n",
      "COMMENT     Contact: Nico Callewaert\n",
      "            Unit for Molecular Glycobiology\n",
      "            Laboratory for Protein Biochemistry and Biomolecular Engineering\n",
      "            Department of Biochemistry, Physiology and Microbiology, UGent VIB\n",
      "            Department for Molecular Biomedical Research, UGent Technologiepark\n",
      "            927, B-9052 Gent, Belgium\n",
      "            Tel: (32)93313630\n",
      "            Fax: (32)93313609\n",
      "            Email: nico.callewaert@dmbr.vib-UGent.be\n",
      "            ----------- sequencing/assembly\n",
      "            Unit for Molecular Glycobiology\n",
      "            Laboratory for Protein Biochemistry and Biomolecular Engineering\n",
      "            Department of Biochemistry, Physiology and Microbiology, UGent VIB\n",
      "            Department for Molecular Biomedical Research, UGent -----------\n",
      "            annotation\n",
      "            Unit for Molecular Glycobiology\n",
      "            Laboratory for Protein Biochemistry and Biomolecular Engineering\n",
      "            Department of Biochemistry, Physiology and Microbiology, UGent VIB\n",
      "            Department for Molecular Biomedical Research, UGent\n",
      "            \n",
      "            Laboratory for Bioinformatics and Evolutionary Genomics VIB\n",
      "            Department of Plant Systems Biology, UGent\n",
      "            Web site: http://bioinformatics.psb.ugent.be/\n",
      "            Bioinformatics Online Genome Annotation System (BOGAS):\n",
      "            http://bioinformatics.psb.ugent.be/webtools/bogas/ -----------\n",
      "            * NOTE: finishing of chromosomes in process\n",
      "            * Final curation of the annotation in progress.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..918\n",
      "                     /organism=\"Komagataella phaffii GS115\"\n",
      "                     /strain=\"GS115\"\n",
      "                     /db_xref=\"taxon:644223\"\n",
      "                     /chromosome=\"1\"\n",
      "     Protein         1..918\n",
      "                     /product=\"Beta subunit of heterooctameric\n",
      "                     phosphofructokinase involved in glycolysis\"\n",
      "     Region          13..143\n",
      "                     /region_name=\"Glo_EDI_BRP_like\"\n",
      "                     /note=\"This domain superfamily is found in a variety of\n",
      "                     structurally related metalloproteins, including the type I\n",
      "                     extradiol dioxygenases, glyoxalase I and a group of\n",
      "                     antibiotic resistance proteins; cd06587\"\n",
      "                     /db_xref=\"CDD:211348\"\n",
      "     Site            order(16,48,59,64,79,132,140,142)\n",
      "                     /site_type=\"active\"\n",
      "                     /db_xref=\"CDD:211348\"\n",
      "     Region          180..913\n",
      "                     /region_name=\"PFK\"\n",
      "                     /note=\"Phosphofructokinase, a key regulatory enzyme in\n",
      "                     glycolysis, catalyzes the phosphorylation of\n",
      "                     fructose-6-phosphate to fructose-1,6-biphosphate. The\n",
      "                     members belong to PFK family that includes ATP- and\n",
      "                     pyrophosphate (PPi)- dependent phosphofructokinases;\n",
      "                     cl00204\"\n",
      "                     /db_xref=\"CDD:294137\"\n",
      "     Region          182..917\n",
      "                     /region_name=\"6PF1K_euk\"\n",
      "                     /note=\"6-phosphofructokinase, eukaryotic type; TIGR02478\"\n",
      "                     /db_xref=\"CDD:274152\"\n",
      "     Site            order(191,221,255,286..288,290..291,331,333,335,375..377,\n",
      "                     432,466,469)\n",
      "                     /site_type=\"active\"\n",
      "                     /db_xref=\"CDD:238216\"\n",
      "     Site            order(191,221,255,286..288,290..291)\n",
      "                     /site_type=\"other\"\n",
      "                     /note=\"ADP/pyrophosphate binding site [chemical binding]\"\n",
      "                     /db_xref=\"CDD:238216\"\n",
      "     Site            order(201,205,237,242,245,341,353,357,360,388..389,391,\n",
      "                     423,478..479,483,490,505,540)\n",
      "                     /site_type=\"other\"\n",
      "                     /note=\"dimerization interface [polypeptide binding]\"\n",
      "                     /db_xref=\"CDD:238216\"\n",
      "     Site            order(201,205,237..238,241..242,360,391,393,421,423..425)\n",
      "                     /site_type=\"other\"\n",
      "                     /note=\"allosteric effector site\"\n",
      "                     /db_xref=\"CDD:238216\"\n",
      "     Site            order(331,333,335,368,375..377,432,460,466,469)\n",
      "                     /site_type=\"other\"\n",
      "                     /note=\"fructose-1,6-bisphosphate binding site\"\n",
      "                     /db_xref=\"CDD:238216\"\n",
      "     CDS             1..918\n",
      "                     /locus_tag=\"PAS_chr1-4_0047\"\n",
      "                     /coded_by=\"join(FN392319.1:1496126..1498556,\n",
      "                     FN392319.1:1498627..1498952)\"\n",
      "                     /db_xref=\"EnsemblGenomes-Gn:PAS_chr1-4_0047\"\n",
      "                     /db_xref=\"EnsemblGenomes-Tr:CAY67878\"\n",
      "                     /db_xref=\"GOA:C4QXA5\"\n",
      "                     /db_xref=\"InterPro:IPR000023\"\n",
      "                     /db_xref=\"InterPro:IPR009161\"\n",
      "                     /db_xref=\"InterPro:IPR015912\"\n",
      "                     /db_xref=\"InterPro:IPR022953\"\n",
      "                     /db_xref=\"InterPro:IPR029068\"\n",
      "ORIGIN      \n",
      "        1 mpdaslfngt sfitlfapni slfqasidfy tdrlgfaike tsnqklvwlq leedsnnvsi\n",
      "       61 qllldpehaa svsqidqnir nltrslyrkd wrsiqsniaf kssslsklvk llkdgghpvq\n",
      "      121 qspneispfe vytldplgsl igfsgfknpf avnersllpk vseekayrte ddseklftpi\n",
      "      181 rktigvmtsg gdspgmnpfv ravvragiyk gckvfciheg yeglvrggek yiketqwhdv\n",
      "      241 rgwlveggtn igtarckefr ersgrlkack nmidmgidal ivcggdgslt gadrfrsewp\n",
      "      301 slieellqte risqqqfety qnlnicgavg sidndmsstd atigafssld ricraidyid\n",
      "      361 atanshsraf ivevmgrhcg wlgllaglat sadyilipek passrewqdq mcdivskhra\n",
      "      421 rgkrktiviv aegaisndls piscdqvkdv lvnrlgldtr vttlghvqrg gtavafdriy\n",
      "      481 atlqgveavn avlecnadtp spmiaikedq itrvplvdav eltqqvaksi esrnfkrais\n",
      "      541 lrdsefvehm knfistnsad hvppslplek rkkvaiinvg apaggmnsav ysmatycmsr\n",
      "      601 ghvpyaihng fsglarhesv rsinwldieg wgslggseig tnrtlpndad igmiayffek\n",
      "      661 ygfdglilvg gfeafislhq lerarinyps lriplvlipa tisnnvpgte yslgsdtcln\n",
      "      721 sfmeycdvik qsaaatrnrv fvvevqggns gyiathaqla cgaqisyvpe egislaqlem\n",
      "      781 dinslkesfa ndqgktkfrq tdfevgecie aipghvqqgg ipspmdrvra srfairavsf\n",
      "      841 iekhsdkcqa fknsisfrqt deitstavvl gihksqlrft pirqlydfes dvprrmrkni\n",
      "      901 fwsnvreisd mlsgrtsl\n",
      "//\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LOCUS       3PP7_B                   498 aa            linear   INV 10-OCT-2012\n",
      "DEFINITION  Chain B, Pyruvate Kinase.\n",
      "ACCESSION   3PP7_B\n",
      "VERSION     3PP7_B\n",
      "DBSOURCE    pdb: molecule 3PP7, chain 66, release Sep 21, 2011;\n",
      "            deposition: Nov 24, 2010;\n",
      "            class: TransferaseTRANSFERASE INHIBITOR;\n",
      "            source: Mmdb_id: 87393, Pdb_id 1: 3PP7;\n",
      "            Exp. method: X-Ray Diffraction.\n",
      "KEYWORDS    .\n",
      "SOURCE      Leishmania mexicana\n",
      "  ORGANISM  Leishmania mexicana\n",
      "            Eukaryota; Euglenozoa; Kinetoplastida; Trypanosomatidae;\n",
      "            Leishmaniinae; Leishmania.\n",
      "REFERENCE   1  (residues 1 to 498)\n",
      "  AUTHORS   Morgan,H.P., McNae,I.W., Nowicki,M.W., Zhong,W., Michels,P.A.,\n",
      "            Auld,D.S., Fothergill-Gilmore,L.A. and Walkinshaw,M.D.\n",
      "  TITLE     The trypanocidal drug suramin and other trypan blue mimetics are\n",
      "            inhibitors of pyruvate kinases and bind to the adenosine site\n",
      "  JOURNAL   J. Biol. Chem. 286 (36), 31232-31240 (2011)\n",
      "   PUBMED   21733839\n",
      "REFERENCE   2  (residues 1 to 498)\n",
      "  AUTHORS   Morgan,H.P., Auld,D.S., Mcnae,I.W., Nowicki,M.W., Michels,P.A.M.,\n",
      "            Fothergill-Gilmore,L.A. and Walkinshaw,M.D.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (24-NOV-2010)\n",
      "COMMENT     Crystal Structure Of Leishmania Mexicana Pyruvate Kinase In Complex\n",
      "            With The Drug Suramin, An Inhibitor Of Glycolysis.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..498\n",
      "                     /organism=\"Leishmania mexicana\"\n",
      "                     /db_xref=\"taxon:5665\"\n",
      "     Region          join(1..87,186..358)\n",
      "                     /region_name=\"Domain 4\"\n",
      "                     /note=\"NCBI Domains\"\n",
      "     Region          18..497\n",
      "                     /region_name=\"Pyruvate_Kinase\"\n",
      "                     /note=\"Pyruvate kinase (PK): Large allosteric enzyme that\n",
      "                     regulates glycolysis through binding of the substrate,\n",
      "                     phosphoenolpyruvate, and one or more allosteric effectors.\n",
      "                     Like other allosteric enzymes, PK has a high substrate\n",
      "                     affinity R state and a low...; cd00288\"\n",
      "                     /db_xref=\"CDD:238178\"\n",
      "     Site            order(20..21,77,79,87..90,177..178,184,187..191,206,214,\n",
      "                     221,232,268,286..287,291,323,325,354,356..357,411,413,415,\n",
      "                     426,429..430,432,434..436)\n",
      "                     /site_type=\"other\"\n",
      "                     /note=\"domain interfaces\"\n",
      "                     /db_xref=\"CDD:238178\"\n",
      "     SecStr          21..27\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 23\"\n",
      "     SecStr          34..43\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 14\"\n",
      "     Region          45..498\n",
      "                     /region_name=\"PTZ00300\"\n",
      "                     /note=\"pyruvate kinase; Provisional\"\n",
      "                     /db_xref=\"CDD:140321\"\n",
      "     SecStr          46..53\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 24\"\n",
      "     Site            order(49,51,83,212,238,240,264,296)\n",
      "                     /site_type=\"active\"\n",
      "                     /db_xref=\"CDD:238178\"\n",
      "     Het             join(bond(51),bond(53),bond(83),bond(84),bond(83))\n",
      "                     /heterogen=\"(K,1005)\"\n",
      "     SecStr          57..74\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 15\"\n",
      "     SecStr          78..85\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 25\"\n",
      "     Region          88..185\n",
      "                     /region_name=\"Domain 5\"\n",
      "                     /note=\"NCBI Domains\"\n",
      "     SecStr          89..92\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 26\"\n",
      "     SecStr          97..102\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 27\"\n",
      "     SecStr          103..111\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 28\"\n",
      "     SecStr          122..128\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 29\"\n",
      "     SecStr          139..144\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 30\"\n",
      "     SecStr          147..156\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 31\"\n",
      "     SecStr          159..165\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 32\"\n",
      "     SecStr          166..170\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 33\"\n",
      "     SecStr          176..179\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 34\"\n",
      "     SecStr          191..203\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 16\"\n",
      "     SecStr          206..212\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 35\"\n",
      "     SecStr          218..226\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 17\"\n",
      "     SecStr          233..240\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 36\"\n",
      "     SecStr          242..254\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 18\"\n",
      "     SecStr          256..263\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 37\"\n",
      "     SecStr          271..288\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 19\"\n",
      "     SecStr          290..297\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 38\"\n",
      "     SecStr          310..322\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 20\"\n",
      "     SecStr          325..330\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 39\"\n",
      "     SecStr          339..356\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 21\"\n",
      "     Het             join(bond(354),bond(357),bond(359),bond(359),bond(359))\n",
      "                     /heterogen=\"(K,1006)\"\n",
      "     Region          359..498\n",
      "                     /region_name=\"Domain 6\"\n",
      "                     /note=\"NCBI Domains\"\n",
      "     SecStr          360..369\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 22\"\n",
      "     SecStr          376..391\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 23\"\n",
      "     SecStr          394..401\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 40\"\n",
      "     SecStr          404..412\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 24\"\n",
      "     SecStr          416..423\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 41\"\n",
      "     SecStr          424..431\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 25\"\n",
      "     SecStr          436..442\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 42\"\n",
      "     SecStr          454..467\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 26\"\n",
      "     SecStr          474..481\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 43\"\n",
      "     SecStr          490..497\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 44\"\n",
      "ORIGIN      \n",
      "        1 sqlahnltls ifdpvanyra ariictigps tqsvealkgl iqsgmsvarm nfshgsheyh\n",
      "       61 qttinnvrqa aaelgvniai aldtkgpeir tgqfvggdav mergatcyvt tdpafadkgt\n",
      "      121 kdkfyidyqn lskvvrpgny iyiddgilil qvqshedeqt lectvtnsht isdrrgvnlp\n",
      "      181 gcdvdlpavs akdrvdlqfg veqgvdmifa sfirsaeqvg dvrkalgpkg rdimiickie\n",
      "      241 nhqgvqnids iieesdgimv argdlgveip aekvvvaqki liskcnvagk pvicatqmle\n",
      "      301 smtynprptr aevsdvanav fngadcvmls getakgkypn evvqymaric leaqsalney\n",
      "      361 vffnsikklq hipmsadeav cssavnsvye tkakamvvls ntgrsarlva kyrpncpivc\n",
      "      421 vttrlqtcrq lnitqgvesv ffdadklghd egkehrvaag vefakskgyv qtgdycvvih\n",
      "      481 adhkvkgyan qtrillve\n",
      "//\n",
      "\n",
      "\n",
      "LOCUS       3PP7_A                   498 aa            linear   INV 10-OCT-2012\n",
      "DEFINITION  Chain A, Pyruvate Kinase.\n",
      "ACCESSION   3PP7_A\n",
      "VERSION     3PP7_A\n",
      "DBSOURCE    pdb: molecule 3PP7, chain 65, release Sep 21, 2011;\n",
      "            deposition: Nov 24, 2010;\n",
      "            class: TransferaseTRANSFERASE INHIBITOR;\n",
      "            source: Mmdb_id: 87393, Pdb_id 1: 3PP7;\n",
      "            Exp. method: X-Ray Diffraction.\n",
      "KEYWORDS    .\n",
      "SOURCE      Leishmania mexicana\n",
      "  ORGANISM  Leishmania mexicana\n",
      "            Eukaryota; Euglenozoa; Kinetoplastida; Trypanosomatidae;\n",
      "            Leishmaniinae; Leishmania.\n",
      "REFERENCE   1  (residues 1 to 498)\n",
      "  AUTHORS   Morgan,H.P., McNae,I.W., Nowicki,M.W., Zhong,W., Michels,P.A.,\n",
      "            Auld,D.S., Fothergill-Gilmore,L.A. and Walkinshaw,M.D.\n",
      "  TITLE     The trypanocidal drug suramin and other trypan blue mimetics are\n",
      "            inhibitors of pyruvate kinases and bind to the adenosine site\n",
      "  JOURNAL   J. Biol. Chem. 286 (36), 31232-31240 (2011)\n",
      "   PUBMED   21733839\n",
      "REFERENCE   2  (residues 1 to 498)\n",
      "  AUTHORS   Morgan,H.P., Auld,D.S., Mcnae,I.W., Nowicki,M.W., Michels,P.A.M.,\n",
      "            Fothergill-Gilmore,L.A. and Walkinshaw,M.D.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (24-NOV-2010)\n",
      "COMMENT     Crystal Structure Of Leishmania Mexicana Pyruvate Kinase In Complex\n",
      "            With The Drug Suramin, An Inhibitor Of Glycolysis.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..498\n",
      "                     /organism=\"Leishmania mexicana\"\n",
      "                     /db_xref=\"taxon:5665\"\n",
      "     Region          join(1..87,186..358)\n",
      "                     /region_name=\"Domain 1\"\n",
      "                     /note=\"NCBI Domains\"\n",
      "     Region          18..497\n",
      "                     /region_name=\"Pyruvate_Kinase\"\n",
      "                     /note=\"Pyruvate kinase (PK): Large allosteric enzyme that\n",
      "                     regulates glycolysis through binding of the substrate,\n",
      "                     phosphoenolpyruvate, and one or more allosteric effectors.\n",
      "                     Like other allosteric enzymes, PK has a high substrate\n",
      "                     affinity R state and a low...; cd00288\"\n",
      "                     /db_xref=\"CDD:238178\"\n",
      "     Site            order(20..21,77,79,87..90,177..178,184,187..191,206,214,\n",
      "                     221,232,268,286..287,291,323,325,354,356..357,411,413,415,\n",
      "                     426,429..430,432,434..436)\n",
      "                     /site_type=\"other\"\n",
      "                     /note=\"domain interfaces\"\n",
      "                     /db_xref=\"CDD:238178\"\n",
      "     SecStr          21..27\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 1\"\n",
      "     SecStr          34..43\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 1\"\n",
      "     Region          45..498\n",
      "                     /region_name=\"PTZ00300\"\n",
      "                     /note=\"pyruvate kinase; Provisional\"\n",
      "                     /db_xref=\"CDD:140321\"\n",
      "     SecStr          46..53\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 2\"\n",
      "     Site            order(49,51,83,212,238,240,264,296)\n",
      "                     /site_type=\"active\"\n",
      "                     /db_xref=\"CDD:238178\"\n",
      "     Het             join(bond(51),bond(53),bond(83),bond(84),bond(238))\n",
      "                     /heterogen=\"(K,1003)\"\n",
      "     SecStr          57..74\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 2\"\n",
      "     SecStr          78..85\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 3\"\n",
      "     Region          88..185\n",
      "                     /region_name=\"Domain 2\"\n",
      "                     /note=\"NCBI Domains\"\n",
      "     SecStr          89..92\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 4\"\n",
      "     SecStr          97..102\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 5\"\n",
      "     SecStr          103..111\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 6\"\n",
      "     SecStr          122..128\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 7\"\n",
      "     SecStr          139..144\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 8\"\n",
      "     SecStr          147..156\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 9\"\n",
      "     SecStr          159..165\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 10\"\n",
      "     SecStr          166..170\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 11\"\n",
      "     SecStr          176..179\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 12\"\n",
      "     SecStr          191..203\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 3\"\n",
      "     SecStr          206..212\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 13\"\n",
      "     SecStr          218..226\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 4\"\n",
      "     SecStr          233..240\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 14\"\n",
      "     SecStr          242..254\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 5\"\n",
      "     SecStr          256..262\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 15\"\n",
      "     SecStr          271..288\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 6\"\n",
      "     SecStr          290..296\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 16\"\n",
      "     SecStr          310..322\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 7\"\n",
      "     SecStr          325..330\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 17\"\n",
      "     SecStr          339..356\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 8\"\n",
      "     Het             join(bond(354),bond(357),bond(359),bond(359),bond(359))\n",
      "                     /heterogen=\"(K,1002)\"\n",
      "     Region          359..498\n",
      "                     /region_name=\"Domain 3\"\n",
      "                     /note=\"NCBI Domains\"\n",
      "     SecStr          360..369\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 9\"\n",
      "     SecStr          376..391\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 10\"\n",
      "     SecStr          394..401\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 18\"\n",
      "     SecStr          404..412\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 11\"\n",
      "     SecStr          416..423\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 19\"\n",
      "     SecStr          424..431\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 12\"\n",
      "     SecStr          436..442\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 20\"\n",
      "     SecStr          454..467\n",
      "                     /sec_str_type=\"helix\"\n",
      "                     /note=\"helix 13\"\n",
      "     SecStr          474..480\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 21\"\n",
      "     SecStr          490..497\n",
      "                     /sec_str_type=\"sheet\"\n",
      "                     /note=\"strand 22\"\n",
      "ORIGIN      \n",
      "        1 sqlahnltls ifdpvanyra ariictigps tqsvealkgl iqsgmsvarm nfshgsheyh\n",
      "       61 qttinnvrqa aaelgvniai aldtkgpeir tgqfvggdav mergatcyvt tdpafadkgt\n",
      "      121 kdkfyidyqn lskvvrpgny iyiddgilil qvqshedeqt lectvtnsht isdrrgvnlp\n",
      "      181 gcdvdlpavs akdrvdlqfg veqgvdmifa sfirsaeqvg dvrkalgpkg rdimiickie\n",
      "      241 nhqgvqnids iieesdgimv argdlgveip aekvvvaqki liskcnvagk pvicatqmle\n",
      "      301 smtynprptr aevsdvanav fngadcvmls getakgkypn evvqymaric leaqsalney\n",
      "      361 vffnsikklq hipmsadeav cssavnsvye tkakamvvls ntgrsarlva kyrpncpivc\n",
      "      421 vttrlqtcrq lnitqgvesv ffdadklghd egkehrvaag vefakskgyv qtgdycvvih\n",
      "      481 adhkvkgyan qtrillve\n",
      "//\n",
      "\n",
      "\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "LOCUS       PKK30041                 266 aa            linear   VRT 13-DEC-2017\n",
      "DEFINITION  TP53 induced glycolysis regulatory phosphatase [Columba livia].\n",
      "ACCESSION   PKK30041\n",
      "VERSION     PKK30041.1\n",
      "DBLINK      BioProject: PRJNA167554\n",
      "            BioSample: SAMN02981415\n",
      "DBSOURCE    accession AKCR02000008.1\n",
      "KEYWORDS    .\n",
      "SOURCE      Columba livia (rock pigeon)\n",
      "  ORGANISM  Columba livia\n",
      "            Eukaryota; Metazoa; Chordata; Craniata; Vertebrata; Euteleostomi;\n",
      "            Archelosauria; Archosauria; Dinosauria; Saurischia; Theropoda;\n",
      "            Coelurosauria; Aves; Neognathae; Columbiformes; Columbidae;\n",
      "            Columba.\n",
      "REFERENCE   1  (residues 1 to 266)\n",
      "  AUTHORS   Shapiro,M.D., Kronenberg,Z., Li,C., Domyan,E.T., Pan,H.,\n",
      "            Campbell,M., Tan,H., Huff,C.D., Hu,H., Vickrey,A.I., Nielsen,S.C.,\n",
      "            Stringham,S.A., Hu,H., Willerslev,E., Gilbert,M.T., Yandell,M.,\n",
      "            Zhang,G. and Wang,J.\n",
      "  TITLE     Genomic diversity and evolution of the head crest in the rock\n",
      "            pigeon\n",
      "  JOURNAL   Science 339 (6123), 1063-1067 (2013)\n",
      "   PUBMED   23371554\n",
      "REFERENCE   2  (residues 1 to 266)\n",
      "  AUTHORS   Holt,C., Campbell,M., Edelman,N., Keays,D., Kapusta,A., Domyan,E.,\n",
      "            Suh,A., Carleton,J., Warren,W., Yandell,M., Gilbert,M.T. and\n",
      "            Shapiro,M.D.\n",
      "  TITLE     Improved genome assembly and annotation for the rock pigeon\n",
      "            (Columba livia)\n",
      "  JOURNAL   Unpublished\n",
      "REFERENCE   3  (residues 1 to 266)\n",
      "  AUTHORS   Zhang,G. and Li,C.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (18-JUN-2012) Bioinformatics, Beijing Genomics Institute,\n",
      "            Shenzhen, 2nd Office Building, Beishan Industrial Zone, Beishan\n",
      "            Road, Yantian District, ShenZhen, GuangDong 518083, China\n",
      "REFERENCE   4  (residues 1 to 266)\n",
      "  AUTHORS   Shapiro,M.D.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (31-AUG-2017) Human Genetics, University of Utah, 15 N.\n",
      "            2030 E. Bldg. #533, Salt Lake City, UT 84112, USA\n",
      "COMMENT     Submitter provided annotation based on NCBI Columba livia\n",
      "            Annotation Release 101\n",
      "            https://www.ncbi.nlm.nih.gov/genome/annotation_euk/Columba_livia/10\n",
      "            1/\n",
      "            \n",
      "            ##Genome-Assembly-Data-START##\n",
      "            Assembly Date          :: 25-AUG-2015\n",
      "            Assembly Method        :: SOAPdenovo v. 2.0; HiRise v. AUG-2015\n",
      "            Assembly Name          :: Cliv_2.1\n",
      "            Expected Final Version :: no\n",
      "            Genome Coverage        :: 60.0x\n",
      "            Sequencing Technology  :: Illumina\n",
      "            ##Genome-Assembly-Data-END##\n",
      "            Method: conceptual translation.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..266\n",
      "                     /organism=\"Columba livia\"\n",
      "                     /db_xref=\"taxon:8932\"\n",
      "                     /sex=\"male\"\n",
      "                     /tissue_type=\"blood\"\n",
      "                     /country=\"Denmark: Vordingborg\"\n",
      "                     /collection_date=\"13-Jun-2010\"\n",
      "                     /note=\"bred by Anders Christiansen, Hans Ove Christiansen,\n",
      "                     and certified by Danmarks Racedueforeninger;\n",
      "                     breed: Danish Tumbler\"\n",
      "     Protein         1..266\n",
      "                     /product=\"TP53 induced glycolysis regulatory phosphatase\"\n",
      "     CDS             1..266\n",
      "                     /gene=\"TIGAR\"\n",
      "                     /locus_tag=\"A306_00004426\"\n",
      "                     /coded_by=\"join(AKCR02000008.1:8754123,\n",
      "                     AKCR02000008.1:8756228..8756271,\n",
      "                     AKCR02000008.1:8758484..8758594,\n",
      "                     AKCR02000008.1:8762090..8762167,\n",
      "                     AKCR02000008.1:8764704..8764814,\n",
      "                     AKCR02000008.1:8765288..8765743)\"\n",
      "                     /inference=\"alignment:Exonerate:2.2.0:RefSeq:XP_005503097.\n",
      "                     1\"\n",
      "                     /db_xref=\"GeneID:102092829\"\n",
      "ORIGIN      \n",
      "        1 mekqdstktr yckvsvkewx xxxfrqadaa gvflsnvkft hvfssdllra kqtaatilgk\n",
      "       61 nkfckdleik ydarlrerky gvaegrpltd lkamakaage qcpsftpsgg etldevrera\n",
      "      121 rdffeflcql avgleqkeqd mhgaasssgt seeqlvfpwt nhcseaelss dnsgaskild\n",
      "      181 anilvvshga ymrnwigyfv sdlnctlptn ltksqlssvs pntgvshfiv klengnllkp\n",
      "      241 eitcvclnqd shlvdvgaec vaskvf\n",
      "//\n",
      "\n",
      "\n",
      "LOCUS       XP_002491345             333 aa            linear   PLN 11-OCT-2017\n",
      "DEFINITION  Glyceraldehyde-3-phosphate dehydrogenase, isozyme 3, involved in\n",
      "            glycolysis and gluconeogenesis [Komagataella phaffii GS115].\n",
      "ACCESSION   XP_002491345\n",
      "VERSION     XP_002491345.1\n",
      "DBLINK      BioProject: PRJNA39439\n",
      "            BioSample: SAMEA2272385\n",
      "DBSOURCE    REFSEQ: accession XM_002491300.1\n",
      "KEYWORDS    RefSeq.\n",
      "SOURCE      Komagataella phaffii GS115\n",
      "  ORGANISM  Komagataella phaffii GS115\n",
      "            Eukaryota; Fungi; Dikarya; Ascomycota; Saccharomycotina;\n",
      "            Saccharomycetes; Saccharomycetales; Phaffomycetaceae; Komagataella.\n",
      "REFERENCE   1\n",
      "  AUTHORS   De Schutter,K., Lin,Y.C., Tiels,P., Van Hecke,A., Glinka,S.,\n",
      "            Weber-Lehmann,J., Rouze,P., Van de Peer,Y. and Callewaert,N.\n",
      "  TITLE     Genome sequence of the recombinant protein production host Pichia\n",
      "            pastoris\n",
      "  JOURNAL   Nat. Biotechnol. 27 (6), 561-566 (2009)\n",
      "   PUBMED   19465926\n",
      "REFERENCE   2  (residues 1 to 333)\n",
      "  CONSRTM   NCBI Genome Project\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (11-OCT-2017) National Center for Biotechnology\n",
      "            Information, NIH, Bethesda, MD 20894, USA\n",
      "REFERENCE   3  (residues 1 to 333)\n",
      "  AUTHORS   Callewaert,N.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (05-MAY-2009) Callewaert N., Department for Molecular\n",
      "            Biomedical Research, Flanders Institute for Biotechnology (VIB) /\n",
      "            Gent University, Technologiepark 927, Ghent, 9052, BELGIUM\n",
      "COMMENT     PROVISIONAL REFSEQ: This record has not yet been subject to final\n",
      "            NCBI review. The reference sequence is identical to CAY69065.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..333\n",
      "                     /organism=\"Komagataella phaffii GS115\"\n",
      "                     /strain=\"GS115\"\n",
      "                     /db_xref=\"taxon:644223\"\n",
      "                     /chromosome=\"2\"\n",
      "     Protein         1..333\n",
      "                     /product=\"Glyceraldehyde-3-phosphate dehydrogenase,\n",
      "                     isozyme 3, involved in glycolysis and gluconeogenesis\"\n",
      "                     /calculated_mol_wt=35472\n",
      "     Region          <5..331\n",
      "                     /region_name=\"Gp_dh_N\"\n",
      "                     /note=\"Glyceraldehyde 3-phosphate dehydrogenase, NAD\n",
      "                     binding domain; cl27764\"\n",
      "                     /db_xref=\"CDD:332585\"\n",
      "     CDS             1..333\n",
      "                     /locus_tag=\"PAS_chr2-1_0437\"\n",
      "                     /coded_by=\"XM_002491300.1:1..1002\"\n",
      "                     /db_xref=\"EnsemblGenomes-Gn:PAS_chr2-1_0437\"\n",
      "                     /db_xref=\"EnsemblGenomes-Tr:CAY69065\"\n",
      "                     /db_xref=\"GeneID:8198905\"\n",
      "                     /db_xref=\"GOA:C4R0P1\"\n",
      "                     /db_xref=\"InterPro:IPR006424\"\n",
      "                     /db_xref=\"InterPro:IPR016040\"\n",
      "                     /db_xref=\"InterPro:IPR020828\"\n",
      "                     /db_xref=\"InterPro:IPR020829\"\n",
      "                     /db_xref=\"InterPro:IPR020830\"\n",
      "                     /db_xref=\"InterPro:IPR020831\"\n",
      "                     /db_xref=\"UniProtKB/TrEMBL:C4R0P1\"\n",
      "ORIGIN      \n",
      "        1 maitvgingf grigrlvlrv alsradikvv aindpfiape yaaymfkyds thkaykgevs\n",
      "       61 asgnkinidg neitvfqerd pvnipwgkag vdyviestgv fttlegaqkh idagakkvvi\n",
      "      121 tapskdapmf vvgvneekyt sdlnivsnas cttnclapla kvvndtfgie sglmttvhsm\n",
      "      181 tatqktvdgp shkdwrggrt asgniipsst gaakavgkvi pelngkltgl afrvptvdvs\n",
      "      241 vvdltvnlkk ettyeeiksv ikaasegklk gvlgytedav vssdflgder ssifdasagi\n",
      "      301 qltpsfvkli swydneygys trvvdllqhv aka\n",
      "//\n",
      "\n",
      "\n",
      "LOCUS       XP_002491303             989 aa            linear   PLN 11-OCT-2017\n",
      "DEFINITION  Alpha subunit of heterooctameric phosphofructokinase involved in\n",
      "            glycolysis [Komagataella phaffii GS115].\n",
      "ACCESSION   XP_002491303\n",
      "VERSION     XP_002491303.1\n",
      "DBLINK      BioProject: PRJNA39439\n",
      "            BioSample: SAMEA2272385\n",
      "DBSOURCE    REFSEQ: accession XM_002491258.1\n",
      "KEYWORDS    RefSeq.\n",
      "SOURCE      Komagataella phaffii GS115\n",
      "  ORGANISM  Komagataella phaffii GS115\n",
      "            Eukaryota; Fungi; Dikarya; Ascomycota; Saccharomycotina;\n",
      "            Saccharomycetes; Saccharomycetales; Phaffomycetaceae; Komagataella.\n",
      "REFERENCE   1\n",
      "  AUTHORS   De Schutter,K., Lin,Y.C., Tiels,P., Van Hecke,A., Glinka,S.,\n",
      "            Weber-Lehmann,J., Rouze,P., Van de Peer,Y. and Callewaert,N.\n",
      "  TITLE     Genome sequence of the recombinant protein production host Pichia\n",
      "            pastoris\n",
      "  JOURNAL   Nat. Biotechnol. 27 (6), 561-566 (2009)\n",
      "   PUBMED   19465926\n",
      "REFERENCE   2  (residues 1 to 989)\n",
      "  CONSRTM   NCBI Genome Project\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (11-OCT-2017) National Center for Biotechnology\n",
      "            Information, NIH, Bethesda, MD 20894, USA\n",
      "REFERENCE   3  (residues 1 to 989)\n",
      "  AUTHORS   Callewaert,N.\n",
      "  TITLE     Direct Submission\n",
      "  JOURNAL   Submitted (05-MAY-2009) Callewaert N., Department for Molecular\n",
      "            Biomedical Research, Flanders Institute for Biotechnology (VIB) /\n",
      "            Gent University, Technologiepark 927, Ghent, 9052, BELGIUM\n",
      "COMMENT     PROVISIONAL REFSEQ: This record has not yet been subject to final\n",
      "            NCBI review. The reference sequence is identical to CAY69023.\n",
      "FEATURES             Location/Qualifiers\n",
      "     source          1..989\n",
      "                     /organism=\"Komagataella phaffii GS115\"\n",
      "                     /strain=\"GS115\"\n",
      "                     /db_xref=\"taxon:644223\"\n",
      "                     /chromosome=\"2\"\n",
      "     Protein         1..989\n",
      "                     /product=\"Alpha subunit of heterooctameric\n",
      "                     phosphofructokinase involved in glycolysis\"\n",
      "                     /calculated_mol_wt=108686\n",
      "     Region          211..985\n",
      "                     /region_name=\"6PF1K_euk\"\n",
      "                     /note=\"6-phosphofructokinase, eukaryotic type; TIGR02478\"\n",
      "                     /db_xref=\"CDD:274152\"\n",
      "     Site            order(220,250,283,314..316,318..319,359,361,363,403..405,\n",
      "                     460,493,496)\n",
      "                     /site_type=\"active\"\n",
      "                     /db_xref=\"CDD:238216\"\n",
      "     Site            order(220,250,283,314..316,318..319)\n",
      "                     /site_type=\"other\"\n",
      "                     /note=\"ADP/pyrophosphate binding site [chemical binding]\"\n",
      "                     /db_xref=\"CDD:238216\"\n",
      "     Site            order(230,234,265,270,273,369,381,385,388,416..417,419,\n",
      "                     451,505..506,510,517,532,578)\n",
      "                     /site_type=\"other\"\n",
      "                     /note=\"dimerization interface [polypeptide binding]\"\n",
      "                     /db_xref=\"CDD:238216\"\n",
      "     Site            order(230,234,265..266,269..270,388,419,421,449,451..453)\n",
      "                     /site_type=\"other\"\n",
      "                     /note=\"allosteric effector site\"\n",
      "                     /db_xref=\"CDD:238216\"\n",
      "     Site            order(359,361,363,396,403..405,460,487,493,496)\n",
      "                     /site_type=\"other\"\n",
      "                     /note=\"fructose-1,6-bisphosphate binding site\"\n",
      "                     /db_xref=\"CDD:238216\"\n",
      "     CDS             1..989\n",
      "                     /locus_tag=\"PAS_chr2-1_0402\"\n",
      "                     /coded_by=\"XM_002491258.1:1..2970\"\n",
      "                     /db_xref=\"EnsemblGenomes-Gn:PAS_chr2-1_0402\"\n",
      "                     /db_xref=\"EnsemblGenomes-Tr:CAY69023\"\n",
      "                     /db_xref=\"GeneID:8198870\"\n",
      "                     /db_xref=\"GOA:Q92448\"\n",
      "                     /db_xref=\"InterPro:IPR000023\"\n",
      "                     /db_xref=\"InterPro:IPR009161\"\n",
      "                     /db_xref=\"InterPro:IPR015912\"\n",
      "                     /db_xref=\"InterPro:IPR022953\"\n",
      "ORIGIN      \n",
      "        1 mpepsisals ftsfvtnddk lfeetfnfyt klgfhatrsy vkdnrsdfel tgistdsike\n",
      "       61 iwlesfplse vvetsagrel rkplqesvgy qseallgysp yqsdgvvikl rlsnhdlqkn\n",
      "      121 kdlpgevtff tasidklrak lieigaeiip seidlvefst kdpmgdvisf ssypslsskk\n",
      "      181 itspdfflhp kkevrsqesi veqvkseegk kkiaiitsgg dapgmnaavr avtragifyg\n",
      "      241 ckvyacyegy tglvkggdml kelqwqdvrg llsiggtiig tarskefrer wgrlqacynm\n",
      "      301 vsngidalvv cggdgsltga dlfrnewpel ikellgegki tkeqyethrn ltivglvgsi\n",
      "      361 dndmcgtdst igayssleri ielvdyidat aashsrafvv evmgrhcgwl glmsgiatga\n",
      "      421 dyifiperpp setnwkddlk kvclrhrekg rrkttvivae gaiddqlnpi tseevkdvlv\n",
      "      481 eigldtritr lghvqrggap cafdrflatv qgvdavravl estpaipspv isilenkivr\n",
      "      541 qplvesvaqt ktvsdaieak dfdkalklrd qefatsyesf lsvskyddgs ylvpessrln\n",
      "      601 iaiihvgapt salnpatrva tlnslakghr vfairngfag lirhgavrel nwidvedwhn\n",
      "      661 tggseigtnr slpsddmgtv ayyfqqykfd gliiiggfea ftalyqldaa raqypifnip\n",
      "      721 mcclpatvsn nvpgteyslg sdtclntlsg ycdavkqsas asrrrtfvve vqggysgyla\n",
      "      781 syaglitgal avytpenpin lqtvqediel ltrtyeeddg knrsgkifih nekaskvytt\n",
      "      841 dliaaiigea gkgrfesrta vpghvqqgks pssidrvnac rlaikccnfi edanfqvkhn\n",
      "      901 anlsaderhl rffyddgvkt savsgkssvi ddntsvvigi qgsevtftpv kqlwekethh\n",
      "      961 kwrkgknvhw eqlnivsdll sgrlsirtt\n",
      "//\n",
      "\n",
      "\n"
     ]
    }
   ],
   "source": [
    "from Bio import Entrez\n",
    "Entrez.email = 'nathanbentolila@berkeley.edu'\n",
    "handle = Entrez.esearch(db= 'protein', \n",
    "                       term='Glycolysis',\n",
    "                       sort='relevance',\n",
    "                       idtype='acc')\n",
    "for i in  Entrez.read(handle)['IdList']:\n",
    "    handle = Entrez.efetch(db='nucleotide', id=i, rettype='gb', retmode='text')\n",
    "    print(handle.read())\n",
    "    \n",
    "protein_rows = []\n",
    "for line in pathways.rstrip().split(\"\\n\"):\n",
    "    entry, name = line.split(\"\\t\")\n",
    "    id_name.append([entry,name])"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 35,
   "metadata": {},
   "outputs": [
    {
     "ename": "SyntaxError",
     "evalue": "positional argument follows keyword argument (<ipython-input-35-c55982c5cc1a>, line 1)",
     "output_type": "error",
     "traceback": [
      "\u001b[1;36m  File \u001b[1;32m\"<ipython-input-35-c55982c5cc1a>\"\u001b[1;36m, line \u001b[1;32m1\u001b[0m\n\u001b[1;33m    Entrez.esearch(db='nucleotide', \"ribosomal protein\", sort)\u001b[0m\n\u001b[1;37m                                   ^\u001b[0m\n\u001b[1;31mSyntaxError\u001b[0m\u001b[1;31m:\u001b[0m positional argument follows keyword argument\n"
     ]
    }
   ],
   "source": [
    "Entrez.esearch(db='nucleotide', \"ribosomal protein\", sort)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "metadata": {},
   "outputs": [],
   "source": [
    "%load_ext sql"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'Connected: @None'"
      ]
     },
     "execution_count": 37,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "%sql sqlite://"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 83,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " * sqlite://\n",
      "Done.\n",
      "Done.\n",
      "Done.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 83,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "%%sql\n",
    "CREATE TABLE genes (id INT PRIMARY KEY ASC, name TEXT, description TEXT, chromosome TEXT, start INT, end INT, strand VARCHAR(1));\n",
    "CREATE TABLE enzymes (id INT PRIMARY KEY ASC, name TEXT, function TEXT, EC INT);"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 89,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " * sqlite://\n",
      "Done.\n",
      "1 rows affected.\n",
      "1 rows affected.\n",
      "1 rows affected.\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 89,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "%%sql\n",
    "CREATE TABLE pathways (id INT PRIMARY KEY ASC, name TEXT, description TEXT);\n",
    "INSERT INTO pathways (id, name, description) VALUES (0, 'Glycolysis / Gluconeogenesis','Glycolysis is the process of converting glucose into pyruvate and generating small amounts of ATP (energy) and NADH (reducing power). It is a central pathway that produces important precursor metabolites: six-carbon compounds of glucose-6P and fructose-6P and three-carbon compounds of glycerone-P, glyceraldehyde-3P, glycerate-3P, phosphoenolpyruvate, and pyruvate [MD:M00001]. Acetyl-CoA, another important precursor metabolite, is produced by oxidative decarboxylation of pyruvate [MD:M00307]. When the enzyme genes of this pathway are examined in completely sequenced genomes, the reaction steps of three-carbon compounds from glycerone-P to pyruvate form a conserved core module [MD:M00002], which is found in almost all organisms and which sometimes contains operon structures in bacterial genomes. Gluconeogenesis is a synthesis pathway of glucose from noncarbohydrate precursors. It is essentially a reversal of glycolysis with minor variations of alternative paths [MD:M00003].' );\n",
    "INSERT INTO pathways (id, name, description) VALUES (1, 'Citrate cycle (TCA cycle)', 'The citrate cycle (TCA cycle, Krebs cycle) is an important aerobic pathway for the final steps of the oxidation of carbohydrates and fatty acids. The cycle starts with acetyl-CoA, the activated form of acetate, derived from glycolysis and pyruvate oxidation for carbohydrates and from beta oxidation of fatty acids. The two-carbon acetyl group in acetyl-CoA is transferred to the four-carbon compound of oxaloacetate to form the six-carbon compound of citrate. In a series of reactions two carbons in citrate are oxidized to CO2 and the reaction pathway supplies NADH for use in the oxidative phosphorylation and other metabolic processes. The pathway also supplies important precursor metabolites including 2-oxoglutarate. At the end of the cycle the remaining four-carbon part is transformed back to oxaloacetate. According to the genome sequence data, many organisms seem to lack genes for the full cycle [MD:M00009], but contain genes for specific segments [MD:M00010 M00011].');\n",
    "INSERT INTO pathways (id, name, description) VALUES (2, 'Pentose phosphate pathway', 'The pentose phosphate pathway is a process of glucose turnover that produces NADPH as reducing equivalents and pentoses as essential parts of nucleotides. There are two different phases in the pathway. One is irreversible oxidative phase in which glucose-6P is converted to ribulose-5P by oxidative decarboxylation, and NADPH is generated [MD:M00006]. The other is reversible non-oxidative phase in which phosphorylated sugars are interconverted to generate xylulose-5P, ribulose-5P, and ribose-5P [MD:M00007]. Phosphoribosyl pyrophosphate (PRPP) formed from ribose-5P [MD:M00005] is an activated compound used in the biosynthesis of histidine and purine/pyrimidine nucleotides. This pathway map also shows the Entner-Doudoroff pathway where 6-P-gluconate is dehydrated and then cleaved into pyruvate and glyceraldehyde-3P [MD:M00008].');"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 67,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "path:ko00010\tGlycolysis / Gluconeogenesis\n",
      "path:ko00020\tCitrate cycle (TCA cycle)\n",
      "path:ko00030\tPentose phosphate pathway\n",
      "path:ko00040\tPentose and glucuronate interconversions\n",
      "path:ko00051\tFructose and mannose metabolism\n",
      "path:ko00052\tGalactose metabolism\n",
      "path:ko00053\tAscorbate and aldarate metabolism\n",
      "path:ko00061\tFatty acid biosynthesis\n",
      "path:ko00062\tFatty acid elongation\n",
      "path:ko00071\tFatty acid degradation\n",
      "path:ko00072\tSynthesis and degradation of ketone bodies\n",
      "path:ko00073\tCutin, suberine and wax biosynthesis\n",
      "path:ko00100\tSteroid biosynthesis\n",
      "path:ko00120\tPrimary bile acid biosynthesis\n",
      "path:ko00121\tSecondary bile acid biosynthesis\n",
      "path:ko00130\tUbiquinone and other terpenoid-quinone biosynthesis\n",
      "path:ko00140\tSteroid hormone biosynthesis\n",
      "path:ko00190\tOxidative phosphorylation\n",
      "path:ko00195\tPhotosynthesis\n",
      "path:ko00196\tPhotosynthesis - antenna proteins\n",
      "path:ko00220\tArginine biosynthesis\n",
      "path:ko00230\tPurine metabolism\n",
      "path:ko00231\tPuromycin biosynthesis\n",
      "path:ko00232\tCaffeine metabolism\n",
      "path:ko00240\tPyrimidine metabolism\n",
      "path:ko00250\tAlanine, aspartate and glutamate metabolism\n",
      "path:ko00253\tTetracycline biosynthesis\n",
      "path:ko00254\tAflatoxin biosynthesis\n",
      "path:ko00260\tGlycine, serine and threonine metabolism\n",
      "path:ko00261\tMonobactam biosynthesis\n",
      "path:ko00270\tCysteine and methionine metabolism\n",
      "path:ko00280\tValine, leucine and isoleucine degradation\n",
      "path:ko00281\tGeraniol degradation\n",
      "path:ko00290\tValine, leucine and isoleucine biosynthesis\n",
      "path:ko00300\tLysine biosynthesis\n",
      "path:ko00310\tLysine degradation\n",
      "path:ko00311\tPenicillin and cephalosporin biosynthesis\n",
      "path:ko00330\tArginine and proline metabolism\n",
      "path:ko00331\tClavulanic acid biosynthesis\n",
      "path:ko00332\tCarbapenem biosynthesis\n",
      "path:ko00333\tProdigiosin biosynthesis\n",
      "path:ko00340\tHistidine metabolism\n",
      "path:ko00350\tTyrosine metabolism\n",
      "path:ko00360\tPhenylalanine metabolism\n",
      "path:ko00361\tChlorocyclohexane and chlorobenzene degradation\n",
      "path:ko00362\tBenzoate degradation\n",
      "path:ko00363\tBisphenol degradation\n",
      "path:ko00364\tFluorobenzoate degradation\n",
      "path:ko00365\tFurfural degradation\n",
      "path:ko00380\tTryptophan metabolism\n",
      "path:ko00400\tPhenylalanine, tyrosine and tryptophan biosynthesis\n",
      "path:ko00401\tNovobiocin biosynthesis\n",
      "path:ko00402\tBenzoxazinoid biosynthesis\n",
      "path:ko00403\tIndole diterpene alkaloid biosynthesis\n",
      "path:ko00404\tStaurosporine biosynthesis\n",
      "path:ko00405\tPhenazine biosynthesis\n",
      "path:ko00410\tbeta-Alanine metabolism\n",
      "path:ko00430\tTaurine and hypotaurine metabolism\n",
      "path:ko00440\tPhosphonate and phosphinate metabolism\n",
      "path:ko00450\tSelenocompound metabolism\n",
      "path:ko00460\tCyanoamino acid metabolism\n",
      "path:ko00471\tD-Glutamine and D-glutamate metabolism\n",
      "path:ko00472\tD-Arginine and D-ornithine metabolism\n",
      "path:ko00473\tD-Alanine metabolism\n",
      "path:ko00480\tGlutathione metabolism\n",
      "path:ko00500\tStarch and sucrose metabolism\n",
      "path:ko00510\tN-Glycan biosynthesis\n",
      "path:ko00511\tOther glycan degradation\n",
      "path:ko00512\tMucin type O-glycan biosynthesis\n",
      "path:ko00513\tVarious types of N-glycan biosynthesis\n",
      "path:ko00514\tOther types of O-glycan biosynthesis\n",
      "path:ko00515\tMannose type O-glycan biosynthesis\n",
      "path:ko00520\tAmino sugar and nucleotide sugar metabolism\n",
      "path:ko00521\tStreptomycin biosynthesis\n",
      "path:ko00522\tBiosynthesis of 12-, 14- and 16-membered macrolides\n",
      "path:ko00523\tPolyketide sugar unit biosynthesis\n",
      "path:ko00524\tNeomycin, kanamycin and gentamicin biosynthesis\n",
      "path:ko00525\tAcarbose and validamycin biosynthesis\n",
      "path:ko00531\tGlycosaminoglycan degradation\n",
      "path:ko00532\tGlycosaminoglycan biosynthesis - chondroitin sulfate / dermatan sulfate\n",
      "path:ko00533\tGlycosaminoglycan biosynthesis - keratan sulfate\n",
      "path:ko00534\tGlycosaminoglycan biosynthesis - heparan sulfate / heparin\n",
      "path:ko00540\tLipopolysaccharide biosynthesis\n",
      "path:ko00550\tPeptidoglycan biosynthesis\n",
      "path:ko00561\tGlycerolipid metabolism\n",
      "path:ko00562\tInositol phosphate metabolism\n",
      "path:ko00563\tGlycosylphosphatidylinositol (GPI)-anchor biosynthesis\n",
      "path:ko00564\tGlycerophospholipid metabolism\n",
      "path:ko00565\tEther lipid metabolism\n",
      "path:ko00571\tLipoarabinomannan (LAM) biosynthesis\n",
      "path:ko00572\tArabinogalactan biosynthesis - Mycobacterium\n",
      "path:ko00590\tArachidonic acid metabolism\n",
      "path:ko00591\tLinoleic acid metabolism\n",
      "path:ko00592\talpha-Linolenic acid metabolism\n",
      "path:ko00600\tSphingolipid metabolism\n",
      "path:ko00601\tGlycosphingolipid biosynthesis - lacto and neolacto series\n",
      "path:ko00603\tGlycosphingolipid biosynthesis - globo and isoglobo series\n",
      "path:ko00604\tGlycosphingolipid biosynthesis - ganglio series\n",
      "path:ko00620\tPyruvate metabolism\n",
      "path:ko00621\tDioxin degradation\n",
      "path:ko00622\tXylene degradation\n",
      "path:ko00623\tToluene degradation\n",
      "path:ko00624\tPolycyclic aromatic hydrocarbon degradation\n",
      "path:ko00625\tChloroalkane and chloroalkene degradation\n",
      "path:ko00626\tNaphthalene degradation\n",
      "path:ko00627\tAminobenzoate degradation\n",
      "path:ko00630\tGlyoxylate and dicarboxylate metabolism\n",
      "path:ko00633\tNitrotoluene degradation\n",
      "path:ko00640\tPropanoate metabolism\n",
      "path:ko00642\tEthylbenzene degradation\n",
      "path:ko00643\tStyrene degradation\n",
      "path:ko00650\tButanoate metabolism\n",
      "path:ko00660\tC5-Branched dibasic acid metabolism\n",
      "path:ko00670\tOne carbon pool by folate\n",
      "path:ko00680\tMethane metabolism\n",
      "path:ko00710\tCarbon fixation in photosynthetic organisms\n",
      "path:ko00720\tCarbon fixation pathways in prokaryotes\n",
      "path:ko00730\tThiamine metabolism\n",
      "path:ko00740\tRiboflavin metabolism\n",
      "path:ko00750\tVitamin B6 metabolism\n",
      "path:ko00760\tNicotinate and nicotinamide metabolism\n",
      "path:ko00770\tPantothenate and CoA biosynthesis\n",
      "path:ko00780\tBiotin metabolism\n",
      "path:ko00785\tLipoic acid metabolism\n",
      "path:ko00790\tFolate biosynthesis\n",
      "path:ko00791\tAtrazine degradation\n",
      "path:ko00830\tRetinol metabolism\n",
      "path:ko00860\tPorphyrin and chlorophyll metabolism\n",
      "path:ko00900\tTerpenoid backbone biosynthesis\n",
      "path:ko00901\tIndole alkaloid biosynthesis\n",
      "path:ko00902\tMonoterpenoid biosynthesis\n",
      "path:ko00903\tLimonene and pinene degradation\n",
      "path:ko00904\tDiterpenoid biosynthesis\n",
      "path:ko00905\tBrassinosteroid biosynthesis\n",
      "path:ko00906\tCarotenoid biosynthesis\n",
      "path:ko00908\tZeatin biosynthesis\n",
      "path:ko00909\tSesquiterpenoid and triterpenoid biosynthesis\n",
      "path:ko00910\tNitrogen metabolism\n",
      "path:ko00920\tSulfur metabolism\n",
      "path:ko00930\tCaprolactam degradation\n",
      "path:ko00940\tPhenylpropanoid biosynthesis\n",
      "path:ko00941\tFlavonoid biosynthesis\n",
      "path:ko00942\tAnthocyanin biosynthesis\n",
      "path:ko00943\tIsoflavonoid biosynthesis\n",
      "path:ko00944\tFlavone and flavonol biosynthesis\n",
      "path:ko00945\tStilbenoid, diarylheptanoid and gingerol biosynthesis\n",
      "path:ko00950\tIsoquinoline alkaloid biosynthesis\n",
      "path:ko00960\tTropane, piperidine and pyridine alkaloid biosynthesis\n",
      "path:ko00965\tBetalain biosynthesis\n",
      "path:ko00966\tGlucosinolate biosynthesis\n",
      "path:ko00970\tAminoacyl-tRNA biosynthesis\n",
      "path:ko00980\tMetabolism of xenobiotics by cytochrome P450\n",
      "path:ko00981\tInsect hormone biosynthesis\n",
      "path:ko00982\tDrug metabolism - cytochrome P450\n",
      "path:ko00983\tDrug metabolism - other enzymes\n",
      "path:ko00984\tSteroid degradation\n",
      "path:ko00999\tBiosynthesis of secondary metabolites - unclassified\n",
      "path:ko01040\tBiosynthesis of unsaturated fatty acids\n",
      "path:ko01051\tBiosynthesis of ansamycins\n",
      "path:ko01052\tType I polyketide structures\n",
      "path:ko01053\tBiosynthesis of siderophore group nonribosomal peptides\n",
      "path:ko01054\tNonribosomal peptide structures\n",
      "path:ko01055\tBiosynthesis of vancomycin group antibiotics\n",
      "path:ko01056\tBiosynthesis of type II polyketide backbone\n",
      "path:ko01057\tBiosynthesis of type II polyketide products\n",
      "path:ko01058\tAcridone alkaloid biosynthesis\n",
      "path:ko01059\tBiosynthesis of enediyne antibiotics\n",
      "path:ko01100\tMetabolic pathways\n",
      "path:ko01110\tBiosynthesis of secondary metabolites\n",
      "path:ko01120\tMicrobial metabolism in diverse environments\n",
      "path:ko01130\tBiosynthesis of antibiotics\n",
      "path:ko01200\tCarbon metabolism\n",
      "path:ko01210\t2-Oxocarboxylic acid metabolism\n",
      "path:ko01212\tFatty acid metabolism\n",
      "path:ko01220\tDegradation of aromatic compounds\n",
      "path:ko01230\tBiosynthesis of amino acids\n",
      "path:ko01501\tbeta-Lactam resistance\n",
      "path:ko01502\tVancomycin resistance\n",
      "path:ko01503\tCationic antimicrobial peptide (CAMP) resistance\n",
      "path:ko01521\tEGFR tyrosine kinase inhibitor resistance\n",
      "path:ko01522\tEndocrine resistance\n",
      "path:ko01523\tAntifolate resistance\n",
      "path:ko01524\tPlatinum drug resistance\n",
      "path:ko02010\tABC transporters\n",
      "path:ko02020\tTwo-component system\n",
      "path:ko02024\tQuorum sensing\n",
      "path:ko02025\tBiofilm formation - Pseudomonas aeruginosa\n",
      "path:ko02026\tBiofilm formation - Escherichia coli\n",
      "path:ko02030\tBacterial chemotaxis\n",
      "path:ko02040\tFlagellar assembly\n",
      "path:ko02060\tPhosphotransferase system (PTS)\n",
      "path:ko03008\tRibosome biogenesis in eukaryotes\n",
      "path:ko03010\tRibosome\n",
      "path:ko03013\tRNA transport\n",
      "path:ko03015\tmRNA surveillance pathway\n",
      "path:ko03018\tRNA degradation\n",
      "path:ko03020\tRNA polymerase\n",
      "path:ko03022\tBasal transcription factors\n",
      "path:ko03030\tDNA replication\n",
      "path:ko03040\tSpliceosome\n",
      "path:ko03050\tProteasome\n",
      "path:ko03060\tProtein export\n",
      "path:ko03070\tBacterial secretion system\n",
      "path:ko03320\tPPAR signaling pathway\n",
      "path:ko03410\tBase excision repair\n",
      "path:ko03420\tNucleotide excision repair\n",
      "path:ko03430\tMismatch repair\n",
      "path:ko03440\tHomologous recombination\n",
      "path:ko03450\tNon-homologous end-joining\n",
      "path:ko03460\tFanconi anemia pathway\n",
      "path:ko04010\tMAPK signaling pathway\n",
      "path:ko04011\tMAPK signaling pathway - yeast\n",
      "path:ko04012\tErbB signaling pathway\n",
      "path:ko04013\tMAPK signaling pathway - fly\n",
      "path:ko04014\tRas signaling pathway\n",
      "path:ko04015\tRap1 signaling pathway\n",
      "path:ko04016\tMAPK signaling pathway - plant\n",
      "path:ko04020\tCalcium signaling pathway\n",
      "path:ko04022\tcGMP-PKG signaling pathway\n",
      "path:ko04024\tcAMP signaling pathway\n",
      "path:ko04060\tCytokine-cytokine receptor interaction\n",
      "path:ko04062\tChemokine signaling pathway\n",
      "path:ko04064\tNF-kappa B signaling pathway\n",
      "path:ko04066\tHIF-1 signaling pathway\n",
      "path:ko04068\tFoxO signaling pathway\n",
      "path:ko04070\tPhosphatidylinositol signaling system\n",
      "path:ko04071\tSphingolipid signaling pathway\n",
      "path:ko04072\tPhospholipase D signaling pathway\n",
      "path:ko04075\tPlant hormone signal transduction\n",
      "path:ko04080\tNeuroactive ligand-receptor interaction\n",
      "path:ko04110\tCell cycle\n",
      "path:ko04111\tCell cycle - yeast\n",
      "path:ko04112\tCell cycle - Caulobacter\n",
      "path:ko04113\tMeiosis - yeast\n",
      "path:ko04114\tOocyte meiosis\n",
      "path:ko04115\tp53 signaling pathway\n",
      "path:ko04120\tUbiquitin mediated proteolysis\n",
      "path:ko04122\tSulfur relay system\n",
      "path:ko04130\tSNARE interactions in vesicular transport\n",
      "path:ko04136\tAutophagy - other\n",
      "path:ko04137\tMitophagy - animal\n",
      "path:ko04138\tAutophagy - yeast\n",
      "path:ko04139\tMitophagy - yeast\n",
      "path:ko04140\tAutophagy - animal\n",
      "path:ko04141\tProtein processing in endoplasmic reticulum\n",
      "path:ko04142\tLysosome\n",
      "path:ko04144\tEndocytosis\n",
      "path:ko04145\tPhagosome\n",
      "path:ko04146\tPeroxisome\n",
      "path:ko04150\tmTOR signaling pathway\n",
      "path:ko04151\tPI3K-Akt signaling pathway\n",
      "path:ko04152\tAMPK signaling pathway\n",
      "path:ko04210\tApoptosis\n",
      "path:ko04211\tLongevity regulating pathway\n",
      "path:ko04212\tLongevity regulating pathway - worm\n",
      "path:ko04213\tLongevity regulating pathway - multiple species\n",
      "path:ko04214\tApoptosis - fly\n",
      "path:ko04215\tApoptosis - multiple species\n",
      "path:ko04216\tFerroptosis\n",
      "path:ko04217\tNecroptosis\n",
      "path:ko04218\tCellular senescence\n",
      "path:ko04260\tCardiac muscle contraction\n",
      "path:ko04261\tAdrenergic signaling in cardiomyocytes\n",
      "path:ko04270\tVascular smooth muscle contraction\n",
      "path:ko04310\tWnt signaling pathway\n",
      "path:ko04320\tDorso-ventral axis formation\n",
      "path:ko04330\tNotch signaling pathway\n",
      "path:ko04340\tHedgehog signaling pathway\n",
      "path:ko04341\tHedgehog signaling pathway - fly\n",
      "path:ko04350\tTGF-beta signaling pathway\n",
      "path:ko04360\tAxon guidance\n",
      "path:ko04370\tVEGF signaling pathway\n",
      "path:ko04371\tApelin signaling pathway\n",
      "path:ko04380\tOsteoclast differentiation\n",
      "path:ko04390\tHippo signaling pathway\n",
      "path:ko04391\tHippo signaling pathway - fly\n",
      "path:ko04392\tHippo signaling pathway - multiple species\n",
      "path:ko04510\tFocal adhesion\n",
      "path:ko04512\tECM-receptor interaction\n",
      "path:ko04514\tCell adhesion molecules (CAMs)\n",
      "path:ko04520\tAdherens junction\n",
      "path:ko04530\tTight junction\n",
      "path:ko04540\tGap junction\n",
      "path:ko04550\tSignaling pathways regulating pluripotency of stem cells\n",
      "path:ko04610\tComplement and coagulation cascades\n",
      "path:ko04611\tPlatelet activation\n",
      "path:ko04612\tAntigen processing and presentation\n",
      "path:ko04614\tRenin-angiotensin system\n",
      "path:ko04620\tToll-like receptor signaling pathway\n",
      "path:ko04621\tNOD-like receptor signaling pathway\n",
      "path:ko04622\tRIG-I-like receptor signaling pathway\n",
      "path:ko04623\tCytosolic DNA-sensing pathway\n",
      "path:ko04624\tToll and Imd signaling pathway\n",
      "path:ko04625\tC-type lectin receptor signaling pathway\n",
      "path:ko04626\tPlant-pathogen interaction\n",
      "path:ko04630\tJAK-STAT signaling pathway\n",
      "path:ko04640\tHematopoietic cell lineage\n",
      "path:ko04650\tNatural killer cell mediated cytotoxicity\n",
      "path:ko04657\tIL-17 signaling pathway\n",
      "path:ko04658\tTh1 and Th2 cell differentiation\n",
      "path:ko04659\tTh17 cell differentiation\n",
      "path:ko04660\tT cell receptor signaling pathway\n",
      "path:ko04662\tB cell receptor signaling pathway\n",
      "path:ko04664\tFc epsilon RI signaling pathway\n",
      "path:ko04666\tFc gamma R-mediated phagocytosis\n",
      "path:ko04668\tTNF signaling pathway\n",
      "path:ko04670\tLeukocyte transendothelial migration\n",
      "path:ko04672\tIntestinal immune network for IgA production\n",
      "path:ko04710\tCircadian rhythm\n",
      "path:ko04711\tCircadian rhythm - fly\n",
      "path:ko04712\tCircadian rhythm - plant\n",
      "path:ko04713\tCircadian entrainment\n",
      "path:ko04714\tThermogenesis\n",
      "path:ko04720\tLong-term potentiation\n",
      "path:ko04721\tSynaptic vesicle cycle\n",
      "path:ko04722\tNeurotrophin signaling pathway\n",
      "path:ko04723\tRetrograde endocannabinoid signaling\n",
      "path:ko04724\tGlutamatergic synapse\n",
      "path:ko04725\tCholinergic synapse\n",
      "path:ko04726\tSerotonergic synapse\n",
      "path:ko04727\tGABAergic synapse\n",
      "path:ko04728\tDopaminergic synapse\n",
      "path:ko04730\tLong-term depression\n",
      "path:ko04740\tOlfactory transduction\n",
      "path:ko04742\tTaste transduction\n",
      "path:ko04744\tPhototransduction\n",
      "path:ko04745\tPhototransduction - fly\n",
      "path:ko04750\tInflammatory mediator regulation of TRP channels\n",
      "path:ko04810\tRegulation of actin cytoskeleton\n",
      "path:ko04910\tInsulin signaling pathway\n",
      "path:ko04911\tInsulin secretion\n",
      "path:ko04912\tGnRH signaling pathway\n",
      "path:ko04913\tOvarian steroidogenesis\n",
      "path:ko04914\tProgesterone-mediated oocyte maturation\n",
      "path:ko04915\tEstrogen signaling pathway\n",
      "path:ko04916\tMelanogenesis\n",
      "path:ko04917\tProlactin signaling pathway\n",
      "path:ko04918\tThyroid hormone synthesis\n",
      "path:ko04919\tThyroid hormone signaling pathway\n",
      "path:ko04920\tAdipocytokine signaling pathway\n",
      "path:ko04921\tOxytocin signaling pathway\n",
      "path:ko04922\tGlucagon signaling pathway\n",
      "path:ko04923\tRegulation of lipolysis in adipocytes\n",
      "path:ko04924\tRenin secretion\n",
      "path:ko04925\tAldosterone synthesis and secretion\n",
      "path:ko04926\tRelaxin signaling pathway\n",
      "path:ko04927\tCortisol synthesis and secretion\n",
      "path:ko04928\tParathyroid hormone synthesis, secretion and action\n",
      "path:ko04930\tType II diabetes mellitus\n",
      "path:ko04931\tInsulin resistance\n",
      "path:ko04932\tNon-alcoholic fatty liver disease (NAFLD)\n",
      "path:ko04933\tAGE-RAGE signaling pathway in diabetic complications\n",
      "path:ko04934\tCushing syndrome\n",
      "path:ko04940\tType I diabetes mellitus\n",
      "path:ko04950\tMaturity onset diabetes of the young\n",
      "path:ko04960\tAldosterone-regulated sodium reabsorption\n",
      "path:ko04961\tEndocrine and other factor-regulated calcium reabsorption\n",
      "path:ko04962\tVasopressin-regulated water reabsorption\n",
      "path:ko04964\tProximal tubule bicarbonate reclamation\n",
      "path:ko04966\tCollecting duct acid secretion\n",
      "path:ko04970\tSalivary secretion\n",
      "path:ko04971\tGastric acid secretion\n",
      "path:ko04972\tPancreatic secretion\n",
      "path:ko04973\tCarbohydrate digestion and absorption\n",
      "path:ko04974\tProtein digestion and absorption\n",
      "path:ko04975\tFat digestion and absorption\n",
      "path:ko04976\tBile secretion\n",
      "path:ko04977\tVitamin digestion and absorption\n",
      "path:ko04978\tMineral absorption\n",
      "path:ko04979\tCholesterol metabolism\n",
      "path:ko05010\tAlzheimer disease\n",
      "path:ko05012\tParkinson disease\n",
      "path:ko05014\tAmyotrophic lateral sclerosis (ALS)\n",
      "path:ko05016\tHuntington disease\n",
      "path:ko05020\tPrion diseases\n",
      "path:ko05030\tCocaine addiction\n",
      "path:ko05031\tAmphetamine addiction\n",
      "path:ko05032\tMorphine addiction\n",
      "path:ko05033\tNicotine addiction\n",
      "path:ko05034\tAlcoholism\n",
      "path:ko05100\tBacterial invasion of epithelial cells\n",
      "path:ko05110\tVibrio cholerae infection\n",
      "path:ko05111\tBiofilm formation - Vibrio cholerae\n",
      "path:ko05120\tEpithelial cell signaling in Helicobacter pylori infection\n",
      "path:ko05130\tPathogenic Escherichia coli infection\n",
      "path:ko05131\tShigellosis\n",
      "path:ko05132\tSalmonella infection\n",
      "path:ko05133\tPertussis\n",
      "path:ko05134\tLegionellosis\n",
      "path:ko05140\tLeishmaniasis\n",
      "path:ko05142\tChagas disease (American trypanosomiasis)\n",
      "path:ko05143\tAfrican trypanosomiasis\n",
      "path:ko05144\tMalaria\n",
      "path:ko05145\tToxoplasmosis\n",
      "path:ko05146\tAmoebiasis\n",
      "path:ko05150\tStaphylococcus aureus infection\n",
      "path:ko05152\tTuberculosis\n",
      "path:ko05160\tHepatitis C\n",
      "path:ko05161\tHepatitis B\n",
      "path:ko05162\tMeasles\n",
      "path:ko05163\tHuman cytomegalovirus infection\n",
      "path:ko05164\tInfluenza A\n",
      "path:ko05165\tHuman papillomavirus infection\n",
      "path:ko05166\tHuman T-cell leukemia virus 1 infection\n",
      "path:ko05167\tKaposi sarcoma-associated herpesvirus infection\n",
      "path:ko05168\tHerpes simplex infection\n",
      "path:ko05169\tEpstein-Barr virus infection\n",
      "path:ko05170\tHuman immunodeficiency virus 1 infection\n",
      "path:ko05200\tPathways in cancer\n",
      "path:ko05202\tTranscriptional misregulation in cancer\n",
      "path:ko05203\tViral carcinogenesis\n",
      "path:ko05204\tChemical carcinogenesis\n",
      "path:ko05205\tProteoglycans in cancer\n",
      "path:ko05206\tMicroRNAs in cancer\n",
      "path:ko05210\tColorectal cancer\n",
      "path:ko05211\tRenal cell carcinoma\n",
      "path:ko05212\tPancreatic cancer\n",
      "path:ko05213\tEndometrial cancer\n",
      "path:ko05214\tGlioma\n",
      "path:ko05215\tProstate cancer\n",
      "path:ko05216\tThyroid cancer\n",
      "path:ko05217\tBasal cell carcinoma\n",
      "path:ko05218\tMelanoma\n",
      "path:ko05219\tBladder cancer\n",
      "path:ko05220\tChronic myeloid leukemia\n",
      "path:ko05221\tAcute myeloid leukemia\n",
      "path:ko05222\tSmall cell lung cancer\n",
      "path:ko05223\tNon-small cell lung cancer\n",
      "path:ko05224\tBreast cancer\n",
      "path:ko05225\tHepatocellular carcinoma\n",
      "path:ko05226\tGastric cancer\n",
      "path:ko05230\tCentral carbon metabolism in cancer\n",
      "path:ko05231\tCholine metabolism in cancer\n",
      "path:ko05310\tAsthma\n",
      "path:ko05320\tAutoimmune thyroid disease\n",
      "path:ko05321\tInflammatory bowel disease (IBD)\n",
      "path:ko05322\tSystemic lupus erythematosus\n",
      "path:ko05323\tRheumatoid arthritis\n",
      "path:ko05330\tAllograft rejection\n",
      "path:ko05332\tGraft-versus-host disease\n",
      "path:ko05340\tPrimary immunodeficiency\n",
      "path:ko05410\tHypertrophic cardiomyopathy (HCM)\n",
      "path:ko05412\tArrhythmogenic right ventricular cardiomyopathy (ARVC)\n",
      "path:ko05414\tDilated cardiomyopathy (DCM)\n",
      "path:ko05416\tViral myocarditis\n",
      "path:ko05418\tFluid shear stress and atherosclerosis\n",
      "\n",
      "[['path:ko00010', 'Glycolysis / Gluconeogenesis'], ['path:ko00020', 'Citrate cycle (TCA cycle)'], ['path:ko00030', 'Pentose phosphate pathway']]\n"
     ]
    }
   ],
   "source": [
    "from Bio.KEGG import REST\n",
    "\n",
    "pathways = REST.kegg_list(\"pathway\", \"ko\").read()\n",
    "print(pathways)\n",
    "id_name = []\n",
    "for line in pathways.rstrip().split(\"\\n\"):\n",
    "    entry, name = line.split(\"\\t\")\n",
    "    id_name.append([entry,name])\n",
    "\n",
    "\n",
    "filtered_id_name = []\n",
    "for i in range(len(id_name)):\n",
    "    if id_name[i][1]=='Glycolysis / Gluconeogenesis' or id_name[i][1]=='Citrate cycle (TCA cycle)' or id_name[i][1]=='Pentose phosphate pathway':\n",
    "        filtered_id_name.append(id_name[i])\n",
    "\n",
    "print(filtered_id_name)\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 61,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      " * sqlite://\n",
      "(sqlite3.OperationalError) no such column: path [SQL: 'INSERT INTO pathways (id, name) VALUES (?,?), (path,name)'] (Background on this error at: http://sqlalche.me/e/e3q8)\n",
      " * sqlite://\n",
      "(sqlite3.OperationalError) no such column: path [SQL: 'INSERT INTO pathways (id, name) VALUES (?,?), (path,name)'] (Background on this error at: http://sqlalche.me/e/e3q8)\n",
      " * sqlite://\n",
      "(sqlite3.OperationalError) no such column: path [SQL: 'INSERT INTO pathways (id, name) VALUES (?,?), (path,name)'] (Background on this error at: http://sqlalche.me/e/e3q8)\n"
     ]
    }
   ],
   "source": [
    "for i in range(len(filtered_id_name)):\n",
    "    path = filtered_id_name[i][0][5:]\n",
    "    name = filtered_id_name[i][1]\n",
    "    %sql INSERT INTO pathways (id, name) VALUES (?,?), (path,name)\n",
    "    "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "anaconda-cloud": {},
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
